<html>
<head><meta http-equiv=Content-Type content="text/html; charset=UTF-8">
<style type="text/css">
<!--
span.cls_002{font-family:Times,serif;font-size:15.9px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_002{font-family:Times,serif;font-size:15.9px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_003{font-family:Times,serif;font-size:10.4px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_003{font-family:Times,serif;font-size:10.4px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_005{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_005{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_006{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_006{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_009{font-family:Times,serif;font-size:6.5px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_009{font-family:Times,serif;font-size:6.5px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_010{font-family:Times,serif;font-size:9.9px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_010{font-family:Times,serif;font-size:9.9px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_012{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_012{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_013{font-family:Times,serif;font-size:12.1px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_013{font-family:Times,serif;font-size:12.1px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_024{font-family:Times,serif;font-size:12.1px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: underline}
div.cls_024{font-family:Times,serif;font-size:12.1px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_015{font-family:Times,serif;font-size:8.0px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_015{font-family:Times,serif;font-size:8.0px;color:rgb(51,51,51);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_025{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: underline}
div.cls_025{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_016{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_016{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_017{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_017{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_018{font-family:Times,serif;font-size:8.0px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_018{font-family:Times,serif;font-size:8.0px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_019{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:italic;text-decoration: none}
div.cls_019{font-family:Times,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:italic;text-decoration: none}
span.cls_026{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: underline}
div.cls_026{font-family:Times,serif;font-size:12.1px;color:rgb(38,37,38);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_007{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
div.cls_007{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:bold;font-style:normal;text-decoration: none}
span.cls_020{font-family:Arial,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_020{font-family:Arial,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_021{font-family:Arial,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_021{font-family:Arial,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_022{font-family:Courier New,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
div.cls_022{font-family:Courier New,serif;font-size:12.1px;color:rgb(0,0,0);font-weight:normal;font-style:normal;text-decoration: none}
span.cls_023{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:normal;font-style:italic;text-decoration: none}
div.cls_023{font-family:Times,serif;font-size:8.0px;color:rgb(0,0,0);font-weight:normal;font-style:italic;text-decoration: none}
-->
</style>
<script type="text/javascript" src="BIORXIV-2019-555854v1-Baker_files/wz_jsgraphics.js"></script>
</head>
<body>
<div style="position:absolute;left:50%;margin-left:-306px;top:0px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-01.jpg" width=612 height=792></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_002"><span class="cls_002">Systemic Lymphoid Architecture Response Assessment (SYLARAS): An</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_002"><span class="cls_002">approach to multi-organ, discovery-based immunophenotyping implicates a</span></div>
<div style="position:absolute;left:54.00px;top:124.32px" class="cls_002"><span class="cls_002">role for CD45R/B220</span><span class="cls_003"><sup>+</sup></span><span class="cls_002"> CD8T cells in glioblastoma immunology</span></div>
<div style="position:absolute;left:54.00px;top:161.04px" class="cls_005"><span class="cls_005">Gregory J. Baker</span><span class="cls_006"><sup>1,2,5*</sup></span><span class="cls_005">, Sucheendra K. Palaniappan</span><span class="cls_006"><sup>3</sup></span><span class="cls_005">, Jodene K. Moore</span><span class="cls_006"><sup>2</sup></span><span class="cls_005">, Stephanie H. Davis</span><span class="cls_006"><sup>1</sup></span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:181.68px" class="cls_005"><span class="cls_005">Sandro Santagata</span><span class="cls_006"><sup>1,4,5,6,7</sup></span><span class="cls_005"> and Peter K. Sorger</span><span class="cls_006"><sup>1,2,5*</sup></span></div>
<div style="position:absolute;left:54.00px;top:211.44px" class="cls_009"><span class="cls_009"><sup>1</sup></span><span class="cls_010">Laboratory of Systems Pharmacology, Program in Therapeutic Science, Harvard Medical School, Boston, MA 02115, USA.</span></div>
<div style="position:absolute;left:54.00px;top:228.72px" class="cls_009"><span class="cls_009"><sup>2</sup></span><span class="cls_010">Department of Systems Biology, Harvard Medical School, Boston, MA 02115 USA.</span></div>
<div style="position:absolute;left:54.00px;top:246.00px" class="cls_009"><span class="cls_009"><sup>3</sup></span><span class="cls_010">The Systems Biology Institute, Tokyo 141-0022 Japan.</span></div>
<div style="position:absolute;left:54.00px;top:263.28px" class="cls_009"><span class="cls_009"><sup>4</sup></span><span class="cls_010">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115 USA</span></div>
<div style="position:absolute;left:54.00px;top:280.56px" class="cls_009"><span class="cls_009"><sup>5</sup></span><span class="cls_010">Ludwig Center for Cancer Research at Harvard, Harvard Medical School, Boston, MA</span></div>
<div style="position:absolute;left:54.00px;top:297.84px" class="cls_009"><span class="cls_009"><sup>6</sup></span><span class="cls_010">Department of Oncologic Pathology, Dana Farber Cancer Institute, Boston, MA</span></div>
<div style="position:absolute;left:54.00px;top:315.12px" class="cls_009"><span class="cls_009"><sup>7</sup></span><span class="cls_010">Department of Pathology, Boston Children’s Hospital, Boston, MA</span></div>
<div style="position:absolute;left:54.00px;top:352.08px" class="cls_012"><span class="cls_012">Key Words:</span><span class="cls_005"> SYLARAS (Systemic Lymphoid Architecture Response Assessment), systems</span></div>
<div style="position:absolute;left:54.00px;top:372.96px" class="cls_005"><span class="cls_005">immunology, glioblastoma (GBM)</span></div>
<div style="position:absolute;left:54.00px;top:408.24px" class="cls_012"><span class="cls_012">Running Title: </span><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:437.52px" class="cls_012"><span class="cls_012">Corresponding Authors:</span></div>
<div style="position:absolute;left:54.00px;top:458.16px" class="cls_013"><span class="cls_013">Gregory J. Baker PharmD, PhD</span></div>
<div style="position:absolute;left:54.00px;top:479.04px" class="cls_013"><span class="cls_013">American Cancer Society Postdoctoral Fellow</span></div>
<div style="position:absolute;left:54.00px;top:499.68px" class="cls_013"><span class="cls_013">Tel: (617) 432-6909/ gregory_baker2@hms.harvard.edu</span></div>
<div style="position:absolute;left:54.00px;top:528.96px" class="cls_013"><span class="cls_013">Peter K. Sorger PhD</span></div>
<div style="position:absolute;left:54.00px;top:549.60px" class="cls_013"><span class="cls_013">Otto Krayer Professor of Systems Pharmacology</span></div>
<div style="position:absolute;left:54.00px;top:570.24px" class="cls_013"><span class="cls_013">Tel: (617) 432-6901/ </span><span class="cls_005">peter_sorger@hms.harvard.edu/ cc</span><span class="cls_013">: christopher_bird@hms.harvard.edu</span></div>
<div style="position:absolute;left:54.00px;top:599.76px" class="cls_013"><span class="cls_013">Laboratory of Systems Pharmacology</span></div>
<div style="position:absolute;left:54.00px;top:620.40px" class="cls_013"><span class="cls_013">Department of Systems Biology</span></div>
<div style="position:absolute;left:54.00px;top:641.04px" class="cls_013"><span class="cls_013">Harvard Medical School</span></div>
<div style="position:absolute;left:54.00px;top:661.68px" class="cls_013"><span class="cls_013">200 Longwood Avenue</span></div>
<div style="position:absolute;left:54.00px;top:682.56px" class="cls_013"><span class="cls_013">Warren Alpert Building, Room 440</span></div>
<div style="position:absolute;left:54.00px;top:703.20px" class="cls_013"><span class="cls_013">Boston, MA 02115</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:802px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-02.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">ABSTRACT</span></div>
<div style="position:absolute;left:90.00px;top:98.64px" class="cls_005"><span class="cls_005">Investigational and computational methods for the systematic exploration of global immune</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">response are needed to gain a more integrated comprehension of disease and to explore the mechanisms</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">of immunotherapy. Here were describe</span><span class="cls_013"> a modular and extensible framework for the acquisition,</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_013"><span class="cls_013">analysis, </span><span class="cls_005">and visual display of </span><span class="cls_013">systems-level immunophenotyping data called SYLARAS (</span><span class="cls_024">Sy</span><span class="cls_013">stemic</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_024"><span class="cls_024">L</span><span class="cls_013">ymphoid </span><span class="cls_024">A</span><span class="cls_013">rchitecture </span><span class="cls_024">R</span><span class="cls_013">esponse </span><span class="cls_024">As</span><span class="cls_013">sessment, </span><A HREF="http://www.sylaras.org/">www.sylaras.org</A>)</span><span class="cls_013">. Leveraging our technology against a</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_013"><span class="cls_013">syngeneic mouse model of glioblastoma (GBM) brain cancer, we reveal tumor-induced perturbation in</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_013"><span class="cls_013">the frequency and cell-to-cell co-variation of peripheral immune cell subsets hitherto undescribed in the</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_013"><span class="cls_013">disease. The most novel and significant of our findings relates to a population of CD8</span><span class="cls_015"><sup>+</sup></span><span class="cls_013"> T cells</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_013"><span class="cls_013">characterized by their expression of the CD45R/B220 isoform of the leukocyte common antigen, CD45</span></div>
<div style="position:absolute;left:54.00px;top:345.60px" class="cls_013"><span class="cls_013">(aka B220</span><span class="cls_015"><sup>+</sup></span><span class="cls_013"> CD8T cells). In follow-on studies, we demonstrate that mouse B220</span><span class="cls_015"><sup>+</sup></span><span class="cls_013"> CD8T cells are a</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_013"><span class="cls_013">transcriptionally and morphologically-distinct subset of GBM-infiltrating lymphocytes whose human</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_013"><span class="cls_013">analog is identified in normal human blood and a subset of primary and recurrent GBM clinical biopsies.</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_013"><span class="cls_013">The ability of cells of this immunophenotype to attenuate immune response to autologous antigens, as</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_013"><span class="cls_013">demonstrated by the work of others, suggests that B220</span><span class="cls_015"><sup>+</sup></span><span class="cls_013"> CD8T cells may play an immunoregulatory</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_013"><span class="cls_013">role in GBM. </span><span class="cls_005">By providing ample content for inductive scientific inference in the era of immunotherapy,</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">SYLARAS has the potential to uncover complex cell and molecular mechanisms governing</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">immunologic response.</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">2</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:1604px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-03.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">INTRODUCTION</span></div>
<div style="position:absolute;left:90.00px;top:98.64px" class="cls_005"><span class="cls_005">Glioblastoma (GBM) is a rapidly progressing, incurable brain cancer that grows within the</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">immunospecialized environment of the central nervous system. Like many other solid tumors, GBM</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">exerts potent immunosuppressive effects on tumor-resident immune cells through its production of anti-</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">inflammatory cytokines and catabolites</span><span class="cls_006"><sup>1,2,3,4,5,6,7</sup></span><span class="cls_005">, lectins</span><span class="cls_006"><sup>8,9</sup></span><span class="cls_005">, and immune checkpoint molecules</span><span class="cls_006"><sup>10,11,12</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">Although the success of immunotherapy has been demonstrated to hinge on immunological events</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">taking place outside of the tumor microenvironment (TME)</span><span class="cls_006"><sup>13,14,15</sup></span><span class="cls_005">, both the magnitude and biological</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">significance of interactions occurring between peripheral and tumor-resident immune cell subsets is not</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">well understood. This is in large part due to a lack of suitable tools for the systematic, longitudinal</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">exploration of immune system composition, function, and network-level connectivity. In the specific</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">case of GBM, the tumor’s impact on the number, distribution, and activity of peripheral immune cells</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">constituting the peripheral immune system is particularly unclear, as an in-depth multi-organ analysis of</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">this tumor’s impact on systemic immunity has not yet been performed.</span></div>
<div style="position:absolute;left:90.00px;top:429.84px" class="cls_005"><span class="cls_005">Changes in immune cell number and allocation across lymphoid tissue as a result of the cellular</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">migration, proliferation, and differentiation that accompanies the response to immunologic stimuli can is</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">readily studied by immunophenotyping. The most common method for this is flow cytometry wherein</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">combinations of antibody-based probes targeting integral membrane proteins, intracellular cytokines,</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">and transcription factors are regularly and cost-efficiently used to characterize the frequency and</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">function immune cell subsets. Evaluation of immunophenotypes from multiple lymphoid organs, among</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">biological replicates, both in the presence and absence of an experimental immune perturbation (GBM</span></div>
<div style="position:absolute;left:54.00px;top:621.60px" class="cls_005"><span class="cls_005">in this work) poses a considerable analytical challenge</span><span class="cls_006"><sup>16</sup></span><span class="cls_005">. Conventional tools that are well-suited to the</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">analysis of a few samples, such as those with a native graphical user-interface (GUI) like FlowJo</span><span class="cls_006"><sup>17</sup></span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">become cumbersome and error-prone when applied to increasingly large datasets.</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">3</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:2406px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-04.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">In this paper, we describe a set of software tools and experimental protocols that overcome the</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">challenges of immunophenotyping across time and tissue that are collectively referred to as SYLARAS</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">(</span><span class="cls_025">Sy</span><span class="cls_005">stemic </span><span class="cls_025">L</span><span class="cls_005">ymphoid </span><span class="cls_025">Ar</span><span class="cls_005">chitecture </span><span class="cls_025">R</span><span class="cls_005">esponse </span><span class="cls_025">As</span><span class="cls_005">sessment). Application of its technology to the</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">systematic identification and analysis of cellular immunophenotypes contained within a single-cell</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">dataset spanning 240 tissue samples collected from 5 different lymphoid organs of age-matched mice</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">inoculated with </span><span class="cls_016">syngeneic</span><span class="cls_005"> </span><span class="cls_016">mouse GBM cells or vehicle alone, reveals the breadth of impact </span><span class="cls_005">intracranial</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">tumor burden has on systemic immune cell populations.</span><span class="cls_016"> In making </span><span class="cls_005">SYLARAS technology accessible to</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">the greatest number of labs, we demonstrate the technique using standard 12-color flow cytometry;</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">however, the approach is readily compatible with highly multiplexed methods such as mass cytometry</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">(CyTOF)</span><span class="cls_006"><sup>18</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">The key aspect of SYLARAS is its open-source computational analysis pipeline scripted in the</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">Python programming language and useful for transforming single-cell data into a comprehensive</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">overview of peripheral immune dynamics is response to immunologic disease and therapy. By providing</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">parallel methods for knowledge-based and data-driven cell subset classification, SYLARAS allows for a</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">direct comparison of the results obtained through supervised and unsupervised analysis. SYLARAS</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">outputs a set of data-rich, easily interpretable graphical dashboards (1 per cell type) that serve as a</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">concise compendium of the time and tissue-dependent changes occurring across multiple tissue sites in</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">response to an arbitrary immune stimulus. We provide experimental protocols, source code, and details</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">on flow cytometry antibodies and cytometer configuration settings, in addition to the full GBM</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">immunophenotyping dataset, as open-access resources at </span><A HREF="http://www.sylaras.org./">www.sylaras.org.</A> </div>
<div style="position:absolute;left:90.00px;top:623.04px" class="cls_005"><span class="cls_005">In demonstrating the utility of the </span><span class="cls_013">SYLARAS approach to discovery-based immunophenotyping,</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_013"><span class="cls_013">we </span><span class="cls_005">follow-up on the most novel and statistically significant of our experimental findings which shows</span></div>
<div style="position:absolute;left:54.00px;top:676.80px" class="cls_005"><span class="cls_005">that CD45R/B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells (aka B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells) behave differently in the presence of GBM</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">4</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:3208px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-05.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">compared to conventional cytotoxic T cells. Through next-generation RNA-sequencing (RNA-seq), we</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">show that B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells are transcriptionally-distinct from their B220- counterparts, while</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">quantitative, spatially-resolved, single-cell analysis of 12-color immunofluorescence of the GBM TME</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">reveals that the cells infiltrate the tumor mass where they are distinguishable in both spatial distribution</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">and morphology from other tumor-infiltrating CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T lymphocytes. Complementary experiments in</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">humans support a role for B220+ CD8T cells in GBM by showing that similar cells circulate in normal</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">human blood and that a subset of primary and recurrent human GBM clinical biopsies. The ability of</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">mouse and human B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells to </span><span class="cls_013">attenuate immune response to autologous antigens</span><span class="cls_006"><sup>19,20</sup></span><span class="cls_013">, as</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_013"><span class="cls_013">demonstrated by the work of others, </span><span class="cls_005">suggests B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells may have the potential to antagonize</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">anti-tumor immune responses.</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">5</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:4010px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-06.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:70.80px" class="cls_012"><span class="cls_012">RESULTS</span></div>
<div style="position:absolute;left:54.00px;top:96.96px" class="cls_012"><span class="cls_012">Multi-organ immunophenotyping of GBM-bearing mice by 12-color flow cytometry</span></div>
<div style="position:absolute;left:90.00px;top:124.56px" class="cls_016"><span class="cls_016">We began our assessment of GBM’s impact on peripheral immune system architecture by first</span></div>
<div style="position:absolute;left:54.00px;top:152.16px" class="cls_005"><span class="cls_005">titrating across a 250-fold dilution range </span><span class="cls_016">11 antibodies targeting various cell surface antigens expressed</span></div>
<div style="position:absolute;left:54.00px;top:179.76px" class="cls_016"><span class="cls_016">by a diverse array of immune cells </span><span class="cls_005">to determine optimal immunolabeling concentrations</span></div>
<div style="position:absolute;left:54.00px;top:207.36px" class="cls_005"><span class="cls_005">(</span><span class="cls_012">Supplementary </span><span class="cls_017">Fig. 1a</span><span class="cls_016">)</span><span class="cls_005">. Optical spillover among cytometry detection channels was tested and</span></div>
<div style="position:absolute;left:54.00px;top:234.96px" class="cls_005"><span class="cls_005">adjusted for through an optical control study (</span><span class="cls_017">Supplementary Fig. 1b</span><span class="cls_016">). As a further prelude to our</span></div>
<div style="position:absolute;left:54.00px;top:262.56px" class="cls_016"><span class="cls_016">longitudinal investigation, we next performed Kaplan-Meier survival analysis on a cohort of 10</span></div>
<div style="position:absolute;left:54.00px;top:288.72px" class="cls_016"><span class="cls_016">C57BL/6J mice intracranially inoculated with 3x10</span><span class="cls_018"><sup>4</sup></span><span class="cls_016"> syngeneic GL261 GBM cells to identify time points</span></div>
<div style="position:absolute;left:54.00px;top:317.76px" class="cls_016"><span class="cls_016">in tumor progression associated with early, middle, and late-stage disease (</span><span class="cls_012">Supplementary </span><span class="cls_017">Fig. 1c</span><span class="cls_016">).</span></div>
<div style="position:absolute;left:54.00px;top:345.36px" class="cls_005"><span class="cls_005">After considering these results, and using a combination of our optimized and validated flow cytometry</span></div>
<div style="position:absolute;left:54.00px;top:372.96px" class="cls_005"><span class="cls_005">immunomarker panel and a methodical workflow </span><span class="cls_016">(</span><span class="cls_012">Supplementary </span><span class="cls_017">Fig. 1d-h</span><span class="cls_016">)</span><span class="cls_005">, we carried out 12-color</span></div>
<div style="position:absolute;left:54.00px;top:400.56px" class="cls_005"><span class="cls_005">flow cytometric analysis (11 immunomarkers plus fixable viability dye) on 5 lymphoid organs (</span><span class="cls_016">blood,</span></div>
<div style="position:absolute;left:54.00px;top:428.16px" class="cls_016"><span class="cls_016">bone marrow, deep/superficial cervical lymph nodes, spleen, and thymus)</span><span class="cls_005"> from three age-matched</span></div>
<div style="position:absolute;left:54.00px;top:455.76px" class="cls_005"><span class="cls_005">cohorts</span><span class="cls_016"> of 8 GBM-bearing and 8 mock-engrafted mice</span><span class="cls_005"> at days </span><span class="cls_016">7, 14, and 30 post-tumor engraftment</span></div>
<div style="position:absolute;left:54.00px;top:483.36px" class="cls_016"><span class="cls_016">(N=48) (</span><span class="cls_017">Fig. 1</span><span class="cls_016">)</span><span class="cls_005">.</span></div>
<div style="position:absolute;left:90.00px;top:510.96px" class="cls_005"><span class="cls_005">The overall</span><span class="cls_016"> the dataset consisted of 14 measurements (11 immunomarkers, cell viability, forward</span></div>
<div style="position:absolute;left:54.00px;top:537.12px" class="cls_016"><span class="cls_016">scatter (FSC) and side scatter (SSC)) made on each of </span><span class="cls_005">9.62x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> </span><span class="cls_016">immune cells spanning 240 tissue</span></div>
<div style="position:absolute;left:54.00px;top:564.72px" class="cls_016"><span class="cls_016">samples for a combined total of 1.25x10</span><span class="cls_018"><sup>9</sup></span><span class="cls_016"> data points</span><span class="cls_005">. </span><span class="cls_016">Average cell viability across samples was 98.6%</span></div>
<div style="position:absolute;left:54.00px;top:593.76px" class="cls_016"><span class="cls_016">(range 90% - 99.7%) and differed </span><span class="cls_005">between control and GBM-bearing animals only in the bone marrow</span></div>
<div style="position:absolute;left:54.00px;top:621.36px" class="cls_005"><span class="cls_005">at t=30-days where the percentage of dead cells was ~1.5-fold higher in control mice compared to those</span></div>
<div style="position:absolute;left:54.00px;top:648.96px" class="cls_005"><span class="cls_005">engrafted with GBM (p = 0.005, two-tailed Mann-Whitney U-test) (</span><span class="cls_017">Supplementary Fig. 1i</span><span class="cls_005">). Because</span></div>
<div style="position:absolute;left:54.00px;top:676.56px" class="cls_005"><span class="cls_005">the different lymphoid tissues varied </span><span class="cls_016">in terms of their cell number (e.g. spleen &gt;&gt; blood), </span><span class="cls_005">we elected to</span></div>
<div style="position:absolute;left:54.00px;top:702.72px" class="cls_005"><span class="cls_005">draw a 1x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> cell weighted random sample (WRS) to equalize the number of cells analyzed per sample,</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">6</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:4812px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-07.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:69.60px" class="cls_005"><span class="cls_005">which yielded an average of </span><span class="cls_016">4x10</span><span class="cls_018"><sup>4</sup></span><span class="cls_016"> cells per tissue sample (</span><span class="cls_017">Supplementary Fig. 1j</span><span class="cls_016">)</span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_012"><span class="cls_012">A computer-assisted approach to systematic cell subset identification</span></div>
<div style="position:absolute;left:90.00px;top:153.84px" class="cls_016"><span class="cls_016">Immune cells are commonly identified based cell surface antigen expression which is scored as</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_016"><span class="cls_016">either present or absent (e.g. CD</span><span class="cls_018"><sub>a+</sub></span><span class="cls_016">, CD</span><span class="cls_018"><sub>b-</sub></span><span class="cls_016">, CD</span><span class="cls_018"><sub>c+</sub></span><span class="cls_016">). To avoid the </span><span class="cls_005">complexity, non-commutativity, and labor</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">intensity inherent in serial gating </span><span class="cls_016">in methodical evaluation of immunophenotypes present within our</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">dataset</span><span class="cls_005">, devised a strategy in which a </span><span class="cls_016">distinction was made between true antibody signal and</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_016"><span class="cls_016">experimental noise on a per histogram basis using a computer-assisted procedure. In this unidimensional</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_016"><span class="cls_016">gating procedure, </span><span class="cls_005">signal intensity histograms were displayed as a scrolling HTML table with kernel</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">density estimates (KDEs) of unlabeled splenocytes super-imposed as fiducial guides aiding objective</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">gate placement between background and foreground signal intensities. After a </span><span class="cls_016">gate value for each of the</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_016"><span class="cls_016">dataset’s </span><span class="cls_005">2,640 histograms (48 mice x 11 immunomarker channels x 5 tissues) had been recorded (a</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">process which took less than 2 hours to complete) (</span><span class="cls_012">Fig. 2a-c</span><span class="cls_005"> and</span><span class="cls_017"> Supplementary Fig. 2</span><span class="cls_005">), the finalized</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">table of gate values was fed back to the SYLARAS. From there SYLARAS zero-centered the signal</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">intensity distributions associated with each histogram at the curated gate value by programmatically</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">substance the gate value from all of the signal intensity measurements of their corresponding</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">distributions. This caused signal intensities falling to the left of the gate, deemed as background</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">autofluorescence, to become negative valued, which in turn allowed each cell in the dataset to be</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">programmatically assigned an 11-dimensional Boolean vector fully specifying their immunophenotypes</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">following a binarization procedure according to the mathematical sign (+/-) of performed on signal</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">intensity values. For example, a cell associated assigned the Boolean vector [1, 0, 1, 1, 0, 0, 0, 1, 1 ,0, 0]</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">would have fallen to the right-hand side of the gate for the A channel, the left-hand side of the gate for</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">7</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:5614px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-08.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">the B channel, the right-hand side of the gate for the C, etc., and would have been associated with the</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">corresponding immunophenotype: CD</span><span class="cls_006"><sub>A+ </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>B- </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>C+ </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>D+ </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>E- </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>F- </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>G- </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>H+ </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>I+ </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>J- </sub></span><span class="cls_005">CD</span><span class="cls_006"><sub>K-</sub></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">Although we settled on a computer-assisted supervised approach to cell subset identification, we had</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">initially tested a </span><span class="cls_016">series of </span><span class="cls_005">fully-automated unsupervised methods for cell subset gating that included</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">various </span><span class="cls_016">Gaussian mixture models</span><span class="cls_006"><sup>21</sup></span><span class="cls_016"> and statistical thresholding methods</span><span class="cls_006"><sup>22</sup></span><span class="cls_005">, but found these and other</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">methods to be insufficient for the task, as each were prone to gate misplacement.</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_012"><span class="cls_012">Characterization of mouse lymphoid tissue cellular architecture in 30 immunophenotypes</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Upon the completion of our cell subset identification procedure, we found that 604 of the 2,048</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">(2</span><span class="cls_006"><sup>11</sup></span><span class="cls_005">) possible Boolean immunophenotypes specified by our 11-immunomarker flow cytometry panel</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">were represented by at least 1 cell. In limiting the scope of our downstream analysis, we elected to only</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">consider those </span><span class="cls_016">immunophenotypes whose frequency </span><span class="cls_005">was </span><span class="cls_025">&gt;</span><span class="cls_005">1% of cells in one or more of the study’s 240</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">tissue samples. This dramatically reduced the number of immunophenotypes to just 30. Seeking to</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">associate each of these high abundance immunophenotypes with an established immune cell type, we</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">next developing a “dictionary” of cell surface antigen expression profiles that characterize landmark</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">immune cell populations based on consensus literature. Fourteen (14) landmarks were defined: B, helper</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">T (CD4T), myeloid dendritic (DC), </span><span class="cls_016">cytotoxic T (CD8T), granulocyte (PMN), immature single-positive</span></div>
<div style="position:absolute;left:54.00px;top:538.80px" class="cls_016"><span class="cls_016">T (ISPT)</span><span class="cls_006"><sup>23</sup></span><span class="cls_016">, immature double-positive T (DPT), natural killer (NK), monocyte (Mono), macrophage</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_016"><span class="cls_016">(Mac), hematopoietic progenitor (Precursor), double-negative T (DNT), lymphoid tissue inducer (LTi)</span><span class="cls_006"><sup>24</sup></span><span class="cls_016">,</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_016"><span class="cls_016">and eosinophil (Eo) </span><span class="cls_005">(</span><span class="cls_012">Fig. 3a</span><span class="cls_005">,</span><span class="cls_012"> </span><span class="cls_005">inner wedges). To indicate the presence of specific variants of a given</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">landmark population, we elected to prefix their aliases using conventional +/- immunophenotyping</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">notation. For example, the subset of CD8T cells expressing Ly6C were assigned the alias Ly6C</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">8</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:6416px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-09.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:69.60px" class="cls_005"><span class="cls_005">and were considered CD8</span><span class="cls_006"><sup>+ </sup></span><span class="cls_005">memory T cells</span><span class="cls_006"><sup>25,26</sup></span><span class="cls_005"> (</span><span class="cls_012">Fig. 3a</span><span class="cls_005">,</span><span class="cls_012"> </span><span class="cls_005">outer wedges). Together, these 30</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">immunophenotypes accounted for 97% of the 10M cells comprising our WRS (</span><span class="cls_012">Fig. 3b</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">FSC and SSC light scatter measurements helped to benchmark the performance of our</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">dictionary-based cell classification system by showing that the values of both parameters were lowest</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">among cells classified as adaptive lymphocytes (e.g. CD4</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> and CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells); intermediate in innate</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">lymphocytes and non-polymorphonuclear myeloid cells (e.g. NK cells, monocytes/macrophages, and</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">dendritic cells); and highest among granulocytes (e.g. neutrophils/basophils) (</span><span class="cls_012">Fig. 3c</span><span class="cls_005">). The</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">exceptionally high SSC of eosinophils allowed us to delineate these cells from macrophages, which in</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">our study, shared the same antigen expression profile. Viewing our dataset from a tissue standpoint, the</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">expected proportions of the different immunophenotypes were observed across the 5 lymphoid organs.</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">For example, consistent with prior knowledge, the spleen, cervical lymph nodes, and blood were</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">primarily composed of B cells and CD4 and CD8T cells, while much of the bone marrow was</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">constituted by PMN cells,  B cells, and monocytes, and the thymus was near completely made up of</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">ISPT and DPT immature thymocytes (</span><span class="cls_012">Fig. 3d</span><span class="cls_005">). Using information entropy (S) as a diversity metric, we</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">next quantified heterogeneity in the distribution of the 30 immunophenotypes across the 5 lymphoid</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">tissues tissues (</span><span class="cls_012">Fig. 3e</span><span class="cls_005">). This also agreed with known immunology, as ISPT and DPT cells were</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">associated with the lowest entropy, due to their restriction to the thymus, while mature CD4 and CD8T</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">cells were associated with relatively high information entropy scores, given that these cells are more</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">evenly distributed throughout the immune system. Taken together, the results of our dictionary-based</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">cell classification method were congruent with the known cellular architecture of the mouse immune</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">system.</span></div>
<div style="position:absolute;left:90.00px;top:650.64px" class="cls_005"><span class="cls_005">Given the existence of sophisticated algorithms for unsupervised cell subset identification,</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">designed to meet the needs of highly multiplexed (30-40 channel) cytometry data and the extraordinary</span></div>
<div style="position:absolute;left:552.00px;top:727.44px" class="cls_005"><span class="cls_005">9</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:7218px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-10.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:69.60px" class="cls_005"><span class="cls_005">insights into immune cell classification and functional state that they can provide</span><span class="cls_006"><sup>27,28,29,30,31</sup></span><span class="cls_005">,  we sought</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">to incorporate into SYLARAS the option for a data-driven perspective of the cell classes contained in a</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">given dataset. In doing so, we elected to integrate a GUI-free implementation of the Phenograph</span><span class="cls_006"><sup>31</sup></span><span class="cls_005"> into</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">our computational data analysis pipeline. To compare the results of our dictionary-based cell</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">classification scheme with the unsupervised clustering results of Phenograph, it was necessary to further</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">reduce the size of our 10M cell dataset subsample by a factor of 5 from ~42K cells per tissue sample to</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">~8K, as nearest-neighbor clustering algorithms like Phenograph tend to be computationally intensive</span><span class="cls_006"><sup>32</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">We tested multiple integer values for the number of nearest neighbors (k, 10-30) using the standard</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">Euclidean distance metric to determine the effect of this tunable parameter on cluster stability before</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">settling on k=20 which returned a set of 30 clusters.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">To quantify the fit of the 30 Phenograph clusters in terms of our dictionary-based cell classifier,</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">we again turned to information entropy as a measure of diversity to evaluate heterogeneity in the</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">dictionary-based immunophenotypes that made up each cluster (</span><span class="cls_012">Fig. 3f </span><span class="cls_005">and</span><span class="cls_012"> Supplementary Fig. 3a</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">Twenty (20) of the Phenograph clusters were in good agreement with prior knowledge, as a single</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">immunophneotype accounted for </span><span class="cls_025">&gt;</span><span class="cls_005">50% of their cell compositions. For example, 97% of the cells in</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">cluster 1 mapped to the B cell immunophenotype. Conversely, the 10 remaining clusters were either</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">conflicted (i.e. required more than one immunophenotype to account for </span><span class="cls_025">&gt;</span><span class="cls_005">50% of their constituency) or</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">were predominantly composed of cells unspecified by our immunophenotype dictionary (i.e.</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">unclassified).</span></div>
<div style="position:absolute;left:90.00px;top:595.44px" class="cls_005"><span class="cls_005">We noted that B cells were the consensus immunophenotype of clusters 1, 7, 9, 15, 16, and 18.</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">While the brighter median CD45 and B220 signal intensity of cells constituting cluster 15 helped</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">explain why they formed a distinct cluster, separation of the other 5 B cell clusters appeared to be due to</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">differences in intrinsic background autofluorescence and/or off-target antibody binding</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">10</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:8020px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-11.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">(</span><span class="cls_012">Supplementary Fig. 3b</span><span class="cls_005">). For example, the difference in CD11c, CD4, CD8α, and Ly6G signal</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">intensity between the cells of cluster 1 and 7 was far greater than the difference in CD45 or B220 which</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">are considered B markers. Although these differences may have </span><span class="cls_019">bona fide</span><span class="cls_005"> biological underpinnings,</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">separation of clusters based on apparent differences in background antibody binding could not be</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">justified given the current experimental conditions. Nevertheless, the supervised and data-driven views</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">of the same dataset offered by SYLARAS open the potential to generate novel hypotheses that might not</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">have been formulated using one or the other method.</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_012"><span class="cls_012">SYLARAS reveals GBM-induced perturbation in the frequency of immune cell subsets hitherto</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_012"><span class="cls_012">undescribed in the disease</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">To identify statistically significant differences in tissue-specific immune cell frequency between</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">age-matched GBM-bearing and mock-engrafted mice, we next performed a set of 450 two-tailed</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">Student’s t-tests (30 cell classes x 5 tissues x 3 time points). After adjusting for multiple hypotheses by</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">false discovery rate (FDR) correction, we identified 28 instances in which mean cell frequency diverged</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">between tumor-burdened and control samples (q-value </span><span class="cls_025">&lt;</span><span class="cls_005"> 0.05) (</span><span class="cls_012">Fig. 4a</span><span class="cls_005">). Despite the majority of</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">perturbations occurring during late-stage disease (t=30-day), increases in the frequency of DNT cells in</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">the spleen and cervical lymph nodes of tumor-engrafted mice detected at the earliest time point (t=7-</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">days). The increased frequency of the cells over control persisted in the spleen at the t=14-day time point</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">and was consistent with the role played by DNT cells, including γ/δ T cells, in acute inflammatory</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_005"><span class="cls_005">immune response</span><span class="cls_006"><sup>33</sup></span><span class="cls_005">. The increased frequency of CD8T cells observed in the bone marrow of late-stage</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">GBM-bearing mice was also congruent with a recent report  demonstrating the sequestration of T cells</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">in the bone marrow of treatment-naïve humans and mice with GBM</span><span class="cls_006"><sup>34</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">11</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:8822px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-12.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">In terms of magnitude, the largest difference in immune cell frequency between control mice and</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">those with tumor burden was a mean increase of 18% in circulating PMN cells (</span><span class="cls_012">Fig. 4b</span><span class="cls_005">). This</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">observation was consistent with known facet of GBM immunobiology: namely, the accumulation of</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">immature and immunosuppressive myeloid-derived suppressor cells (MDSCs) of the same</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">immunophenotype (i.e. CD45</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD11b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> Ly6C</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> Ly6G</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">) in the blood of GBM-bearing mice and</span></div>
<div style="position:absolute;left:54.00px;top:207.60px" class="cls_005"><span class="cls_005">humans</span><span class="cls_006"><sup>35,36,37</sup></span><span class="cls_005">. Additional differences of sizable magnitude included a reduction of 18% in circulating B</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">cells and a 4% decrease in bone marrow-resident Ly6C</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> PMN cells, each occurring during the late-stage</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">time point.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">The fact that SYLARAS was capable of recapitulating known facets of GBM immunology</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">bolstered our confident in interpreting perturbation in several other cell populations that were less</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">expected. For example, we observed a wide-spread reduction in the abundance of hematopoietic</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_005"><span class="cls_005">progenitor cells (i.e. CD45</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> lin</span><span class="cls_006"><sup>-</sup></span><span class="cls_005">) throughout the bone marrow, cervical lymph nodes, and spleen of late-</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">stage tumor-burdened mice. Although we are not the first to report on GBM’s ability to influence bone</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">marrow hematopoiesis, the ability of SYLARAS to uncover the breadth of the tumor’s impact across</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">multiple lymphoid organs is remarkable. Additional unexpected changes were reductions in splenic</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">CD3ε</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> CD4</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> LTi cells thought to be important for lymphoid tissue organization, affinity maturation of</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_005"><span class="cls_005">germinal centers, and memory antibody responses</span><span class="cls_006"><sup>24</sup></span><span class="cls_005"> and increases in Ly6C</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> B cells in the cervical lymph</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">nodes. Since the cervical lymph nodes a known sink for brain-derived antigens (PMID: 25100741) and</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">Ly6C is a marker of antibody-producing plasma cells in mice</span><span class="cls_006"><sup>38</sup></span><span class="cls_005">, we postulated that the increase of these</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">cells in response to late-stage tumor burden might have been indicative of an active, yet ineffectual,</span></div>
<div style="position:absolute;left:54.00px;top:621.60px" class="cls_005"><span class="cls_005">humoral immune response to the tumor. Also unexpected was a drop in the frequency of F4/80</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> B cells</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">in the spleen of late-stage disease-bearing animals. Consistent with B cells bears macrophage-like</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">characteristics called biphenotypic B/macrophages, the function of cells of this immunophenotype</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">12</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:9624px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-13.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">remains enigmatic; however, the work of others suggests the requirements of macrophage colony-</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">stimulating factor (M-CSF) and low levels of </span><span class="cls_019">Pax5</span><span class="cls_005"> transcriptional activity for their development</span><span class="cls_006"><sup>39,40,41</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">Before leaving the rest of the tumor-induced changes identified in peripheral immune cell frequency up</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">to the reader’s interest, we wish to point out that monotonic decreases in both splenic NK cells and</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">circulating B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells changes were the two most statistically significant changes in the dataset</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">with increasingly significant q-values across time points. Although NK cells have been previously</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">implicated in GBM immunobiology</span><span class="cls_006"><sup>8,9,42</sup></span><span class="cls_005">, the tumor’s effect on B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells appeared to be a</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">novel. This previously documented subset of CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells expressing the CD45R/B220 isoform of the</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_005"><span class="cls_005">leukocyte common antigen, CD45</span><span class="cls_006"><sup>43</sup></span><span class="cls_005">. Combining the statistics on fold-change and absolute abundance</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">between test and control mice   with the results of our exploratory analysis on cell class antigen</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">expression, light scatter properties, and baseline tissue distribution, we programmatically generated a</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">concise compendium of data-rich graphical dashboards (one per immunophenotype) that allowed</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">complex, multi-organ immunophenotyping data on the tumor-induced changes in time and tissue-</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">specific cell frequency to be portrayed in a readily interpretable format (</span><span class="cls_012">Fig. 4c </span><span class="cls_005">and</span><span class="cls_012"> Supplementary</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_012"><span class="cls_012">Fig. 4</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_012"><span class="cls_012">Mouse-to-mouse variation in immunophenotype frequency holds network-level insight into</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_012"><span class="cls_012">GBM’s immunomodulatory control over the peripheral immune system</span></div>
<div style="position:absolute;left:90.00px;top:567.84px" class="cls_016"><span class="cls_016">Despite our use of in-bred and age-matched animals, substantial between-subject biological</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_016"><span class="cls_016">variation (BSBV)</span><span class="cls_006"><sup>44</sup></span><span class="cls_016"> was observed between replicate control and tumor-burdened mice. </span><span class="cls_005">The ability to</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">identify biological outliers (aka exceptional responders) in complex immunologic response data can</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">potentially increase our understanding of the individual and combinatorial features of immune response</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">allied with disease progression and therapeutically favorable outcomes. </span><span class="cls_016">Seeking to use mouse-to-mouse</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">13</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:10426px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-14.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_016"><span class="cls_016">variation </span><span class="cls_005">to our advantage to test for biological outliers</span><span class="cls_016"> in our dataset and gain an </span><span class="cls_005">overall perspective of</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">variation in our dataset</span><span class="cls_016">, we performed a principle component analysis (PCA). Together, </span><span class="cls_005">PC1 and PC2</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">accounted for &gt;60% of dataset variance; a good performance for PCA. The scores plot associated with</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">these first 2 PCs revealed five primary clusters (</span><span class="cls_012">Fig. 5a</span><span class="cls_005">). The bone marrow and thymus samples</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">(primary lymphoid tissues) were well-separated from the blood, spleen, cervical lymph nodes</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">(secondary lymphoid tissues), and the blood and spleen exhibited substantial overlap. Variation was also</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">evident within each cluster, being particularly apparent among blood and bone marrow samples.</span></div>
<div style="position:absolute;left:90.00px;top:264.24px" class="cls_005"><span class="cls_005">Late-stage GBM-burdened mouse 3 was flagged as a significant outlier; its bone marrow,</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">thymus, and blood samples were far removed from their respective primary clusters. Looking in further</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">detail at the immunophenotypes contributing to mouse 3’s outlier tendency, we identified higher</span></div>
<div style="position:absolute;left:54.00px;top:345.60px" class="cls_005"><span class="cls_005">fractions of mature myeloid cells, including DCs, macrophages, and Ly6C</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> PMN cells among its blood</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_005"><span class="cls_005">and bone marrow, and increased fractions of CD3ε</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> DPT cells (putative α/β-committed post-selection</span></div>
<div style="position:absolute;left:54.00px;top:400.80px" class="cls_005"><span class="cls_005">thymocytes</span><span class="cls_006"><sup>23</sup></span><span class="cls_005">) and mature CD4</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> and CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cell populations in its thymus that were suspicious of an</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">ongoing immune response (</span><span class="cls_012">Fig. 5b</span><span class="cls_005">). Inspection of the SYLARAS dashboards revealed that the fraction</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">of bone marrow Ly6C- PMN cells increased in control mice by 10-fold between the t=14 and t=30-day</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">time point, a phenomemnon which failed to occur in tumor-burdened mice with the exception of mouse</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">3, whose fraction of cells was on par with that of controls (</span><span class="cls_012">Fig. 5c</span><span class="cls_005">). The increase in bone-marrow-</span></div>
<div style="position:absolute;left:54.00px;top:538.80px" class="cls_005"><span class="cls_005">resident Ly6C</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> PMN cells over time in control mice suggested that the cells may be a useful surrogate</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">indicator of age-related changes in bone marrow composition; their failure to increase in GBM-bearing</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">mice further suggests they may act as a cell-based biomarker of disease progression.</span></div>
<div style="position:absolute;left:90.00px;top:623.04px" class="cls_016"><span class="cls_016">Although BSBV confounded our two-sample hypothesis tests by increasing dataset standard</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_016"><span class="cls_016">deviation and weakening p-values, we reasoned that it might contain additional information related to</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_016"><span class="cls_016">GBM’s effect on peripheral immune system architecture. Using Spearman’s rank-order correlation</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">14</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:11228px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-15.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_016"><span class="cls_016">(which requires replicate variation to be computed), we computed pairwise correlation between all time</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_016"><span class="cls_016">and tissue-specific immunophenotype pairs in control and GBM-burdened animals (e.g. splenic CD4T</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_016"><span class="cls_016">cells vs. thymic DPT cells in tumor-bearing mice at the t=30-day time point). In do so, we identified a</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">set of 7 significant cell-to-cell correlations in control mice that were completely abrogated by the</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">presence of GBM (q </span><span class="cls_026">&lt;</span><span class="cls_016"> </span><span class="cls_005">0.05) </span><span class="cls_016">(</span><span class="cls_017">Fig. 5e</span><span class="cls_016">). The fact that the mean cell frequency of populations involved in</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_016"><span class="cls_016">significant correlations failed to reach statistical significance between control and tumor-burdened mice</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">in our t-tests demonstrated that cell-to-cell covariation</span><span class="cls_005"> among biological replicates can arise</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">independently of differences in population average value (</span><span class="cls_012">Fig. 5d</span><span class="cls_005">). </span><span class="cls_016">many of the correlation pairs were</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_016"><span class="cls_016">readily explained by established immunology, such correlation #1, which can be explained based on the</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_016"><span class="cls_016">ability of monocyte/macrophages to induce B cell differentiation and survival</span><span class="cls_006"><sup>45</sup></span><span class="cls_016">, and correlation 5, which</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_016"><span class="cls_016">is in line with textbook interaction between MHC class II-expressing antigen-presenting cells and T</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_016"><span class="cls_016">helper cells. The potential biology underlying in some of the correlations was less apparent, however,</span></div>
<div style="position:absolute;left:54.00px;top:400.80px" class="cls_016"><span class="cls_016">we were interested to see correlation #2 involving B220</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> CD8T and B cells in the spleen, which served</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_016"><span class="cls_016">as an additional line of evidence suggesting GBM may influence these cells.</span></div>
<div style="position:absolute;left:90.00px;top:457.44px" class="cls_016"><span class="cls_016">Given that correlation coefficients had been computed for all tissue-specific immunophenotype</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_016"><span class="cls_016">pairs in control and GBM-burdened mice, we next sought to infer GBM’s impact on immunophenotypes</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_016"><span class="cls_016">involved in biologically-related processes by performing agglomerative hierarchical clustering on</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_016"><span class="cls_016">correlation profiles (vectors of correlation coefficients) of control and tumor-burdened mice (</span><span class="cls_012">Fig. 5f</span><span class="cls_016">).</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_016"><span class="cls_016">Silhouette analysis</span><span class="cls_006"><sup>46</sup></span><span class="cls_016"> was used to identifying the number of clusters (i.e. optimal linkage threshold) in</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_016"><span class="cls_016">each treatment group: 3 clusters were identified in control data, while 4 clusters where identified in data</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_016"><span class="cls_016">from GBM-burdened mice (</span><span class="cls_012">Fig. 5g, </span><span class="cls_016">(</span><span class="cls_012">Supplementary Fig. 5</span><span class="cls_016">). The most salient difference between mice</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_016"><span class="cls_016">of the two groups was the presence of cluster 3 in GBM-bearing mice. The cell populations constituting</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_016"><span class="cls_016">this cluster were strongly correlated, heavily enriched for DNT and PMN cells, and failed to co-cluster</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">15</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:12030px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-16.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_016"><span class="cls_016">in control mice. GBM cluster 3 was anti-correlated GBM clusters 1 and 2, which themselves were</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_016"><span class="cls_016">enriched for CD4T cells and various immune precursor cells. The observation of cluster 3 in mice with</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_016"><span class="cls_016">GBM burden is in accord with the results of a</span><span class="cls_005"> recent study in a spontaneous model of murine breast</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">cancer wherein the authors provide compelling evidence for the ability of IL-17A-producing γδ T cells</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">(consistent with the DNT immunophenotype in our study work given that &gt;70% of γδ T cells are</span></div>
<div style="position:absolute;left:54.00px;top:207.60px" class="cls_005"><span class="cls_005">negative for both CD4 and CD8</span><span class="cls_006"><sup>47</sup></span><span class="cls_005">) to induce an expansion of circulating neutrophils in tumor-bearing</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">mice that are immunophenotypically matched with the PMN cells of our study</span><span class="cls_006"><sup>48</sup></span><span class="cls_005">. We therefore postulate</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">that co-variation between DNT cells and PMN cells in tumor-bearing hosts may reflect a more general</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">phenomenon of cancer-induced immunomodulation discoverable through the purveyance of tools for</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">systems-level immunophenotyping like SYLARAS.</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_012"><span class="cls_012">B220</span><span class="cls_007"><sup>+</sup></span><span class="cls_012"> CD8T cells are a transcriptionally-distinct subset of CD8</span><span class="cls_007"><sup>+</sup></span><span class="cls_012"> T lymphocytes differentially</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_012"><span class="cls_012">responsive to GBM-induced biological cues</span></div>
<div style="position:absolute;left:90.00px;top:429.84px" class="cls_005"><span class="cls_005">Based on the statistical significance and novelty of our SYLARAS-based findings up to this</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">point related to B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells and the combined works of other groups who have attributed to these</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">cells an ability to attenuate immune response to self-antigens both in mice and humans</span><span class="cls_006"><sup>19,20</sup></span><span class="cls_005">, we were</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">compelled to continue investigating the nature of these cells and their relationship to GBM. Further</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">analysis of our SYLARAS data showed that unlike CD4T cells, CD8T cells, in general, exhibit a broad</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">and unimodal CD45R/B220 signal intensity distribution, with ~10% of cells in the blood, spleen, and</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_005"><span class="cls_005">cervical lymph nodes expressing the antigen at similar levels to B cells, which was defining of B220</span><span class="cls_006"><sup>+</sup></span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">CD8T cells (</span><span class="cls_012">Fig. 6a</span><span class="cls_005">). While the fraction of conventional CD8T cells in the thymus was ~4% across the</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">study’s 48 mice, the percentage of thymic B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells was in every mouse &lt; 0.02% (</span><span class="cls_012">Fig. 6b</span><span class="cls_005">),</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">suggesting that either B220 expression is induced after the cells have egressed from the thymus of that</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">16</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:12832px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-17.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">they have extra-thymic developmental origins. The fact that the monotonic decrease in the frequency of</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">circulating B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells in GBM-burdened mice was not mimicked by conventional CD8T cells</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">(</span><span class="cls_012">Fig. 6c</span><span class="cls_005">) was further evidence that B220+ CD8T cells are distinct from other CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells. And finally,</span></div>
<div style="position:absolute;left:54.00px;top:152.40px" class="cls_005"><span class="cls_005">regression analysis revealed a strong positive relationship between the frequency of B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> and B220</span><span class="cls_006"><sup>-</sup></span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">CD8T cells in the spleen across multiple experimental conditions (R</span><span class="cls_006"><sup>2 </sup></span><span class="cls_005">= 0.6-0.9) that was selectively</span></div>
<div style="position:absolute;left:54.00px;top:207.60px" class="cls_005"><span class="cls_005">abrogated in mice bearing late-stage disease </span><span class="cls_016">(</span><span class="cls_005">R</span><span class="cls_006"><sup>2</sup></span><span class="cls_005"> = 0.004</span><span class="cls_016">) (</span><span class="cls_012">Fig. 6d</span><span class="cls_016">), arguing for the differential-response</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_016"><span class="cls_016">of the CD8</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> T cell pool to the presence of GBM.</span></div>
<div style="position:absolute;left:90.00px;top:262.80px" class="cls_016"><span class="cls_016">To directly test the hypothesis that B220</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> CD8T cells are indeed a biologically-distinct subset of</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_016"><span class="cls_016">CD8</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> T cells, we next performed </span><span class="cls_005">RNA-seq on the coding transcriptome of B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> and B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005"> fractions</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">of CD3ε</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8α</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells derived from the spleen of tumor-naïve C57BL/6J mice in biological replicate</span></div>
<div style="position:absolute;left:54.00px;top:345.60px" class="cls_005"><span class="cls_005">(n=2 B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005">; n=3 B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005">). Because B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells only account for ~1% of total splenocytes, if was</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">necessary for us to combine the spleens of 5 mice prior to purifying these cells by fluorescence-activated</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">cell sorting (FACS) </span><span class="cls_016">(</span><span class="cls_017">Fig. 6e</span><span class="cls_016">)</span><span class="cls_005">. To expedite the sorting procedure, we B cells (~50-60% of splenocytes)</span></div>
<div style="position:absolute;left:54.00px;top:428.40px" class="cls_016"><span class="cls_016">were</span><span class="cls_005"> immunodepleted using anti-CD19 magnetic beads; the recovery of B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> CD8T cells from CD19</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">immunodepleted spleens was itself evidence that B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells are indeed T cells, not B cells.</span></div>
<div style="position:absolute;left:90.00px;top:483.60px" class="cls_005"><span class="cls_005">A PCA on the resulting transcript counts using the RNA-seq data analysis program, sleuth</span><span class="cls_006"><sup>49</sup></span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_005"><span class="cls_005">revealed clear separation between B220</span><span class="cls_006"><sup>hi </sup></span><span class="cls_005">and B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005"> CD8T cell samples along PC1 (65% of dataset</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">variation) (</span><span class="cls_017">Fig. 6f,g</span><span class="cls_005">). The contribution scores for PC1 showed that mRNA transcripts of mitochondrial</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">genes most significantly impacted PC1’s component scores, whereas a predicted gene (Gm26822)</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">dominated as the most significant contributor to PC2’s component scores (18% of dataset variation) and</span></div>
<div style="position:absolute;left:54.00px;top:621.60px" class="cls_005"><span class="cls_005">was, therefore, a likely of separation between the two B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> CD8T cell samples (</span><span class="cls_017">Fig. 6h</span><span class="cls_005">). We were</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">interested to also identify beta-2-microglobulin (</span><span class="cls_019">B2m</span><span class="cls_005">)—a component of the MHC class I complex—</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">among the top 5 transcripts influencing PC2’s component scores, which might reflect differences in the</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">17</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:13634px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-18.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">ratio of classical MHC-class 1a and non-classical MHC-class 1b molecules present due to differences in</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">sort purity between replicate samples .</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">To identify transcripts differentially expressed between the two populations, we subsequently</span></div>
<div style="position:absolute;left:54.00px;top:152.40px" class="cls_005"><span class="cls_005">performed a Wald test</span><span class="cls_006"><sup>50</sup></span><span class="cls_005">, which revealed 25 statistically significant (q-value </span><span class="cls_025">&lt;</span><span class="cls_005"> 0.05) (</span><span class="cls_017">Fig. 6i</span><span class="cls_005">). The largest</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">difference between B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> and</span><span class="cls_006"><sup> </sup></span><span class="cls_005">B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005"> CD8T cells was a 7-fold reduction in B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> cells in one of two</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">splice variants encoding subunit 5 of the ARP2/3 complex (</span><span class="cls_019">Arpc5</span><span class="cls_005">, ENSMUST00000097536)</span><span class="cls_019"> </span><span class="cls_005">involved</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">in actin polymerization</span><span class="cls_006"><sup>51</sup></span><span class="cls_005"> and maintenance of T cell receptors (TCRs) at the cell surface</span><span class="cls_006"><sup>52</sup></span><span class="cls_005"> (beta-value = -</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">7.0; q-value = 1.4x10</span><span class="cls_006"><sup>-7</sup></span><span class="cls_005">). Other differentially expressed transcripts with large fold-change and</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">exceedingly low q-values encoded for Cullin 1 (</span><span class="cls_019">Cul1</span><span class="cls_005">), a component of the E3 ubiquitin ligase complex,</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">and Diphthine methyl ester synthase (</span><span class="cls_019">Dph5</span><span class="cls_005">), a component of the diphthamide biosynthetic pathway</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">which leads to the post-translational modification of histidine on elongation factor 2 (</span><span class="cls_019">Eef2</span><span class="cls_005">), which was</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">interesting considering the fact that another translation elongation factor, 1α (</span><span class="cls_019">Eef1a</span><span class="cls_005">), provided the</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">second largest contribution score to PC1. Taken together, the high contribution scores to PC1 given by</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">transcripts associated with mitochondrial-encoded metabolic genes (</span><span class="cls_019">mt-Co1</span><span class="cls_005">, </span><span class="cls_019">mt-Atp6</span><span class="cls_005">, </span><span class="cls_019">mt-Cytb</span><span class="cls_005">, </span><span class="cls_019">mt-Nd4</span><span class="cls_005">),</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">together with differentially expressed genes related to translation elongation (</span><span class="cls_019">Eef1a</span><span class="cls_005">, </span><span class="cls_019">Dph5</span><span class="cls_005">, and </span><span class="cls_019">Lars2</span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_019"><span class="cls_019">Rps9</span><span class="cls_005">) proteasome-mediated protein degradation (</span><span class="cls_019">Cul1</span><span class="cls_005">, </span><span class="cls_019">Ube2d3</span><span class="cls_005">,</span><span class="cls_019"> Dtx2</span><span class="cls_005">), and glucose metabolism (</span><span class="cls_019">Gpi1</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_005"><span class="cls_005">suggested that the B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> cells may be characterized by an altered metabolic state relative to</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">conventional cytotoxic T cells.</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_012"><span class="cls_012">B220</span><span class="cls_007"><sup>+</sup></span><span class="cls_012"> CD8T cells are a morphologically and spatially distinguishable subset of GBM-infiltrating</span></div>
<div style="position:absolute;left:54.00px;top:621.60px" class="cls_012"><span class="cls_012">CD8</span><span class="cls_007"><sup>+</sup></span><span class="cls_012"> T lymphocytes</span></div>
<div style="position:absolute;left:90.00px;top:649.20px" class="cls_016"><span class="cls_016">To determine </span><span class="cls_005">whether B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T infiltrate the GBM TME, we performed </span><span class="cls_016">12-color cyclic</span></div>
<div style="position:absolute;left:54.00px;top:676.80px" class="cls_016"><span class="cls_016">immunofluorescence imaging (CyCIF)</span><span class="cls_006">53</span><span class="cls_018">,</span><span class="cls_006">54</span><span class="cls_016"> on tumor-bearing formalin-fixed paraffin-embedded (FFPE)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">18</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:14436px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-19.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_016"><span class="cls_016">brain tissue sections using antibodies targeting many of the same cell surface antigens assayed by</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_016"><span class="cls_016">SYLARAS. Immunomarkers were first validated on mouse lymph node tissue sections (</span><span class="cls_017">Supplementary</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_017"><span class="cls_017">Fig. 7a,b </span><span class="cls_016">and</span><span class="cls_005"> </span><span class="cls_012">Supplementary Table 1</span><span class="cls_016">) prior to use on 5µm-thick mouse brain tissue sections bearing</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_016"><span class="cls_016">GL261 GBM at t=36-days. Scanning fluorescence imaging was performed using a </span><span class="cls_005">0.6 NA </span><span class="cls_016">40X</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_016"><span class="cls_016">objective to yield a mosaic image consisting of 168 400x300µm tiles from which quantitative, spatially-</span></div>
<div style="position:absolute;left:54.00px;top:207.60px" class="cls_016"><span class="cls_016">resolved, single-cell data on ~9 x10</span><span class="cls_018"><sup>4</sup></span><span class="cls_016"> cells was obtained following image segmentation (</span><span class="cls_017">Fig. 7a</span><span class="cls_016">). Using</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">SYLARAS to analyze the data, we identified 7 abundant immunophenotypes in the tumor ipsilateral</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_016"><span class="cls_016">brain hemisphere among several other low abundance immunophenotypes (between 2 and 10 cells each)</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_016"><span class="cls_016">(</span><span class="cls_017">Fig. 7b</span><span class="cls_016"> and </span><span class="cls_017">Supplementary Fig. 7c-f</span><span class="cls_016">). No immune infiltrate was observed in the contralateral</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_016"><span class="cls_016">hemisphere (data not shown).</span></div>
<div style="position:absolute;left:90.00px;top:345.60px" class="cls_016"><span class="cls_016">Both B220</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> CD8T and conventional CD8T cells were present in the TME at approximately the</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_016"><span class="cls_016">same 1:10 ratio as that observed in the blood and spleen by SYLARAS. Compared to CD8T cells found</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_016"><span class="cls_016">in the TME, those expressing B220 tended to be more</span><span class="cls_005"> elongated and lacking in plasma membrane</span></div>
<div style="position:absolute;left:54.00px;top:428.40px" class="cls_005"><span class="cls_005">ruffling as observed in the CD8α channel (</span><span class="cls_017">Fig. 7c</span><span class="cls_005">). Tumor-infiltrating B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells also tended to</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">exhibit stronger CD8α staining, which agreed with our data on circulating B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells whose</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">median CD8α signal intensity was significantly higher than that of the conventional CD8T cell type (p &lt;</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_005"><span class="cls_005">1x10</span><span class="cls_006"><sup>-6</sup></span><span class="cls_005">) (</span><span class="cls_017">Fig. 7d</span><span class="cls_005">). The distribution of tumor-infiltrating B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> and B220</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> CD8T cells also differed. By</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">comparing the density of the 7 abundant immune cell populations with that of the tumor (defined as</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">vimentin</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> lin</span><span class="cls_006"><sup>- </sup></span><span class="cls_005">cells), we found that conventional CD8T cells tended to localize to the tumor’s margins,</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_005"><span class="cls_005">while the majority of the B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells were found at the center of the tumor mass </span><span class="cls_016">(</span><span class="cls_017">Fig. 7e</span><span class="cls_016">)</span><span class="cls_005">. In</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">quantifying the degree of overlap between these two immunophenotypes, we computed the spatial</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">correlation of among all pairs of the 7 abundant immune cell types and the tumor itself </span><span class="cls_016">(</span><span class="cls_017">Fig. 7f</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:676.80px" class="cls_005"><span class="cls_005">Correlation between B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> and B220</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> CD8T cells was far lower than what would have been expected if</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">19</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:15238px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-20.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:69.60px" class="cls_005"><span class="cls_005">the cells were drawn from the same underlying distribution (R</span><span class="cls_006"><sup>2</sup></span><span class="cls_005"> = 0.26). Based on the results of our</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">multiplex immunofluorescence study, we conclude that B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells infiltrate the GBM TME</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">where they look and behave differently from other tumor-infiltrating CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T lymphocytes.</span></div>
<div style="position:absolute;left:90.00px;top:152.40px" class="cls_016"><span class="cls_016">Given that B220</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> CD8T cells were identified in the blood of tumor-naïve mice, we next sought</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_016"><span class="cls_016">to determine whether an analogous cell type could be found in normal human blood. In doing so, we</span></div>
<div style="position:absolute;left:54.00px;top:207.60px" class="cls_016"><span class="cls_016">surveyed ~1.6x10</span><span class="cls_018"><sup>5</sup></span><span class="cls_016"> peripheral blood mononuclear cells (PBMCs) obtained from a healthy human donor</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">and fixed onto a glass microscope slide for immunolabeling with human-specific antibodies targeting</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">CD3δ, CD8α and CD45R and scanning at 20x with a fluorescence microscope. Although anti-mouse</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">CD45R/B220 antibodies (clone: RA3-6B2) are documented to cross react with human immune cells, we</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">found that that these antibodies gave high background and lacked specific staining in the context of</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">FFPE samples. Thus, as an alternative to the RA3-6B2 clone, we used anti-human CD45R (clone:</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">MB1). These antibodies also recognize an epitope on exon-A of a high molecular weight isoform of</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">CD45 expressed by human B cells and a subset of peripheral T cytotoxic/suppressor cells are validated</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">on FFPE human tissue. Six (</span><span class="cls_016">6) high-abundance immunophenotypes were observed (</span><span class="cls_017">Fig. 8a</span><span class="cls_005">)</span><span class="cls_016">. </span><span class="cls_005">The bulk</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">(~60%) of cells were negative for the 3 immunomarkers, which were followed in cell number by CD3δ</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">single-positive cells, CD45R single-positive cells, CD3δ/CD8α double-positive T cells, and</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_005"><span class="cls_005">CD3δ/CD8α/CD45R triple-positive T cells (immunophenotypically analogous to mouse B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">cells) </span><span class="cls_016">(</span><span class="cls_017">Fig. 8b</span><span class="cls_016">)</span><span class="cls_005">. Least abundant cell type was double-positive for CD3δ/CD45R which were thought to</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">be enriched for CD45R</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> helper T cells</span><span class="cls_006"><sup>55</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:90.00px;top:594.00px" class="cls_016"><span class="cls_016">To ultimately determine whether a B220</span><span class="cls_018"><sup>+</sup></span><span class="cls_016"> CD8T cell analogue could be identified in human</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_016"><span class="cls_016">GBM</span><span class="cls_005">, we constructed and immunolabeled a tissue microarray (TMA) consisting of 312 FFPE 0.6mm in</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">diameter core biopsies from 173 different brain tumor patients. The TMA contained 140 primary or</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">recurrent grade-IV GBMs, 14 grade-II oligodendrogliomas, 14 grade-III anaplastic oligodendrogliomas,</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">20</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:16040px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-21.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">2 grade-III anaplastic astrocytomas, and single cases of grade-III anaplastic astroblastoma, grade III</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">anaplastic ganglioglioma, and FGFR3-TACC3 fusion glioma. Duplicate specimens from were present</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">for the majority of patients, however, small differences in the level at which each core was paraffin</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">embedded caused the failure of 16 cores to be transferred to the slide. Thus, a small subset of the</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">patient’s tumors we represented by just 1 core. The TMA was immunolabeled with the same antibodies</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">used to probe human the blood and scanned at 20x using a fluorescence microscope to create a single</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">mosaic image consisting of 875-tiles (</span><span class="cls_017">Fig. 8c</span><span class="cls_005">). Nineteen (19) of the 173 patients contained at least 1</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">CD3δ/CD8α/CD45R triple-positive cell; in one case, duplicate cores were both positive for these cells</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">(patient #127)  (</span><span class="cls_017">Fig. 8d-f</span><span class="cls_016">)</span><span class="cls_005">. Six (6) of the 19 tumors containing triple-positive cells were primary GBM,</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">10 were recurrent GBM, and the remaining 3 were a mixture of primary or recurrent anaplastic</span></div>
<div style="position:absolute;left:54.00px;top:345.60px" class="cls_005"><span class="cls_005">oligodendroglioma and astrocytoma, indicating that cells analogous to mouse B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells can be</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">found in a variety of gliomas. Low numbers of triple-positive cells were attributed to small sample area,</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">inhomogeneity in immune cell distribution across tissue, and the fact that the cells represent but a small</span></div>
<div style="position:absolute;left:54.00px;top:428.40px" class="cls_005"><span class="cls_005">fraction of total CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells (~10%). We conclude that B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells circulate in human blood</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">and penetrate the TME of a subset of human GBM. Based on the work on others attributing</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">immunosuppressive function to B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T both in mice and humans, we predict the cells to play an</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">immunoregulatory role in the disease. In future studies, we aim to determine the diagnostic and/or</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">immunotherapeutic potential of these cells in GBM and other cancers.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">21</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:16842px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-22.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">DISCUSSION</span></div>
<div style="position:absolute;left:90.00px;top:98.64px" class="cls_005"><span class="cls_005">Given that immune outcomes depend on the coordinated activity of a network of specialized</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">immune cells residing across multiple biological compartments</span><span class="cls_006"><sup>56</sup></span><span class="cls_005">, measures of individual aspects of the</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">immune system are often poorly correlated with disease progression and therapeutic response. This fact</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">underscores the value of studying model organisms that preserve in high-fidelity the biological context</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">of complex immune repertoires through methods that integrate multi-tissue information at several</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">biological scales. In achieving a more integrated comprehension of GBM’s immunomodulatory</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">influence, we have established an approach to multi-organ immunophenotyping of preclinical model</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">systems called SYLARAS. The ultimately transforms quantitative single-cell data into a data-rich</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">graphical compendium of the time and tissue-dependent changes occurring in lymphoid organ cellular</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">constituency in response to an arbitrary immune stimulus. The systematic and concise format of</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">presenting single-cell data through the SYLARAS dashboard facilities data interpretation and allows</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">biological outliers to be easily identified while overcoming the inherent difficulty of representing large</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">datasets using space-inefficient histograms and scatter plots. Supervised cell subset identification in</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">SYLARAS is also streamlined, as an expedited unidimensional gating procedure leads to the rapid, yet</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">systematic, identification of immune cell subsets preset within a dataset in a way that avoids the labor-</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">intensity and non-commutativity inherent to serial gating. Integration of a GUI-free implementation of</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">the nearest-neighbor clustering also allows for a unsupervised view of the cell subsets present in the</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">same dataset.</span></div>
<div style="position:absolute;left:90.00px;top:595.44px" class="cls_005"><span class="cls_005">Our work with SYLARAS in the context of a syngeneic mouse model of GBM has recapitulated</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">known facets of GBM immunology while uncovering several others hitherto undescribed in the disease.</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">The fact that CD45R/B220 expression by CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells does not create a bimodal distribution has likely</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">contributed to the failure to identify a role for these cells in GBM and perhaps other cancers. The varied</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">22</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:17644px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-23.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">works of others demonstrating the ability of immunophenotypically similar cells to attenuate immune</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">response to autologous antigens suggests that the cells identified by our study may be capable of</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">mitigating immune responses to potential tumor-derived neoantigens. Although true identity of B220</span><span class="cls_006"><sup>+</sup></span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">CD8T cells is unclear at present, at least two populations of T cells expressing CD45, CD3ε, CD8α,</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">andCD45R/B220 have been previously characterized. One of which are known as CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T suppressor</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">cells, which are capable of indirectly constraining immune response to self-antigens through interaction</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">with specific subsets of CD4</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells</span><span class="cls_006"><sup>57,58,59</sup></span><span class="cls_005">. The other of which are intraepithelial lymphocytes (IELs)</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">most commonly found in the gut mucosal lining</span><span class="cls_006"><sup>60</sup></span><span class="cls_005">. Both cell types have also been documented to</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_005"><span class="cls_005">produce IL-10 and TGF-β—potent anti-inflammatory cytokines</span><span class="cls_006"><sup>59,60,61</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:90.00px;top:319.44px" class="cls_005"><span class="cls_005">The </span><span class="cls_019">Ptprc</span><span class="cls_005"> gene encodes an evolutionarily conserved phosphotyrosine phosphatase referred to as</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">cluster of differentiation 45 (CD45). As an integral membrane protein expressed by all cells of the</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">hematopoietic cell lineage, antibodies indiscriminately recognizing all isoforms of this alternatively</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">spliced glycoprotein (referred to as anti-CD45) are useful probes in identifying immune cells as a class.</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">However, antibodies whose binding to the protein is dependent on the presence of specific exons and</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">carbohydrate moieties are referred to as anti-CD45R indicating their recognition of a “restricted” subset</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">of splice variants. Antibodies such as this have been useful in delineating specific immune subsets</span><span class="cls_006"><sup>62,63</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">For example, the anti-CD45R/B220 (clone: RA3-6B2) antibodies used in this study were initially</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">included in the flow cytometry panel to label B cells despite their known  immunoreactivity with subsets</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">of NK cells</span><span class="cls_006"><sup>64</sup></span><span class="cls_005"> and T cells</span><span class="cls_006"><sup>65</sup></span><span class="cls_005">. While the developmental regulation and biological significance of</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_005"><span class="cls_005">CD45R/B220 by CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells is incompletely understood, its expression is functional; its ligation with</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">antibodies recognizing exon A of CD45R increase the proliferative response of T cells expressing the</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">isoform to phytohemagglutinin (a mitogenic lectin)</span><span class="cls_006"><sup>43,66</sup></span><span class="cls_005">, which leaves open the possibility that</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">CD45R/B220 may not just be a biomarker of these cells, but also play a direct role in mediating their</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">23</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:18446px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-24.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">behavior in GBM. Whether the anti-CD45R/B220 (clone: RA3-6B2) and anti-CD45R (clone: MB1)</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">used in this work in fact recognize an identical population of CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T cells in mice and man is uncertain.</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">However, both antibodies are documented to react with restricted epitopes on exon A at the extracellular</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">domain of a high molecular weight isoform of the CD45 glycoprotein expressed by human B cells and a</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">subset of peripheral T cytotoxic/suppressor cells.</span></div>
<div style="position:absolute;left:90.00px;top:209.04px" class="cls_005"><span class="cls_005">In conclusion, the potential of SYLARAS to provide ample content for inductive scientific</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">inference of the cell and molecular mechanisms governing immunologically-relevant diseases as</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">demonstrated through this work showcase its utility as a tool for immunologic discovery. Due to its</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">modularity of design and its neutrality to the method of data acquisition, SYLARAS is compatible with</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">various platforms for single-cell data acquisition, including flow cytometry (as demonstrated by the</span></div>
<div style="position:absolute;left:54.00px;top:345.60px" class="cls_005"><span class="cls_005">current work), CyTOF, and a series of rapidly emerging image-based technologies such as CyCIF</span><span class="cls_006">53</span><span class="cls_018">,</span><span class="cls_006">54</span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_005"><span class="cls_005">CODEX</span><span class="cls_006"><sup>67</sup></span><span class="cls_005">, and 4i</span><span class="cls_006"><sup>68</sup></span><span class="cls_005"> which offer the added benefit of providing insight into the spatial relationships</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">among cells. Swapping the current set of 11 immunomarkers with different ones conjugated to the same</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">(or similar) fluorophores will allow the investigator to follow-up on leads uncovered through initial</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">immunophenotyping experiments with SYLARAS by focusing on cell surface and intracellular markers</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">of specific immune lineages ensuring experimental continuity.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">24</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:19248px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-25.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:70.80px" class="cls_012"><span class="cls_012">METHODS</span></div>
<div style="position:absolute;left:54.00px;top:96.96px" class="cls_005"><span class="cls_005">Methods, including statements of data availability and any associated accession codes and references,</span></div>
<div style="position:absolute;left:54.00px;top:124.56px" class="cls_005"><span class="cls_005">are available in the online version of the paper.</span></div>
<div style="position:absolute;left:54.00px;top:179.76px" class="cls_005"><span class="cls_005">Note: any Supplementary Information and Source Data files are available in the online version of the</span></div>
<div style="position:absolute;left:54.00px;top:207.36px" class="cls_005"><span class="cls_005">paper.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">25</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:20050px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-26.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">ACKNOWLEDGMENTS</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">This work was supported by an American Cancer Society Postdoctoral Fellowship (PF-16-197-01-LIB)</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">to G.J.B, and by NIH/NCI grants P50-GM107618 and U54-CA225088 to P.K.S. We gratefully</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">acknowledge Gabriel Berriz, Jeremy Muhlich, and Artem Sokolov for their support with computation.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">26</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:20852px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-27.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">AUTHOR CONTRIBUTIONS</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">G.J.B. conceived of the project; G.J.B and P.K.S developed the research; strategy G.J.B., S.H.D., and</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">J.K.M. performed the experiments and analysis; G.J.B. and S.K.P. designed and performed the</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">computational analysis; G.J.B. and P.K.S. wrote the paper.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">27</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:21654px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-28.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">COMPETING FINANCIAL INTERESTS</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">The authors declare no competing financial interests.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">28</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:22456px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-29.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">REFERENCES</span></div>
<div style="position:absolute;left:57.00px;top:98.64px" class="cls_005"><span class="cls_005">1.Maxwell, M., Galanopoulos, T., Neville-Golden, J. &amp; Antoniades, H. N. Effect of the expression of</span></div>
<div style="position:absolute;left:67.20px;top:126.24px" class="cls_005"><span class="cls_005">transforming growth factor-beta 2 in primary human glioblastomas on immunosuppression and loss</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_005"><span class="cls_005">of immune surveillance. </span><span class="cls_019">J. Neurosurg.</span><span class="cls_005"> </span><span class="cls_012">76</span><span class="cls_005">, 799-804 (1992).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">2. Huettner, C., Czub, S., Kerkau, S., Roggendorf, W. &amp; Tonn, J. C. Interleukin 10 is expressed in</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">human gliomas in vivo and increases glioma cell proliferation and motility in vitro. </span><span class="cls_019">Anticancer Res.</span></div>
<div style="position:absolute;left:67.20px;top:236.64px" class="cls_012"><span class="cls_012">17</span><span class="cls_005">, 3217-3224 (1997).</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">3. Akasaki, Y. </span><span class="cls_019">et al.</span><span class="cls_005"> Induction of a CD4+ T regulatory type 1 response by cyclooxygenase-2-</span></div>
<div style="position:absolute;left:67.20px;top:291.84px" class="cls_005"><span class="cls_005">overexpressing glioma. </span><span class="cls_019">J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">173</span><span class="cls_005">, 4352-4359 (2004).</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">4. Crane, C. A. </span><span class="cls_019">et al.</span><span class="cls_005"> Immune evasion mediated by tumor-derived lactate dehydrogenase induction of</span></div>
<div style="position:absolute;left:67.20px;top:347.04px" class="cls_005"><span class="cls_005">NKG2D ligands on myeloid cells in glioblastoma patients. </span><span class="cls_019">Proc. Natl. Acad. Sci. U.S.A.</span><span class="cls_005"> </span><span class="cls_012">111</span><span class="cls_005">, 12823-</span></div>
<div style="position:absolute;left:67.20px;top:374.64px" class="cls_005"><span class="cls_005">12828 (2014).</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">5. Wainwright, D. A. </span><span class="cls_019">et al.</span><span class="cls_005"> IDO expression in brain tumors increases the recruitment of regulatory T</span></div>
<div style="position:absolute;left:67.20px;top:429.84px" class="cls_005"><span class="cls_005">cells and negatively impacts survival. </span><span class="cls_019">Clin. Cancer Res.</span><span class="cls_005"> </span><span class="cls_012">18</span><span class="cls_005">, 6110-6121 (2012).</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">6. Zhou, W. </span><span class="cls_019">et al.</span><span class="cls_005"> Periostin secreted by glioblastoma stem cells recruits M2 tumour-associated</span></div>
<div style="position:absolute;left:67.20px;top:485.04px" class="cls_005"><span class="cls_005">macrophages and promotes malignant growth. </span><span class="cls_019">Nat. Cell Biol.</span><span class="cls_005"> </span><span class="cls_012">17</span><span class="cls_005">, 170-182 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">7. Yan, J. </span><span class="cls_019">et al.</span><span class="cls_005"> FGL2 as a Multimodality Regulator of Tumor-Mediated Immune Suppression and</span></div>
<div style="position:absolute;left:67.20px;top:540.24px" class="cls_005"><span class="cls_005">Therapeutic Target in Gliomas. </span><span class="cls_019">J. Natl. Cancer Inst.</span><span class="cls_005"> </span><span class="cls_012">107</span><span class="cls_005">, (2015).</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">8. Baker, G. J. </span><span class="cls_019">et al.</span><span class="cls_005"> Natural killer cells eradicate galectin-1-deficient glioma in the absence of adaptive</span></div>
<div style="position:absolute;left:67.20px;top:595.44px" class="cls_005"><span class="cls_005">immunity. </span><span class="cls_019">Cancer Res.</span><span class="cls_005"> </span><span class="cls_012">74</span><span class="cls_005">, 5079-5090 (2014).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">9. Baker, G. J., Chockley, P., Zamler, D., Castro, M. G. &amp; Lowenstein, P. R. Natural killer cells require</span></div>
<div style="position:absolute;left:67.20px;top:650.64px" class="cls_005"><span class="cls_005">monocytic Gr-1(+)/CD11b(+) myeloid cells to eradicate orthotopically engrafted glioma cells.</span></div>
<div style="position:absolute;left:67.20px;top:678.24px" class="cls_019"><span class="cls_019">Oncoimmunology</span><span class="cls_005"> </span><span class="cls_012">5</span><span class="cls_005">, e1163461 (2016).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">29</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:23258px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-30.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">10.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Wainwright, D. A. </span><span class="cls_019">et al.</span><span class="cls_005"> Durable therapeutic efficacy utilizing combinatorial blockade against</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">IDO, CTLA-4, and PD-L1 in mice with brain tumors. </span><span class="cls_019">Clin. Cancer Res.</span><span class="cls_005"> </span><span class="cls_012">20</span><span class="cls_005">, 5290-5301 (2014).</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">11.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Bloch, O. </span><span class="cls_019">et al.</span><span class="cls_005"> Gliomas promote immunosuppression through induction of B7-H1 expression in</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_005"><span class="cls_005">tumor-associated macrophages. </span><span class="cls_019">Clin. Cancer Res.</span><span class="cls_005"> </span><span class="cls_012">19</span><span class="cls_005">, 3165-3175 (2013).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">12.</span></div>
<div style="position:absolute;left:90.00px;top:181.44px" class="cls_005"><span class="cls_005">Saas, P. </span><span class="cls_019">et al.</span><span class="cls_005"> Fas ligand expression by astrocytoma in vivo: maintaining immune privilege in the</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">brain? </span><span class="cls_019">J. Clin. Invest.</span><span class="cls_005"> </span><span class="cls_012">99</span><span class="cls_005">, 1173-1178 (1997).</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">13.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">Spitzer, M. H. </span><span class="cls_019">et al.</span><span class="cls_005"> Systemic Immunity Is Required for Effective Cancer Immunotherapy. </span><span class="cls_019">Cell</span></div>
<div style="position:absolute;left:67.20px;top:264.24px" class="cls_012"><span class="cls_012">168</span><span class="cls_005">, 487-502.e15 (2017).</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">14.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Chen, D. S. &amp; Mellman, I. Elements of cancer immunity and the cancer-immune set point.</span></div>
<div style="position:absolute;left:67.20px;top:319.44px" class="cls_019"><span class="cls_019">Nature</span><span class="cls_005"> </span><span class="cls_012">541</span><span class="cls_005">, 321-330 (2017).</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">15.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">Chen, D. S. &amp; Mellman, I. Oncology meets immunology: the cancer-immunity cycle. </span><span class="cls_019">Immunity</span></div>
<div style="position:absolute;left:67.20px;top:374.64px" class="cls_012"><span class="cls_012">39</span><span class="cls_005">, 1-10 (2013).</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">16.</span></div>
<div style="position:absolute;left:90.00px;top:402.24px" class="cls_005"><span class="cls_005">Saeys, Y., Van Gassen, S. &amp; Lambrecht, B. N. Computational flow cytometry: helping to make</span></div>
<div style="position:absolute;left:67.20px;top:429.84px" class="cls_005"><span class="cls_005">sense of high-dimensional immunology data. </span><span class="cls_019">Nat. Rev. Immunol.</span><span class="cls_005"> </span><span class="cls_012">16</span><span class="cls_005">, 449-462 (2016).</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">17.</span></div>
<div style="position:absolute;left:90.00px;top:457.44px" class="cls_005"><span class="cls_005">FlowJo v10 Documentation. (2018).</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">18.</span></div>
<div style="position:absolute;left:90.00px;top:485.04px" class="cls_005"><span class="cls_005">Spitzer, M. H. &amp; Nolan, G. P. Mass Cytometry: Single Cells, Many Features. </span><span class="cls_019">Cell</span><span class="cls_005"> </span><span class="cls_012">165</span><span class="cls_005">, 780-791</span></div>
<div style="position:absolute;left:67.20px;top:512.64px" class="cls_005"><span class="cls_005">(2016).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">19.</span></div>
<div style="position:absolute;left:90.00px;top:540.24px" class="cls_005"><span class="cls_005">Takeuchi, T. </span><span class="cls_019">et al.</span><span class="cls_005"> Functional characterization of the CD45R (2H4) molecule on CD8 (T8) cells</span></div>
<div style="position:absolute;left:67.20px;top:567.84px" class="cls_005"><span class="cls_005">in the autologous mixed lymphocyte reaction system. </span><span class="cls_019">Eur. J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">19</span><span class="cls_005">, 747-755 (1989).</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">20.</span></div>
<div style="position:absolute;left:90.00px;top:595.44px" class="cls_005"><span class="cls_005">James, E. A. &amp; Kwok, W. W. CD8+ suppressor-mediated regulation of human CD4+ T cell</span></div>
<div style="position:absolute;left:67.20px;top:623.04px" class="cls_005"><span class="cls_005">responses to glutamic acid decarboxylase 65. </span><span class="cls_019">Eur. J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">37</span><span class="cls_005">, 78-86 (2007).</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">21.</span></div>
<div style="position:absolute;left:90.00px;top:650.64px" class="cls_005"><span class="cls_005">Melnykov, V &amp; Maitra, R. Finite mixture models and model-based clustering. </span><span class="cls_019">Stat. Surv.</span><span class="cls_005"> </span><span class="cls_012">4</span><span class="cls_005">, 80-</span></div>
<div style="position:absolute;left:67.20px;top:678.24px" class="cls_005"><span class="cls_005">116 (2010).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">30</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:24060px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-31.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">22.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Otsu, N. A Threshold Selection Method from Gray-Level Histograms. </span><span class="cls_019">IEEE SMCS</span><span class="cls_005"> </span><span class="cls_012">9</span><span class="cls_005">, 62-66</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">(1979).</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">23.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Weerkamp, F., Pike-Overzet, K. &amp; Staal, F. J. T. T-sing progenitors to commit. </span><span class="cls_019">Trends Immunol.</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_012"><span class="cls_012">27</span><span class="cls_005">, 125-131 (2006).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">24.</span></div>
<div style="position:absolute;left:90.00px;top:181.44px" class="cls_005"><span class="cls_005">Lane, P. J. L., Gaspal, F. M. C. &amp; Kim, M.-Y. Two sides of a cellular coin: CD4(+)CD3- cells</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">regulate memory responses and lymph-node organization. </span><span class="cls_019">Nat. Rev. Immunol.</span><span class="cls_005"> </span><span class="cls_012">5</span><span class="cls_005">, 655-660 (2005).</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">25.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">Walunas, T. L., Bruce, D. S., Dustin, L., Loh, D. Y. &amp; Bluestone, J. A. Ly-6C is a marker of</span></div>
<div style="position:absolute;left:67.20px;top:264.24px" class="cls_005"><span class="cls_005">memory CD8+ T cells. </span><span class="cls_019">J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">155</span><span class="cls_005">, 1873-1883 (1995).</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">26.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Hänninen, A., Maksimow, M., Alam, C., Morgan, D. J. &amp; Jalkanen, S. Ly6C supports</span></div>
<div style="position:absolute;left:67.20px;top:319.44px" class="cls_005"><span class="cls_005">preferential homing of central memory CD8+ T cells into lymph nodes. </span><span class="cls_019">Eur. J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">41</span><span class="cls_005">, 634-644</span></div>
<div style="position:absolute;left:67.20px;top:347.04px" class="cls_005"><span class="cls_005">(2011).</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">27.</span></div>
<div style="position:absolute;left:90.00px;top:374.64px" class="cls_005"><span class="cls_005">Spitzer, M. H. </span><span class="cls_019">et al.</span><span class="cls_005"> IMMUNOLOGY. An interactive reference framework for modeling a</span></div>
<div style="position:absolute;left:67.20px;top:402.24px" class="cls_005"><span class="cls_005">dynamic immune system. </span><span class="cls_019">Science</span><span class="cls_005"> </span><span class="cls_012">349</span><span class="cls_005">, 1259425 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">28.</span></div>
<div style="position:absolute;left:90.00px;top:429.84px" class="cls_005"><span class="cls_005">Qiu, P. </span><span class="cls_019">et al.</span><span class="cls_005"> Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE.</span></div>
<div style="position:absolute;left:67.20px;top:457.44px" class="cls_019"><span class="cls_019">Nat. Biotechnol.</span><span class="cls_005"> </span><span class="cls_012">29</span><span class="cls_005">, 886-891 (2011).</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">29.</span></div>
<div style="position:absolute;left:90.00px;top:485.04px" class="cls_005"><span class="cls_005">Van Gassen, S. </span><span class="cls_019">et al.</span><span class="cls_005"> FlowSOM: Using self-organizing maps for visualization and interpretation</span></div>
<div style="position:absolute;left:67.20px;top:512.64px" class="cls_005"><span class="cls_005">of cytometry data. </span><span class="cls_019">Cytometry A</span><span class="cls_005"> </span><span class="cls_012">87</span><span class="cls_005">, 636-645 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">30.</span></div>
<div style="position:absolute;left:90.00px;top:540.24px" class="cls_005"><span class="cls_005">Amir, E. D. </span><span class="cls_019">et al.</span><span class="cls_005"> viSNE enables visualization of high dimensional single-cell data and reveals</span></div>
<div style="position:absolute;left:67.20px;top:567.84px" class="cls_005"><span class="cls_005">phenotypic heterogeneity of leukemia. </span><span class="cls_019">Nat. Biotechnol.</span><span class="cls_005"> </span><span class="cls_012">31</span><span class="cls_005">, 545-552 (2013).</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">31.</span></div>
<div style="position:absolute;left:90.00px;top:595.44px" class="cls_005"><span class="cls_005">Levine, J. H. </span><span class="cls_019">et al.</span><span class="cls_005"> Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells</span></div>
<div style="position:absolute;left:67.20px;top:623.04px" class="cls_005"><span class="cls_005">that Correlate with Prognosis. </span><span class="cls_019">Cell</span><span class="cls_005"> </span><span class="cls_012">162</span><span class="cls_005">, 184-197 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">32.</span></div>
<div style="position:absolute;left:90.00px;top:650.64px" class="cls_005"><span class="cls_005">Weber, L. M. &amp; Robinson, M. D. Comparison of clustering methods for high-dimensional</span></div>
<div style="position:absolute;left:67.20px;top:678.24px" class="cls_005"><span class="cls_005">single-cell flow and mass cytometry data. </span><span class="cls_019">Cytometry A</span><span class="cls_005"> </span><span class="cls_012">89</span><span class="cls_005">, 1084-1096 (2016).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">31</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:24862px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-32.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">33.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">D’Acquisto, F. &amp; Crompton, T. CD3+CD4-CD8- (double negative) T cells: saviours or villains</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">of the immune response? </span><span class="cls_019">Biochem. Pharmacol.</span><span class="cls_005"> </span><span class="cls_012">82</span><span class="cls_005">, 333-340 (2011).</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">34.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Chongsathidkiet, P. </span><span class="cls_019">et al.</span><span class="cls_005"> Sequestration of T cells in bone marrow in the setting of glioblastoma</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_005"><span class="cls_005">and other intracranial tumors. </span><span class="cls_019">Nat. Med.</span><span class="cls_005"> </span><span class="cls_012">24</span><span class="cls_005">, 1459-1468 (2018).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">35.</span></div>
<div style="position:absolute;left:90.00px;top:181.44px" class="cls_005"><span class="cls_005">Kamran, N. </span><span class="cls_019">et al.</span><span class="cls_005"> Immunosuppressive Myeloid Cells’ Blockade in the Glioma</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">Microenvironment Enhances the Efficacy of Immune-Stimulatory Gene Therapy. </span><span class="cls_019">Mol. Ther.</span><span class="cls_005"> </span><span class="cls_012">25</span><span class="cls_005">, 232-</span></div>
<div style="position:absolute;left:67.20px;top:236.64px" class="cls_005"><span class="cls_005">248 (2017).</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">36.</span></div>
<div style="position:absolute;left:90.00px;top:264.24px" class="cls_005"><span class="cls_005">Gabrilovich, D. I. &amp; Nagaraj, S. Myeloid-derived suppressor cells as regulators of the immune</span></div>
<div style="position:absolute;left:67.20px;top:291.84px" class="cls_005"><span class="cls_005">system. </span><span class="cls_019">Nat. Rev. Immunol.</span><span class="cls_005"> </span><span class="cls_012">9</span><span class="cls_005">, 162-174 (2009).</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">37.</span></div>
<div style="position:absolute;left:90.00px;top:319.44px" class="cls_005"><span class="cls_005">Raychaudhuri, B. </span><span class="cls_019">et al.</span><span class="cls_005"> Myeloid derived suppressor cell infiltration of murine and human</span></div>
<div style="position:absolute;left:67.20px;top:347.04px" class="cls_005"><span class="cls_005">gliomas is associated with reduction of tumor infiltrating lymphocytes. </span><span class="cls_019">J. Neurooncol.</span><span class="cls_005"> </span><span class="cls_012">122</span><span class="cls_005">, 293-301</span></div>
<div style="position:absolute;left:67.20px;top:374.64px" class="cls_005"><span class="cls_005">(2015).</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">38.</span></div>
<div style="position:absolute;left:90.00px;top:402.24px" class="cls_005"><span class="cls_005">Wrammert, J., Källberg, E., Agace, W. W. &amp; Leanderson, T. Ly6C expression differentiates</span></div>
<div style="position:absolute;left:67.20px;top:429.84px" class="cls_005"><span class="cls_005">plasma cells from other B cell subsets in mice. </span><span class="cls_019">Eur. J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">32</span><span class="cls_005">, 97-103 (2002).</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">39.</span></div>
<div style="position:absolute;left:90.00px;top:457.44px" class="cls_005"><span class="cls_005">Graf, B. A. </span><span class="cls_019">et al.</span><span class="cls_005"> Biphenotypic B/macrophage cells express COX-1 and up-regulate COX-2</span></div>
<div style="position:absolute;left:67.20px;top:485.04px" class="cls_005"><span class="cls_005">expression and prostaglandin E(2) production in response to pro-inflammatory signals. </span><span class="cls_019">Eur. J.</span></div>
<div style="position:absolute;left:67.20px;top:512.64px" class="cls_019"><span class="cls_019">Immunol.</span><span class="cls_005"> </span><span class="cls_012">29</span><span class="cls_005">, 3793-3803 (1999).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">40.</span></div>
<div style="position:absolute;left:90.00px;top:540.24px" class="cls_005"><span class="cls_005">Borrello, M. A. &amp; Phipps, R. P. Fibroblast-secreted macrophage colony-stimulating factor is</span></div>
<div style="position:absolute;left:67.20px;top:567.84px" class="cls_005"><span class="cls_005">responsible for generation of biphenotypic B/macrophage cells from a subset of mouse B</span></div>
<div style="position:absolute;left:67.20px;top:595.44px" class="cls_005"><span class="cls_005">lymphocytes. </span><span class="cls_019">J. Immunol.</span><span class="cls_005"> </span><span class="cls_012">163</span><span class="cls_005">, 3605-3611 (1999).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">41.</span></div>
<div style="position:absolute;left:90.00px;top:623.04px" class="cls_005"><span class="cls_005">Simmons, S. </span><span class="cls_019">et al.</span><span class="cls_005"> Biphenotypic B-lymphoid/myeloid cells expressing low levels of Pax5:</span></div>
<div style="position:absolute;left:67.20px;top:650.64px" class="cls_005"><span class="cls_005">potential targets of BAL development. </span><span class="cls_019">Blood</span><span class="cls_005"> </span><span class="cls_012">120</span><span class="cls_005">, 3688-3698 (2012).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">32</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:25664px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-33.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">42.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Baker, G. J., Castro, M. G. &amp; Lowenstein, P. R. Isolation and Flow Cytometric Analysis of</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">Glioma-infiltrating Peripheral Blood Mononuclear Cells. </span><span class="cls_019">J Vis Exp</span><span class="cls_005"> (2015). doi:10.3791/53676</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">43.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Marvel, J. &amp; Mayer, A. CD45R gives immunofluorescence and transduces signals on mouse T</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_005"><span class="cls_005">cells. </span><span class="cls_019">European Journal of Immunology</span><span class="cls_005"> </span><span class="cls_012">18</span><span class="cls_005">, 825-828 (1988).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">44.</span></div>
<div style="position:absolute;left:90.00px;top:181.44px" class="cls_005"><span class="cls_005">Sebastián-Gámbaro, M. A., Lirón-Hernández, F. J. &amp; Fuentes-Arderiu, X. Intra- and inter-</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">individual biological variability data bank. </span><span class="cls_019">Eur J Clin Chem Clin Biochem</span><span class="cls_005"> </span><span class="cls_012">35</span><span class="cls_005">, 845-852 (1997).</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">45.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">Moore, P. A. </span><span class="cls_019">et al.</span><span class="cls_005"> BLyS: member of the tumor necrosis factor family and B lymphocyte</span></div>
<div style="position:absolute;left:67.20px;top:264.24px" class="cls_005"><span class="cls_005">stimulator. </span><span class="cls_019">Science</span><span class="cls_005"> </span><span class="cls_012">285</span><span class="cls_005">, 260-263 (1999).</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">46.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Rousseeuw, PJ. Silhouettes: A graphical aid to the interpretation and validation of cluster</span></div>
<div style="position:absolute;left:67.20px;top:319.44px" class="cls_005"><span class="cls_005">analysis. </span><span class="cls_019">J. Comput. Appl. Math</span><span class="cls_005"> </span><span class="cls_012">20</span><span class="cls_005">, 53-65 (1987).</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">47.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">Garcillán, B. </span><span class="cls_019">et al.</span><span class="cls_005"> γδ T Lymphocytes in the Diagnosis of Human T Cell Receptor</span></div>
<div style="position:absolute;left:67.20px;top:374.64px" class="cls_005"><span class="cls_005">Immunodeficiencies. </span><span class="cls_019">Front Immunol</span><span class="cls_005"> </span><span class="cls_012">6</span><span class="cls_005">, 20 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">48.</span></div>
<div style="position:absolute;left:90.00px;top:402.24px" class="cls_005"><span class="cls_005">Coffelt, S. B. </span><span class="cls_019">et al.</span><span class="cls_005"> IL-17-producing γδ T cells and neutrophils conspire to promote breast cancer</span></div>
<div style="position:absolute;left:67.20px;top:429.84px" class="cls_005"><span class="cls_005">metastasis. </span><span class="cls_019">Nature</span><span class="cls_005"> </span><span class="cls_012">522</span><span class="cls_005">, 345-348 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">49.</span></div>
<div style="position:absolute;left:90.00px;top:457.44px" class="cls_005"><span class="cls_005">Pimentel, H., Bray, N. L., Puente, S., Melsted, P. &amp; Pachter, L. Differential analysis of RNA-seq</span></div>
<div style="position:absolute;left:67.20px;top:485.04px" class="cls_005"><span class="cls_005">incorporating quantification uncertainty. </span><span class="cls_019">Nat. Methods</span><span class="cls_005"> </span><span class="cls_012">14</span><span class="cls_005">, 687-690 (2017).</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">50.</span></div>
<div style="position:absolute;left:90.00px;top:512.64px" class="cls_005"><span class="cls_005">Wald A. Sequential Tests of Statistical Hypotheses. </span><span class="cls_019">Ann. Math. Statist.</span><span class="cls_005"> </span><span class="cls_012">16</span><span class="cls_005">, 117-186 (1945).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">51.</span></div>
<div style="position:absolute;left:90.00px;top:540.24px" class="cls_005"><span class="cls_005">Welch, M. D., DePace, A. H., Verma, S., Iwamatsu, A. &amp; Mitchison, T. J. The human Arp2/3</span></div>
<div style="position:absolute;left:67.20px;top:567.84px" class="cls_005"><span class="cls_005">complex is composed of evolutionarily conserved subunits and is localized to cellular regions of</span></div>
<div style="position:absolute;left:67.20px;top:595.44px" class="cls_005"><span class="cls_005">dynamic actin filament assembly. </span><span class="cls_019">J. Cell Biol.</span><span class="cls_005"> </span><span class="cls_012">138</span><span class="cls_005">, 375-384 (1997).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">52.</span></div>
<div style="position:absolute;left:90.00px;top:623.04px" class="cls_005"><span class="cls_005">Zhang, Y. </span><span class="cls_019">et al.</span><span class="cls_005"> Arp2/3 complex controls T cell homeostasis by maintaining surface TCR levels</span></div>
<div style="position:absolute;left:67.20px;top:650.64px" class="cls_005"><span class="cls_005">via regulating TCR+ endosome trafficking. </span><span class="cls_019">Sci Rep</span><span class="cls_005"> </span><span class="cls_012">7</span><span class="cls_005">, 8952 (2017).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">33</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:26466px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-34.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">53.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Lin, J.-R., Fallahi-Sichani, M. &amp; Sorger, P. K. Highly multiplexed imaging of single cells using a</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">high-throughput cyclic immunofluorescence method. </span><span class="cls_019">Nat Commun</span><span class="cls_005"> </span><span class="cls_012">6</span><span class="cls_005">, 8390 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">54.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Lin, J.-R. </span><span class="cls_019">et al.</span><span class="cls_005"> Highly multiplexed immunofluorescence imaging of human tissues and tumors</span></div>
<div style="position:absolute;left:67.20px;top:153.84px" class="cls_005"><span class="cls_005">using t-CyCIF and conventional optical microscopes. </span><span class="cls_019">Elife</span><span class="cls_005"> </span><span class="cls_012">7</span><span class="cls_005">, (2018).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">55.</span></div>
<div style="position:absolute;left:90.00px;top:181.44px" class="cls_005"><span class="cls_005">Bell, E. B. &amp; Sparshott, S. M. Interconversion of CD45R subsets of CD4 T cells in vivo. </span><span class="cls_019">Nature</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_012"><span class="cls_012">348</span><span class="cls_005">, 163-166 (1990).</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">56.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">Rieckmann, J. C. </span><span class="cls_019">et al.</span><span class="cls_005"> Social network architecture of human immune cells unveiled by</span></div>
<div style="position:absolute;left:67.20px;top:264.24px" class="cls_005"><span class="cls_005">quantitative proteomics. </span><span class="cls_019">Nat. Immunol.</span><span class="cls_005"> </span><span class="cls_012">18</span><span class="cls_005">, 583-593 (2017).</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">57.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Yu, Y. </span><span class="cls_019">et al.</span><span class="cls_005"> Recent advances in CD8+ regulatory T cell research. </span><span class="cls_019">Oncol Lett</span><span class="cls_005"> </span><span class="cls_012">15</span><span class="cls_005">, 8187-8194</span></div>
<div style="position:absolute;left:67.20px;top:319.44px" class="cls_005"><span class="cls_005">(2018).</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">58.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">Filaci, G. &amp; Suciu-Foca, N. CD8+ T suppressor cells are back to the game: are they players in</span></div>
<div style="position:absolute;left:67.20px;top:374.64px" class="cls_005"><span class="cls_005">autoimmunity? </span><span class="cls_019">Autoimmun Rev</span><span class="cls_005"> </span><span class="cls_012">1</span><span class="cls_005">, 279-283 (2002).</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">59.</span></div>
<div style="position:absolute;left:90.00px;top:402.24px" class="cls_005"><span class="cls_005">Vukmanovic-Stejic, M., Thomas, M. J., Noble, A. &amp; Kemeny, D. M. Specificity, restriction and</span></div>
<div style="position:absolute;left:67.20px;top:429.84px" class="cls_005"><span class="cls_005">effector mechanisms of immunoregulatory CD8 T cells. </span><span class="cls_019">Immunology</span><span class="cls_005"> </span><span class="cls_012">102</span><span class="cls_005">, 115-122 (2001).</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">60.</span></div>
<div style="position:absolute;left:90.00px;top:457.44px" class="cls_005"><span class="cls_005">Cheroutre, H., Lambolez, F. &amp; Mucida, D. The light and dark sides of intestinal intraepithelial</span></div>
<div style="position:absolute;left:67.20px;top:485.04px" class="cls_005"><span class="cls_005">lymphocytes. </span><span class="cls_019">Nat. Rev. Immunol.</span><span class="cls_005"> </span><span class="cls_012">11</span><span class="cls_005">, 445-456 (2011).</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">61.</span></div>
<div style="position:absolute;left:90.00px;top:512.64px" class="cls_005"><span class="cls_005">Sun, M., He, C., Cong, Y. &amp; Liu, Z. Regulatory immune cells in regulation of intestinal</span></div>
<div style="position:absolute;left:67.20px;top:540.24px" class="cls_005"><span class="cls_005">inflammatory response to microbiota. </span><span class="cls_019">Mucosal Immunol</span><span class="cls_005"> </span><span class="cls_012">8</span><span class="cls_005">, 969-978 (2015).</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">62.</span></div>
<div style="position:absolute;left:90.00px;top:567.84px" class="cls_005"><span class="cls_005">Streuli, M., Hall, L. R., Saga, Y., Schlossman, S. F. &amp; Saito, H. Differential usage of three exons</span></div>
<div style="position:absolute;left:67.20px;top:595.44px" class="cls_005"><span class="cls_005">generates at least five different mRNAs encoding human leukocyte common antigens. </span><span class="cls_019">J. Exp. Med.</span></div>
<div style="position:absolute;left:67.20px;top:623.04px" class="cls_012"><span class="cls_012">166</span><span class="cls_005">, 1548-1566 (1987).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">34</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:27268px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-35.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">63.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Johnson, P., Greenbaum, L., Bottomly, K. &amp; Trowbridge, I. S. Identification of the alternatively</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">spliced exons of murine CD45 (T200) required for reactivity with B220 and other T200-restricted</span></div>
<div style="position:absolute;left:67.20px;top:126.24px" class="cls_005"><span class="cls_005">antibodies. </span><span class="cls_019">J. Exp. Med.</span><span class="cls_005"> </span><span class="cls_012">169</span><span class="cls_005">, 1179-1184 (1989).</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">64.</span></div>
<div style="position:absolute;left:90.00px;top:153.84px" class="cls_005"><span class="cls_005">Blasius, A. L., Barchet, W., Cella, M. &amp; Colonna, M. Development and function of murine</span></div>
<div style="position:absolute;left:67.20px;top:181.44px" class="cls_005"><span class="cls_005">B220+CD11c+NK1.1+ cells identify them as a subset of NK cells. </span><span class="cls_019">J. Exp. Med.</span><span class="cls_005"> </span><span class="cls_012">204</span><span class="cls_005">, 2561-2568</span></div>
<div style="position:absolute;left:67.20px;top:209.04px" class="cls_005"><span class="cls_005">(2007).</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">65.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">McNeill, L. </span><span class="cls_019">et al.</span><span class="cls_005"> CD45 isoforms in T cell signalling and development. </span><span class="cls_019">Immunol. Lett.</span><span class="cls_005"> </span><span class="cls_012">92</span><span class="cls_005">, 125-</span></div>
<div style="position:absolute;left:67.20px;top:264.24px" class="cls_005"><span class="cls_005">134 (2004).</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">66.</span></div>
<div style="position:absolute;left:90.00px;top:291.84px" class="cls_005"><span class="cls_005">Marvel, J., Poirier, G. &amp; Lightstone, E. Anti-CD45RA antibodies increase the proliferation of</span></div>
<div style="position:absolute;left:67.20px;top:319.44px" class="cls_005"><span class="cls_005">mouse T cells to phytohemagglutinin through the interleukin 2/interleukin 2 receptor pathway. </span><span class="cls_019">Eur. J.</span></div>
<div style="position:absolute;left:67.20px;top:347.04px" class="cls_019"><span class="cls_019">Immunol.</span><span class="cls_005"> </span><span class="cls_012">19</span><span class="cls_005">, 2005-2010 (1989).</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">67.</span></div>
<div style="position:absolute;left:90.00px;top:374.64px" class="cls_005"><span class="cls_005">Goltsev, Y. </span><span class="cls_019">et al.</span><span class="cls_005"> Deep Profiling of Mouse Splenic Architecture with CODEX Multiplexed</span></div>
<div style="position:absolute;left:67.20px;top:402.24px" class="cls_005"><span class="cls_005">Imaging. </span><span class="cls_019">Cell</span><span class="cls_005"> </span><span class="cls_012">174</span><span class="cls_005">, 968-981.e15 (2018).</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">68.</span></div>
<div style="position:absolute;left:90.00px;top:429.84px" class="cls_005"><span class="cls_005">Gut, G., Herrmann, M. D. &amp; Pelkmans, L. Multiplexed protein maps link subcellular</span></div>
<div style="position:absolute;left:67.20px;top:457.44px" class="cls_005"><span class="cls_005">organization to cellular states. </span><span class="cls_019">Science</span><span class="cls_005"> </span><span class="cls_012">361</span><span class="cls_005">, (2018).</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">69.</span></div>
<div style="position:absolute;left:90.00px;top:485.04px" class="cls_005"><span class="cls_005">Telford, W. G., Babin, S. A., Khorev, S. V. &amp; Rowe, S. H. Green fiber lasers: an alternative to</span></div>
<div style="position:absolute;left:67.20px;top:512.64px" class="cls_005"><span class="cls_005">traditional DPSS green lasers for flow cytometry. </span><span class="cls_019">Cytometry A</span><span class="cls_005"> </span><span class="cls_012">75</span><span class="cls_005">, 1031-1039 (2009).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">70.</span></div>
<div style="position:absolute;left:90.00px;top:540.24px" class="cls_005"><span class="cls_005">Parks, D. R., Roederer, M. &amp; Moore, W. A. A new ‘Logicle’ display method avoids deceptive</span></div>
<div style="position:absolute;left:67.20px;top:567.84px" class="cls_005"><span class="cls_005">effects of logarithmic scaling for low signals and compensated data. </span><span class="cls_019">Cytometry A</span><span class="cls_005"> </span><span class="cls_012">69</span><span class="cls_005">, 541-551</span></div>
<div style="position:absolute;left:67.20px;top:595.44px" class="cls_005"><span class="cls_005">(2006).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">71.</span></div>
<div style="position:absolute;left:90.00px;top:623.04px" class="cls_005"><span class="cls_005">McDonald, B., Urrutia, R., Yipp, B. G., Jenne, C. N. &amp; Kubes, P. Intravascular neutrophil</span></div>
<div style="position:absolute;left:67.20px;top:650.64px" class="cls_005"><span class="cls_005">extracellular traps capture bacteria from the bloodstream during sepsis. </span><span class="cls_019">Cell Host Microbe</span><span class="cls_005"> </span><span class="cls_012">12</span><span class="cls_005">, 324-</span></div>
<div style="position:absolute;left:67.20px;top:678.24px" class="cls_005"><span class="cls_005">333 (2012).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">35</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:28070px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-36.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">72.</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Barnard, M. R. </span><span class="cls_019">et al.</span><span class="cls_005"> Effects of platelet binding on whole blood flow cytometry assays of</span></div>
<div style="position:absolute;left:67.20px;top:98.64px" class="cls_005"><span class="cls_005">monocyte and neutrophil procoagulant activity. </span><span class="cls_019">J. Thromb. Haemost.</span><span class="cls_005"> </span><span class="cls_012">3</span><span class="cls_005">, 2563-2570 (2005).</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">36</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:28872px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-37.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">ONLINE METHODS</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_019"><span class="cls_019">General reagents</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">ddH</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">0; RPMI-1640 (Corning, Cat. No. 10-040-CV); L-glutamine (Gibco, Cat. No. 25030-081);</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">penicillin-streptomycin (10,000 U/mL) (ThermoFisher Scientific, Cat. No. 15140-163); heat-inactivated</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">(HI) fetal bovine serum (FBS) (Gibco, Cat. No. 16140-071); Dulbecco’s phosphate buffered saline</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">(DPBS) w/o CaCl</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">, MgCl</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> (Corning, Cat. No. 21-040-CV); 5% (w/v) sodium azide (NaN</span><span class="cls_006"><sub>3</sub></span><span class="cls_005">) (BDH, Cat.</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">No. BDH7465-2); ethylenediaminetetraacetic acid (EDTA) disodium salt dehydrate</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">(C</span><span class="cls_006"><sub>10</sub></span><span class="cls_005">H</span><span class="cls_006"><sub>14</sub></span><span class="cls_005">N</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">Na</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O</span><span class="cls_006"><sub>8</sub></span><span class="cls_005">•2H</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O) (Sigma Aldrich, Cat. No. ED2SS); 15 mL polypropylene conical tubes (Falcon,</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">Cat. No. 352097); 50 mL polypropylene conical tubes (Falcon, Cat. No. 352098); 5 mL polystyrene</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">serological pipettes (Corning, Cat. No. 4050); 10 mL polystyrene serological pipettes (Corning, Cat. No.</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">4100); micropipettes (1000 µL, 200 µL, 20 µL, 10 µL) (Gilson); research plus 12-channel pipette (50-</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">300 µl), (Eppendorf, Cat. No. 3122000060); 0.1-10 µl TipOne natural pipet tips (USA Scientific, Cat.</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">No. 1111-3200); 1.0-20 µl TipOne natural pipet tips (USA Scientific, Cat. No. 1120-1810); 1-200 µl</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">TipOne natural pipet tips (USA Scientific, Cat. No. 1111-1200); 101-1,000 µl TipOne natural pipet tips</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">(USA Scientific, Cat. No. 1111-2820); 40 µm nylon mesh cell strainers (Falcon, Cat. No. 352340); 2 L</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">polyethylene Dewar flask (Nalgene, Cat. No. 4150-2000); sterile cryogenic storage vials (Sigma-</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">Aldrich, Cat. No. V7634); mini vortexer 120V (VWR, Cat. No. 58816-121); polypropylene general-</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">purpose test tube racks (Nalgene, Cat. No.  5930-0020); 96-well reversible microcentrifuge tube rack</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">(Bio Plas, Cat. No. 0091); S1 pipet filler (ThermoFisher Scientific, Cat. No. 9531); 9 L TruCool</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">rectangular ethylene-vinyl acetate foam ice pans (BioCision, Cat. No. BCS-112); 1.5 mL</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">microcentrifuge tubes (USA Scientific, Cat. No. 1615-5500); gel loading tips (Costar, Cat. No. 4853);</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">60 mm x 15 mm polystyrene tissue culture dishes (Falcon, Cat. No. 353002); 0.4% trypan blue solution</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">(Gibco, Cat. No. 15250061)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">37</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:29674px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-38.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_019"><span class="cls_019">Reagents germane to mouse euthanasia, perfusion, and tissue processing</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">ketamine hydrochloride injection (VEDCO, NDC: 50989-996-06); xylazine hydrochloride injection</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">(AKORN INC, NDC: 59399-111-50); 0.9% sodium chloride (NaCl) injection, USP (Hospira, NDC</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">0409-4888-10); 1 mL Norm-Ject® sterile Luer-slip syringes (Henke Sass Wolf, Cat. No. 4010.200V0);</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">PrecisionGlide needles - 26G x ½ (0.45 mm x 13 mm) (BD, Cat. No. 305111); sodium chloride (NaCl)</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">(Sigma Aldrich, Cat. No. S9888); calcium chloride (CaCl</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">•2H</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O) (Sigma Aldrich, Cat. No. C8106);</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">sodium phosphate monobasic (NaH</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">PO</span><span class="cls_006"><sub>4</sub></span><span class="cls_005">•2H</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O) (Sigma Aldrich, Cat. No. 71505); D-glucose (C</span><span class="cls_006"><sub>6</sub></span><span class="cls_005">H</span><span class="cls_006"><sub>12</sub></span><span class="cls_005">O</span><span class="cls_006"><sub>6</sub></span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">(Sigma Aldrich, Cat. No. G8270); sodium bicarbonate (NaHCO</span><span class="cls_006"><sub>3</sub></span><span class="cls_005">) (Sigma Aldrich, Cat. No. S5761);</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">potassium chloride (KCl) (Sigma Aldrich, Cat. No. P9333); heparin sodium salt from porcine intestinal</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">mucosa (Sigma-Aldrich, Cat. No. H4784); extruded polystyrene foam block (2); Halsted-mosquito</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">hemostat (2) (Fine Science Tools, Cat. No. 13008-12); fine scissors—martensitic stainless steel (2) (Fine</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">Science Tools, Cat. No. 14094-11); Friedman rongeur (Fine Science Tools, Cat. No. 16000-14); Littauer</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">bone cutters (Fine Science Tools, Cat. No. 16152-12); cover-glass forceps (Fine Science Tools, Cat. No.</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">11073-10); Dumont #5 forceps (2) (Fine Science Tools, Cat. No. 11252-40); Graefe forceps (2) (Fine</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">Science Tools, Cat. No. 11051-10); Masterflex L/S digital pump system with easy-load II pump head,</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">600 RPM, 115/230V (Cole-Parmer, Cat. No. EW-77921-75); 20 G x 1 1⁄2&quot; aluminum hub blunt needles</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">(Kendall, Cat. No. 8881202363); razor blades (VWR, Cat. No. 55411-050); frosted microscope slides</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">(Fisher Scientific, Cat. No. 12-550-343); 3 mL Luer-Lok® syringes (BD, Cat. No. 309657);</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">PrecisionGlide needles - 23G x 1 (0.6 mm x 25 mm) (BD, Cat. No. 305145); Falcon 3 mL polyethylene</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">transfer pipets (Corning, Cat. No. 357524)</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_019"><span class="cls_019">Reagents germane to immunolabeling</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">38</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:30476px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-39.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">Brilliant Stain Buffer (BD Biosciences, Cat. No. 563794); TruStain FcX anti-mouse CD16/32 antibody</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">(BioLegend, Cat. No. 101320); fixable viability dye, eFluor 455UV (eBioscience, Cat. No. 65-0868-14);</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">Brilliant Ultraviolet 737-conjugated anti-mouse CD11b, clone: M1/70, isotype: rat DA/HA IgG2b, κ</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">(BD Biosciences, Cat. No. 564443); V500-conjugated anti-mouse CD45, clone: 30-F11, isotype: rat</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">LOU/M IgG2b, κ (BD Biosciences, Cat. No. 561487); Brilliant Violet 605-conjugated anti-mouse CD4,</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">clone: RM4-5, isotype: rat IgG2a, κ (BioLegend, Cat. No. 100548); Brilliant Violet 711-conjugated anti-</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">mouse Ly6G, clone: 1A8, isotype: rat IgG2a, κ (BioLegend, Cat. No. 127643); Alexa Fluor 488-</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">conjugated anti-mouse CD3ε, clone: 145-2C11, isotype: Armenian hamster IgG (eBioscience, Cat. No.</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">53-0031-82); PE/Cy7-conjugated anti-mouse CD49b, clone: HMα2, isotype: Armenian hamster IgG</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">(BioLegend, Cat. No. 103518); PE-conjugated anti-mouse F4/80, clone: BM8, isotype: rat IgG2a, κ</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">(BioLegend, Cat. No. 123110); PE-CF594-conjugated anti-mouse CD8α, clone: 53-6.7, isotype: rat</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">LOU/M IgG2a, κ (BD Biosciences, Cat. No. 562283); PerCP/Cy5.5-conjugated anti-mouse/human</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">CD45R/B220, clone: RA3-6B2, isotype: rat IgG2a, κ (BioLegend, Cat. No. 103236); Alexa Fluor 647-</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">conjugated anti-mouse CD11c, clone: N418, isotype: Armenian hamster IgG (BioLegend, Cat. No.</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">117312); APC/Cy7-conjugated anti-mouse Ly6C, clone: HK1.4, isotype: rat IgG2c, κ (BioLegend, Cat.</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">No. 128026); V500-conjugated rat IgG2b, κ isotype control antibody, clone: A95-1, isotype: rat LOU/M</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">IgG2b, κ (BD Biosciences, Cat. No. 560784); purified anti-human CD45R, clone: MB1, isotype: mouse</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">IgG1 (Abnova, Cat. No. MAB16165); Alexa Fluor 488-conjugated anti-human CD8α, clone: AMC908,</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">isotype: mouse IgG2a, κ (eBioscience, Cat. No. 53-0008-80); Alexa Fluor 555-conjugated anti-human</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">CD3δ, clone: EP4426, isotype: rabbit IgG (abcam, Cat. No. AB208514); fixation/permeabilization</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">solution kit (BD Biosciences, Cat. No. 554714); 4&apos;,6-diamidino-2-phenylindole, dihydrochloride (DAPI)</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">(ThermoFisher Scientific, Cat. No. D1306); 96-well V-bottom, non-treated, polystyrene microplate</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">39</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:31278px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-40.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">(Costar, Cat No. 3897); 12-well V-bottom reagent reservoir (Argos Technologies, Cat. No. B3135);</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">Microseal ‘F’ foil seal (Bio-Rad; Cat. No. MSF1001)</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_019"><span class="cls_019">Reagents germane to flow cytometry</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">Sphero rainbow fluorescent particles (3.0-3.4 µm) (BD Biosciences, Cat. No. 556291); FACSDiva</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">CS&amp;T research beads (BD Biosciences, Cat. No. 655051)</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_012"><span class="cls_012">Mice</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">Twelve-week-old female C57BL/6J mice syngeneic to the mouse GL261 GBM model were used in this</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">study (Jackson Laboratory, Bar Harbor, ME). All animal experiments were conducted in accordance</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">with procedures preapproved by the Institutional Animal Care and Use Committee (IACUC) and</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">conformed to the policies and procedures of the Center for Comparative Medicine at Harvard Medical</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">School in agreement with the National Research Council’s “Guide for the Care and Use of Laboratory</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">Animals”.</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_012"><span class="cls_012">Cell Line</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">The GL261 GBM cell line was obtained from the Developmental Therapeutics Program (DTP), Division</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">of Cancer Treatment and Diagnosis (DCTD) tumor repository through a material transfer agreement</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">with the Biological Testing Branch (BTB) of the National Cancer Institute (NCI).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_012"><span class="cls_012">Clinical Specimens</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">40</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:32080px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-41.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">All deidentified human brain tumor resections used in this study were obtained from consented patients</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">under care at Brigham and Women’s Hospital or Dana-Farber Cancer Institute in association with</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">Institutional Review Board (IRB) protocols: 10-417 or 11-104 (17-000).</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_012"><span class="cls_012">Major Equipment</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">BD LSR II Special Order Research Product (SORP) flow cytometer w/ BD High Throughput Sampler</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">(HTS):</span></div>
<div style="position:absolute;left:72.00px;top:265.20px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  Laser line - power:</span></div>
<div style="position:absolute;left:108.00px;top:292.80px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:292.80px" class="cls_005"><span class="cls_005">488 nm laser - 20 mw (run at 20 mw)</span></div>
<div style="position:absolute;left:108.00px;top:320.40px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:320.40px" class="cls_005"><span class="cls_005">405 nm laser - 50 mw (run at 50 mw)</span></div>
<div style="position:absolute;left:108.00px;top:347.76px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:347.76px" class="cls_005"><span class="cls_005">594 nm laser - 200 mw (run at 125 mw)</span></div>
<div style="position:absolute;left:108.00px;top:375.36px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:375.36px" class="cls_005"><span class="cls_005">355 nm laser - 20 mw (run at 20 mw)</span></div>
<div style="position:absolute;left:54.00px;top:413.04px" class="cls_005"><span class="cls_005">BioTek EL406 automated microplate washer/dispenser:</span></div>
<div style="position:absolute;left:72.00px;top:441.60px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  In an effort to minimize cell loss, aspiration steps involving 96-well V-bottom microplates were</span></div>
<div style="position:absolute;left:90.00px;top:469.20px" class="cls_005"><span class="cls_005">performed using this instrument such that a 50 µL residual volume remained after each</span></div>
<div style="position:absolute;left:90.00px;top:496.80px" class="cls_005"><span class="cls_005">aspiration. This factor was accounted for in all reported dilutions. Instrument configurations were</span></div>
<div style="position:absolute;left:90.00px;top:524.40px" class="cls_005"><span class="cls_005">as follows:</span></div>
<div style="position:absolute;left:108.00px;top:552.00px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  plate type: 96-well</span></div>
<div style="position:absolute;left:108.00px;top:579.60px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  W-aspirate</span></div>
<div style="position:absolute;left:108.00px;top:607.20px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  vacuum filtration: false</span></div>
<div style="position:absolute;left:108.00px;top:634.80px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  travel rate: 1 (4.1 &amp; 1.0 mm/sec)</span></div>
<div style="position:absolute;left:108.00px;top:662.40px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  delay: 0 msec</span></div>
<div style="position:absolute;left:108.00px;top:690.00px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  z-offset: 55 steps (6.99 mm above carrier)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">41</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:32882px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-42.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:108.00px;top:71.04px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  x-offset: 0 steps (center of well)</span></div>
<div style="position:absolute;left:108.00px;top:98.64px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  y-offset: 0 steps (center of well)</span></div>
<div style="position:absolute;left:108.00px;top:126.24px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  secondary aspirate: no</span></div>
<div style="position:absolute;left:54.00px;top:163.92px" class="cls_005"><span class="cls_005">Beckman Coulter Avanti J-26XP centrifuge:</span></div>
<div style="position:absolute;left:72.00px;top:192.48px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  JS-5.3 anodized aluminum swinging-bucket rotor (Beckman Coulter, Cat. No. 368690)</span></div>
<div style="position:absolute;left:54.00px;top:229.92px" class="cls_005"><span class="cls_005">Bransonic CPXH ultrasonic cleaning bath:</span></div>
<div style="position:absolute;left:72.00px;top:258.48px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  model 3800</span></div>
<div style="position:absolute;left:54.00px;top:323.52px" class="cls_012"><span class="cls_012">Reagent Preparation</span></div>
<div style="position:absolute;left:54.00px;top:351.12px" class="cls_005"><span class="cls_005">supplemented RPMI-1640 (RPMI-1640 with L-glutamine, 100 U/mL penicillin-100 U/mL</span></div>
<div style="position:absolute;left:54.00px;top:378.72px" class="cls_005"><span class="cls_005">streptomycin, 10% HI-FBS, 0.05% sodium azide):</span></div>
<div style="position:absolute;left:72.00px;top:407.28px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to a 500 mL bottle of RPMI-1640 with L-glutamine were added:</span></div>
<div style="position:absolute;left:108.00px;top:434.88px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:434.88px" class="cls_005"><span class="cls_005">5 mL of a 10,000 U/mL penicillin-10,000 µg/mL streptomycin solution</span></div>
<div style="position:absolute;left:108.00px;top:462.48px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:462.48px" class="cls_005"><span class="cls_005">50 mL of HI-FBS</span></div>
<div style="position:absolute;left:108.00px;top:490.08px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:490.08px" class="cls_005"><span class="cls_005">2.5 mL of a 5% w/v sodium azide solution</span></div>
<div style="position:absolute;left:108.00px;top:517.68px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:517.68px" class="cls_005"><span class="cls_005">5 mL of a 10% EDTA solution</span></div>
<div style="position:absolute;left:54.00px;top:555.12px" class="cls_005"><span class="cls_005">heparinized Tyrode’s solution:</span></div>
<div style="position:absolute;left:72.00px;top:583.68px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to a 1 L glass screw-cap storage bottle on a stirring plate and containing a magnetic stirring bar</span></div>
<div style="position:absolute;left:90.00px;top:611.28px" class="cls_005"><span class="cls_005">and 1 L ddH</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O were added:</span></div>
<div style="position:absolute;left:108.00px;top:638.88px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:638.88px" class="cls_005"><span class="cls_005">8.0 g sodium chloride</span></div>
<div style="position:absolute;left:108.00px;top:666.48px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:666.48px" class="cls_005"><span class="cls_005">0.264 g calcium chloride</span></div>
<div style="position:absolute;left:108.00px;top:694.08px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:694.08px" class="cls_005"><span class="cls_005">0.05 g sodium phosphate monobasic</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">42</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:33684px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-43.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:108.00px;top:71.04px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:71.04px" class="cls_005"><span class="cls_005">1.0 g D-glucose</span></div>
<div style="position:absolute;left:108.00px;top:98.64px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:98.64px" class="cls_005"><span class="cls_005">1.0 g sodium bicarbonate</span></div>
<div style="position:absolute;left:108.00px;top:126.24px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:126.24px" class="cls_005"><span class="cls_005">0.2 g potassium chloride</span></div>
<div style="position:absolute;left:108.00px;top:153.84px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:153.84px" class="cls_005"><span class="cls_005">100 U of heparin sodium</span></div>
<div style="position:absolute;left:72.00px;top:182.40px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  Salts were allowed to completely dissolve prior to storage at 4 °C.</span></div>
<div style="position:absolute;left:54.00px;top:219.84px" class="cls_005"><span class="cls_005">ammonium-chloride-potassium (ACK) lysis buffer (1X):</span></div>
<div style="position:absolute;left:72.00px;top:248.40px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to 1 L of stirring ddH</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">0 were added:</span></div>
<div style="position:absolute;left:108.00px;top:276.00px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:276.00px" class="cls_005"><span class="cls_005">8.29 g of ammonium chloride</span></div>
<div style="position:absolute;left:108.00px;top:303.60px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:303.60px" class="cls_005"><span class="cls_005">1.0 g of potassium bicarbonate</span></div>
<div style="position:absolute;left:108.00px;top:331.20px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:331.20px" class="cls_005"><span class="cls_005">37.2 mg of sodium EDTA</span></div>
<div style="position:absolute;left:72.00px;top:359.76px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  pH was adjusted to 7.4</span></div>
<div style="position:absolute;left:54.00px;top:397.20px" class="cls_005"><span class="cls_005">flow buffer (DPBS + 0.5% HI-FBS):</span></div>
<div style="position:absolute;left:72.00px;top:425.76px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to 95 mL of DPBS was added:</span></div>
<div style="position:absolute;left:108.00px;top:453.36px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:129.00px;top:453.36px" class="cls_005"><span class="cls_005">0.5 mL of HI-FBS</span></div>
<div style="position:absolute;left:72.00px;top:481.92px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  stored at 4°C</span></div>
<div style="position:absolute;left:54.00px;top:519.36px" class="cls_005"><span class="cls_005">flow buffer + azide (flow buffer + 0.05% sodium azide):</span></div>
<div style="position:absolute;left:72.00px;top:547.92px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to 49.5 mL of flow buffer was added:</span></div>
<div style="position:absolute;left:108.00px;top:575.28px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:129.00px;top:575.28px" class="cls_005"><span class="cls_005">0.5 mL of a 5% w/v sodium azide solution</span></div>
<div style="position:absolute;left:72.00px;top:603.84px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  stored at 4°C</span></div>
<div style="position:absolute;left:54.00px;top:641.52px" class="cls_005"><span class="cls_005">EDTA solution (1X DPBS containing 10% EDTA):</span></div>
<div style="position:absolute;left:72.00px;top:670.08px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  to 50 mL of DPBS was added:</span></div>
<div style="position:absolute;left:108.00px;top:697.44px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:129.00px;top:697.44px" class="cls_005"><span class="cls_005">5 g of disodium EDTA</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">43</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:34486px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-44.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:72.00px;top:72.00px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  placed in ultrasonic bath to facilitate dissolution</span></div>
<div style="position:absolute;left:72.00px;top:100.56px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">   stored at 4°C</span></div>
<div style="position:absolute;left:54.00px;top:138.00px" class="cls_005"><span class="cls_005">Fc block (flow buffer + azide containing 22.5 µg/mL anti-mouse CD16/32):</span></div>
<div style="position:absolute;left:72.00px;top:166.56px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  refer to accompanying spreadsheet for preparation details (</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:204.00px" class="cls_005"><span class="cls_005">fixable viability dye:</span></div>
<div style="position:absolute;left:72.00px;top:232.56px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  eFluor 455UV fixable viability dye was diluted 1.5:1,000 in 1X DPBS</span></div>
<div style="position:absolute;left:72.00px;top:261.12px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  refer to accompanying spreadsheet for preparation details (</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:326.16px" class="cls_012"><span class="cls_012">Antibody Titration</span></div>
<div style="position:absolute;left:54.00px;top:352.32px" class="cls_005"><span class="cls_005">Immunolabeling concentrations yielding the highest staining index (SI)</span><span class="cls_006"><sup>69</sup></span><span class="cls_005"> for each antibody in the study</span></div>
<div style="position:absolute;left:54.00px;top:381.36px" class="cls_005"><span class="cls_005">were determined by first harvesting the spleens of two 12-week-old female C57BL/6J mice placed under</span></div>
<div style="position:absolute;left:54.00px;top:408.96px" class="cls_005"><span class="cls_005">terminal anesthesia with a dose of 150 mg/kg of ketamine hydrochloride and 20 mg/kg xylazine</span></div>
<div style="position:absolute;left:54.00px;top:436.56px" class="cls_005"><span class="cls_005">hydrochloride diluted in sterile 0.9% NaCl delivered with a 1 mL tuberculin syringe equipped with a</span></div>
<div style="position:absolute;left:54.00px;top:464.16px" class="cls_005"><span class="cls_005">26G needle as a single intraperitoneal (i.p.) injection. Using opposing frosted ends of 2 glass microscope</span></div>
<div style="position:absolute;left:54.00px;top:491.76px" class="cls_005"><span class="cls_005">slides, each spleen was gently macerated and rinsed with 4 mL of supplemented RPMI-1640 into a 60 x</span></div>
<div style="position:absolute;left:54.00px;top:519.36px" class="cls_005"><span class="cls_005">15 mm polystyrene petri dish on ice. Splenocytes were aspirated from the dish, dispensed into a 15 mL</span></div>
<div style="position:absolute;left:54.00px;top:546.96px" class="cls_005"><span class="cls_005">conical tube, and centrifuged at 350 x g (max RCF) for 10 minutes at 4°C. The cell pellet was</span></div>
<div style="position:absolute;left:54.00px;top:574.56px" class="cls_005"><span class="cls_005">resuspended in 8 mL of a 1X ACK lysing buffer and placed on ice for 5 minutes to lyse red blood cells</span></div>
<div style="position:absolute;left:54.00px;top:602.16px" class="cls_005"><span class="cls_005">(RBCs). Six (6) mL of flow buffer + azide was added to the tube prior to filtering the cell suspension</span></div>
<div style="position:absolute;left:54.00px;top:629.76px" class="cls_005"><span class="cls_005">through a 40 µm nylon mesh into a fresh 15 mL conical tube. Splenocytes were again centrifuged at 350</span></div>
<div style="position:absolute;left:54.00px;top:657.36px" class="cls_005"><span class="cls_005">x g (max RCF) for 10 minutes at 4°C then resuspended in 2 mL of flow buffer + azide. Cell counting</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">44</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:35288px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-45.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">was performed using a trypan blue and a glass hemocytometer. Cell concentration was adjusted</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">accordingly to achieve a final concentration of 1x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> cells/mL.</span></div>
<div style="position:absolute;left:90.00px;top:124.80px" class="cls_005"><span class="cls_005">Two-hundred (200) µL of the 1x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> cell/mL splenocyte suspension were added to wells A-H of</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">11 concentric columns of a 96-well V-bottom microplate using a multichannel pipette followed by</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">centrifugstion at 100 x g (max RCF) for 3 minutes at 4°C. One-hundred and fifty (150) µL of cell</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">supernatant was aspirated from each well followed by resuspension in 100 µL of Fc block (15 µg/mL</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">final concentration). Splenocytes were incubated on ice for 5 minutes prior to centrifugation at 100 x g</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">(max RCF) for 3 minutes at 4°C. One-hundred (100) µL was aspirated from each well before</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">resuspension with 100 µL of target antibodies pre-diluted in Brilliant Stain Buffer to achieve the</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">following two-fold serial dilution series per column for each of the 11 antibodies used in the study: 24,</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">12, 6, 3, 1.5, 0.75, 0.375, and 0.1875 µg/mL. Splenocytes were immunolabeled on ice for 15 minutes in</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">the dark prior to centrifugation at 100 x g (max RCF) for 3 minutes at 4°C. Two-hundred (200) µL of</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">supernatant were then aspirated followed by resuspension with 100 µL of flow buffer + azide. This</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">washing step was repeated once except that 200 µL (instead of 100 µL) of flow buffer + azide was used</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">in the resuspension step. DAPI was added to each well at a final concentration of 1µg/mL and allowed to</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">incubate for 3-5 minutes prior to data acquisition.</span></div>
<div style="position:absolute;left:90.00px;top:512.64px" class="cls_005"><span class="cls_005">Immunolabeled splenocytes were analyzed on a BD LSR II SORP flow cytometer equipped with</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">a BD HTS for 96-well high-throughput sampling. The following acquisition gating strategy was used:</span></div>
<div style="position:absolute;left:54.00px;top:568.32px" class="cls_005"><span class="cls_005">(FSC-A vs. SSC-A)     (SSC-H vs. SSC-W)     (FSC-H vs. FSC-W)     (DAPI-A vs. FSC-A)     (CD</span><span class="cls_006"><sub>x</sub></span></div>
<div style="position:absolute;left:54.00px;top:599.04px" class="cls_005"><span class="cls_005">vs. count). A total of 10,000 viable singlets were analyzed per well. The median fluorescence intensities</span></div>
<div style="position:absolute;left:54.00px;top:625.20px" class="cls_005"><span class="cls_005">(MFIs) of the first (background) and second (first true positive) peaks, and the 84</span><span class="cls_006"><sup>th</sup></span><span class="cls_005"> percentile of the first</span></div>
<div style="position:absolute;left:54.00px;top:654.24px" class="cls_005"><span class="cls_005">peak were identified using the layout editor tool of FlowJo software from which a SI for each antibody</span></div>
<div style="position:absolute;left:54.00px;top:681.84px" class="cls_005"><span class="cls_005">was calculated according to the following formula: SI = (MFI</span><span class="cls_006"><sub>pos</sub></span><span class="cls_005">-MFI</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">)/[(84%</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">-MFI</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">)/0.995]. The</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">45</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:36090px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-46.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">maximum SI (SI</span><span class="cls_006"><sub>max</sub></span><span class="cls_005">) of each antibody was used as the target immunolabeling concentration in our</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">longitudinal study.</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_012"><span class="cls_012">Stereotactic Engraftment of GBM Cells into the Mouse Brain</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">The reader is referred here</span><span class="cls_006"><sup>42</sup></span><span class="cls_005"> for detailed instruction on how to stereotactically engraft glioma cells into</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">the mouse brain.</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_012"><span class="cls_012">Tissue Harvesting and Processing</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">The following disposable reagents were gathered and labeled before each of our study’s 3 time points:</span></div>
<div style="position:absolute;left:72.00px;top:320.40px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:320.40px" class="cls_005"><span class="cls_005">15 mL conical tubes (16)</span></div>
<div style="position:absolute;left:108.00px;top:347.76px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  labeled: “blood”, &lt;condition&gt;, &lt;replicate&gt;  (note: &lt;  &gt; indicates a variable)</span></div>
<div style="position:absolute;left:72.00px;top:376.32px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:376.32px" class="cls_005"><span class="cls_005">3 mL syringes equipped with 23G needles (16)</span></div>
<div style="position:absolute;left:112.50px;top:403.92px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">labeled: “marrow”, &lt;condition&gt;, &lt;replicate&gt;</span></div>
<div style="position:absolute;left:72.00px;top:432.48px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:432.48px" class="cls_005"><span class="cls_005">1 mL tuberculin syringes equipped with 26G needles (17)</span></div>
<div style="position:absolute;left:108.00px;top:460.08px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:460.08px" class="cls_005"><span class="cls_005">(16) labeled: “blood”, &lt;condition&gt;, &lt;replicate&gt;</span></div>
<div style="position:absolute;left:108.00px;top:487.68px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:487.68px" class="cls_005"><span class="cls_005">(1) unlabeled: used for injectable anesthesia</span></div>
<div style="position:absolute;left:72.00px;top:516.24px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:516.24px" class="cls_005"><span class="cls_005">60 x 15 mm polystyrene petri dishes (64)</span></div>
<div style="position:absolute;left:108.00px;top:543.60px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  labeled: &lt;tissue&gt; (excluding blood), &lt;condition&gt;, &lt;replicate&gt;</span></div>
<div style="position:absolute;left:108.00px;top:571.20px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:126.00px;top:571.20px" class="cls_005"><span class="cls_005">4 mL of supplemented RPMI-1640 was added to each dish</span></div>
<div style="position:absolute;left:72.00px;top:599.76px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:599.76px" class="cls_005"><span class="cls_005">0.5 mL microcentrifuge tubes (80)</span></div>
<div style="position:absolute;left:108.00px;top:627.36px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  labeled: &lt;tissue&gt;, &lt;condition&gt;, &lt;replicate&gt;</span></div>
<div style="position:absolute;left:54.00px;top:664.80px" class="cls_005"><span class="cls_005">One-hundred (100) µL of a 10% EDTA solution was added to each 15 mL conical tube. Four (4) mL of</span></div>
<div style="position:absolute;left:54.00px;top:692.40px" class="cls_005"><span class="cls_005">supplemented RPMI-1640 was added to each 60 x 15 mm polystyrene petri dish with the exception of</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">46</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:36892px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-47.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">those labeled “marrow”, to which only 2 mL of supplemented media was added (the other 2 mL of</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">supplemented RPMI-1640 was to be placed into each of the (16) 3 mL syringes). One-hundred ninety-</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">eight (198) µL of flow buffer was added to each microcentrifuge tube. Fifty (50) µL of a 10% EDTA</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">solution was added to each 1 mL tuberculin syringe to coat the inner barrel with EDTA by operating the</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">plunger several times. All conical tubes, petri dishes, 3 mL syringes, and microcentrifuge tubes were</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">stored on ice or at 4°C.</span></div>
<div style="position:absolute;left:90.00px;top:236.64px" class="cls_005"><span class="cls_005">Sixteen (16) mice were terminally anesthetized in series with a 150 mg/kg of ketamine</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">hydrochloride and 20 mg/kg xylazine hydrochloride diluted in sterile 0.9% NaCl delivered with the</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">unlabeled 1 mL tuberculin syringe as a single i.p. injection. Refer to accompanying spreadsheet for</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">preparation details (</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">). Once non-responsive to both toe and tail pinch, each</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">mouse was pinned ventral side up to an extruded polystyrene foam block by its front and hind paws</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">using four 26G needles and sprayed down with 70% EtOH to prevent fur from entering the dissection</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">cavity. Lymphoid organs were harvested in the following order: blood, thymus, spleen, superficial/deep</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">cervical lymph nodes, bone marrow. </span><span class="cls_019">Blood</span><span class="cls_005">: after making a “y” incision from the gut to the rib cage to</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">expose the heart, whole blood was aspirated from the right ventricle directly into one of the</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">aforementioned EDTA-coated 1 mL tuberculin syringes. The needle was removed before expelling</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">blood into its respectively labeled 15 mL conical tube stored on ice. Each mouse was then transcardially</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">perfused with heparinized and oxygenated (95% O</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">/5% CO</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">) Tyrode’s solution at a rate of 4.0</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">mL/minute for at least 2 minutes in a laminar flow hood to fully exsanguinate the circulatory system.</span></div>
<div style="position:absolute;left:54.00px;top:594.00px" class="cls_005"><span class="cls_005">The method for mouse transcardial perfusion has been described in detail elsewhere</span><span class="cls_006"><sup>42</sup></span><span class="cls_005">. </span><span class="cls_019">Thymus</span><span class="cls_005">: Once</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">exsanguinated, each mouse was returned to the extruded polystyrene foam block for thymus excision</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">with small dissection scissors and fine-tipped bent forceps being careful to remove any contaminating</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">adipose and the mediastinal lymph nodes. Thymi were placed into respectively labeled 60 x 15 mm</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">47</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:37694px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-48.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">polystyrene petri dishes on ice. </span><span class="cls_019">Spleen</span><span class="cls_005">: Spleens were excised using small dissection scissors and fine-</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">tipped bent forceps and placed into respectively labeled 60 x 15 mm polystyrene petri dishes on ice.</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_019"><span class="cls_019">Superficial/deep cervical lymph nodes</span><span class="cls_005">: Lymph nodes were dissected under a dissection microscope</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">using small dissection scissors and fine-tipped bent forceps. Excised nodes were placed into respectively</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">labeled 60 x 15 mm polystyrene petri dishes on ice. </span><span class="cls_019">Bone marrow</span><span class="cls_005">: The right hind limb of each mouse</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">was removed using bone cutters. Musculature and tendons were stripped away from the femur and tibia</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">and the proximal and distal epiphyses of each bone were removed using a single-edged razor blade.</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">Marrow from the two bones was flushed into the respectively labeled 60 x 15 mm polystyrene petri dish</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">on ice using the 2 mL of supplemented RPMI-1640 in the respectively labeled 3 mL syringe. Flushed</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">marrow was gently aspirated and expelled back into the petri dish one time to facilitate cell dissociation.</span></div>
<div style="position:absolute;left:90.00px;top:347.04px" class="cls_005"><span class="cls_005">Thymi, spleens, and cervical lymph nodes were macerated using opposing frosted ends of 2 glass</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">microscope slides which were dipped into the 4 mL of supplemented RPMI-1640 of the respectively</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">labeled 60 x 15 mm polystyrene petri dish to collect as may cells as possible. Plastic Pasteur pipettes</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">were used to transfer lymph nodes onto the frosted end of one glass microscope slide for maceration. A</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">fresh pair of slides were used for each tissue to prevent sample cross-contamination. Five (5) mL of ice-</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">cold DPBS was added to each petri dish prior to filtering each cell suspension through a fresh 40 µm</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">nylon mesh into respectively labeled 15 mL conical tubes on ice using a fresh 10 mL serological pipette</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">for every sample. Filtered samples were centrifuged at 400 x g (max RCF) for 10 minutes at 4°C. Cell</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">supernatants were aspirated and each pellet was resuspend in 4 mL of a 1X ACK lysing buffer using a</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">fresh 5 mL serological pipette for each sample to avoid cross-contamination. Samples were placed on</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">ice for 5 minutes to lyse RBCs. Tissue samples were centrifuged at 400 x g (max RCF) for 10 minutes at</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">4°C, had their supernatants aspirated, and their cells resuspended in flow buffer + azide using the</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">48</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:38496px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-49.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">following volumes: 1000 µL for thymi; 1000 µL for spleens; 200 µL for bone marrow; 100 µL for</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">deep/superficial cervical lymph nodes.</span></div>
<div style="position:absolute;left:90.00px;top:126.24px" class="cls_005"><span class="cls_005">Blood samples were next lysed by adding 10 mL of 1X ACK lysing buffer to each 15 mL</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">polypropylene conical tube. Tubes were placed back on ice for 5 minutes (or until blood color changed</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">from dark burgundy to bright red). Blood samples were then centrifuged at 400 x g (max RCF) for 10</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">minutes at 4°C, their supernatants aspirated, and their WBC pellets resuspended with 200 µL of flow</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">buffer + azide. Samples were stored on ice.</span></div>
<div style="position:absolute;left:90.00px;top:264.24px" class="cls_005"><span class="cls_005">Two (2) µL of each tissue sample were then added to the 198 µL of flow buffer (a 1:100 dilution)</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">in the respectively labeled 0.5 mL microcentrifuge tubes using a P20 micropipette equipped with a gel</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">loading tip. Ten (10) µL of each 1:100 dilution was further diluted 1:1 with a 0.4% trypan blue solution.</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">Ten (10) µL of the resultant solution was then used to estimate cell number using a brightfield</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_005"><span class="cls_005">microscope and a glass hemocytometer. Typical cell yields were as follows: 3x10</span><span class="cls_006"><sup>5</sup></span><span class="cls_005"> - 1x10</span><span class="cls_006"><sup>6</sup></span><span class="cls_005"> cells from</span></div>
<div style="position:absolute;left:54.00px;top:400.80px" class="cls_005"><span class="cls_005">the blood; 3x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> - 8x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> cells from the spleen; 6x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> - 8x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> from the thymus; 6x10</span><span class="cls_006"><sup>6</sup></span><span class="cls_005"> - 1.5x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> from</span></div>
<div style="position:absolute;left:54.00px;top:428.40px" class="cls_005"><span class="cls_005">the combined deep/superficial cervical lymph nodes; 9x10</span><span class="cls_006"><sup>6</sup></span><span class="cls_005"> - 1.5x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> from the bone marrow. Cell</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">counts were recorded in an spreadsheet which returned the volume of additional flow buffer + azide</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">required per sample to achieve a final concentration of approximately 2x10</span><span class="cls_006"><sup>7</sup></span><span class="cls_005"> cells/mL (</span><span class="cls_012">Supplementary</span></div>
<div style="position:absolute;left:54.00px;top:511.20px" class="cls_012"><span class="cls_012">Table 2</span><span class="cls_005">). Because mouse blood typically contained less than 2x10</span><span class="cls_006"><sup>6</sup></span><span class="cls_005"> total cells, the 200 µL volume of</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">each blood sample was simply split between the respective experimental well and the CD49b single-</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">positive compensation control well (</span><span class="cls_012">see Supplementary Fig. 2e for plate details</span><span class="cls_005">).</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_012"><span class="cls_012">Immunolabeling</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">The following disposable reagents were gathered and labeled before each of our study’s 3 time points:</span></div>
<div style="position:absolute;left:72.00px;top:679.20px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:679.20px" class="cls_005"><span class="cls_005">1.5 mL microcentrifuge tubes (12)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">49</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:39298px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-50.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:108.00px;top:71.04px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  labeled in duplicate: &lt;target antibody or CD45 isotype&gt;</span></div>
<div style="position:absolute;left:108.00px;top:98.64px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  tubes were placed into a microcentrifuge tube rack on ice</span></div>
<div style="position:absolute;left:72.00px;top:127.20px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:127.20px" class="cls_005"><span class="cls_005">12-well V-bottom reagent reservoir (1)</span></div>
<div style="position:absolute;left:108.00px;top:154.80px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  individual compartments labeled: &lt;target antibody or CD45 isotype&gt;</span></div>
<div style="position:absolute;left:72.00px;top:182.40px" class="cls_022"><span class="cls_022">o</span></div>
<div style="position:absolute;left:90.00px;top:182.40px" class="cls_005"><span class="cls_005">15 mL conical tubes (2)</span></div>
<div style="position:absolute;left:108.00px;top:210.00px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  one labeled “cocktail”, the other labeled “FVD”</span></div>
<div style="position:absolute;left:108.00px;top:237.60px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  placed on ice</span></div>
<div style="position:absolute;left:54.00px;top:275.04px" class="cls_005"><span class="cls_005">Dilutions of each antibody (1:10) were prepared using the 12 microcentrifuge tubes. Refer to</span></div>
<div style="position:absolute;left:54.00px;top:302.64px" class="cls_005"><span class="cls_005">accompanying spreadsheet for preparation details (</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">). From these 1:10 dilutions,</span></div>
<div style="position:absolute;left:54.00px;top:330.24px" class="cls_005"><span class="cls_005">antibodies were further diluted in the respective wells of a 12-well V-bottom reagent reservoir to</span></div>
<div style="position:absolute;left:54.00px;top:357.84px" class="cls_005"><span class="cls_005">achieve 100 µL volumes at the final antibody concentration to immunolabel the cells of each single-</span></div>
<div style="position:absolute;left:54.00px;top:385.44px" class="cls_005"><span class="cls_005">color compensation control. The remaining 1:10 antibody dilutions were used to prepare a master mix of</span></div>
<div style="position:absolute;left:54.00px;top:413.04px" class="cls_005"><span class="cls_005">combined target antibodies (excluding the CD45 isotype) placed in the 15 mL tube labeled “cocktail”.</span></div>
<div style="position:absolute;left:54.00px;top:440.64px" class="cls_005"><span class="cls_005">All final antibody dilutions were stored on ice in the dark.</span></div>
<div style="position:absolute;left:90.00px;top:468.24px" class="cls_005"><span class="cls_005">One-hundred (100) µL aliquots of each cell suspension were added to the wells of a 96-well V-</span></div>
<div style="position:absolute;left:54.00px;top:495.84px" class="cls_005"><span class="cls_005">bottom microplate using a multichannel pipette according to the plate layout specified in</span></div>
<div style="position:absolute;left:54.00px;top:523.44px" class="cls_005"><span class="cls_005">(</span><span class="cls_012">Supplementary Fig. 2e</span><span class="cls_005">). The plate was then centrifuged at 100 x g (max RCF) for 6 minutes at 4°C.</span></div>
<div style="position:absolute;left:54.00px;top:551.04px" class="cls_005"><span class="cls_005">Fifty (50) µL of supernatant was aspirated and cells were resuspended with 100 µL of Fc block (15</span></div>
<div style="position:absolute;left:54.00px;top:578.64px" class="cls_005"><span class="cls_005">µg/mL final concentration) the total volume of which was determined using the spreadsheet</span></div>
<div style="position:absolute;left:54.00px;top:606.24px" class="cls_005"><span class="cls_005">(</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">). Cells were blocked on ice for 5 minutes before being centrifuged at 100 x g</span></div>
<div style="position:absolute;left:54.00px;top:633.84px" class="cls_005"><span class="cls_005">(max RCF) for 5 minutes at 4°C. Fifty (50) µL of cell supernatant was aspirated. The 80 experimental</span></div>
<div style="position:absolute;left:54.00px;top:661.44px" class="cls_005"><span class="cls_005">tissue samples were resuspended with 91 µL of the combined target antibodies from the tube labeled</span></div>
<div style="position:absolute;left:54.00px;top:689.04px" class="cls_005"><span class="cls_005">“cocktail”. The 100 µL volumes in the 12-well V-bottom reagent reservoir were added to the respective</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">50</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:40100px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-51.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">single-color compensation control wells; CD45 isotype antibodies were added to the “ISO” well. Cells</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">in the “UNS” and “FVD” wells were resuspended with 91 µL of Brilliant Stain Buffer. The microplate</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">was allowed to incubate on ice in the dark for 15 minutes before adding 100 µL of DPBS to each well</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">and mixing thoroughly with a multichannel pipette. The plate was then centrifuged at 100 x g (max</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">RCF) for 5 minutes at 4°C followed by aspirating 191 µL from each well. Two-hundred (200) µL of</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">DPBS were added to each well using a multichannel pipette and mixed thoroughly by pipetting. The</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">plate was centrifuged at 100 x g (max RCF) for 5 minutes at 4°C followed by a 200 µL aspiration of</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">supernatant. Fixable viability dye was diluted 1.5:1,000 in 1X DPBS in the 15 mL conical tube labeled</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">“FVD” according to the spreadsheet (</span><span class="cls_012">Supplementary Table 2</span><span class="cls_005">). One-hundred (100) µL of the diluted</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">FVD were added to each well (a 1:1,000 final dilution) except for the “UNS”, which was resuspended</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">with 100 µL of DPBS only. Cells were incubated on ice in the dark for 30 minutes. One-hundred (100)</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">µL of DPBS were then added to each well and with a multichannel pipette and pipetted thoroughly to</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">wash. The plate was centrifuged at 100 x g (max RCF) for 5 minutes at 4°C followed by a 200 µL</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">aspiration, addition of 200 µL of DPBS using a multichannel pipette, and through mixing. The plate was</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">centrifuged at 100 x g (max RCF) for 5 minutes at 4°C followed by a 200 µL aspiration.</span></div>
<div style="position:absolute;left:90.00px;top:485.04px" class="cls_005"><span class="cls_005">One-hundred (100) µL of a fixation/permeabilization solution were next added to each well and</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">immediately resuspended with a multichannel pipette to prevent cell-to-cell crosslinking. Cells were</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">incubated on ice in the dark for 20 minutes followed by the addition of 100 µL of flow buffer and</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">thorough mixing with a multichannel pipette. The plate was centrifuged at 100 x g (max RCF) for 5</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">minutes at 4°C followed by a 200 µL aspiration and resuspension with 200 µL of flow buffer. A</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">Microseal ‘F’ foil seal was applied to the top of the 96-well microplate to prevent dehydration, wrapped</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">in aluminum foil to block light, and stored at 4°C prior to data acquisition by flow cytometry.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">51</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:40902px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-52.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">PMT Calibration</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">Detection channel PMT voltages were calibrated such that the signal intensity distribution corresponding</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">to background autofluorescence of FVD-labeled splenocytes was on scale and to the left of center. FVD-</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">labeled single-color compensation control splenocytes were run to verify that the assigned PMT voltages</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">were compatible with the dynamic range of each immunomarker’s expression profile (i.e. that</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">immunopositive cells were on scale). To prevent downstream compensation values from exceeding</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">100%, optical spillover of each single-color compensation control into off-target detection channels was</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">checked to ensure that peak signal intensity occurred in the target detection channel. Sphero Rainbow</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">Fluorescent Particles (single-positive beads) were run to predefine tolerability ranges for laser intensity,</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">stability, and alignment before initiating our study so that changes in laser emission power could be</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">monitored and accounted for between successive data acquisitions to prevent run-to-run variation.</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">Single-positive beads were gated according to the following strategy:</span></div>
<div style="position:absolute;left:72.00px;top:403.20px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  FSC-A vs. SSC-A: on single-positive beads</span></div>
<div style="position:absolute;left:72.00px;top:431.52px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  biexponential histograms of all detection channels</span></div>
<div style="position:absolute;left:108.00px;top:459.12px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  An interval gate was placed around the peak in each detection channel to define a narrow</span></div>
<div style="position:absolute;left:126.00px;top:486.72px" class="cls_005"><span class="cls_005">tolerability range to calibrate to between runs.</span></div>
<div style="position:absolute;left:54.00px;top:552.00px" class="cls_012"><span class="cls_012">Data Acquisition</span></div>
<div style="position:absolute;left:54.00px;top:579.60px" class="cls_005"><span class="cls_005">Cytometer setup &amp; tracking was performed using FACSDiva CS&amp;T research beads to optimize and</span></div>
<div style="position:absolute;left:54.00px;top:607.20px" class="cls_005"><span class="cls_005">standardize instrument performance prior to each data acquisition. The 96-well V-bottom plate</span></div>
<div style="position:absolute;left:54.00px;top:634.80px" class="cls_005"><span class="cls_005">containing the 80 immunolabeled experimental tissue samples and 16 optical controls was then loaded</span></div>
<div style="position:absolute;left:54.00px;top:662.40px" class="cls_005"><span class="cls_005">into a BD HTS affixed to a BD LSR II SORP flow cytometer. The gating strategy used at each</span></div>
<div style="position:absolute;left:54.00px;top:690.00px" class="cls_005"><span class="cls_005">acquisition was as follows:</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">52</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:41704px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-53.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:72.00px;top:72.00px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:72.00px" class="cls_005"><span class="cls_005">FSC-A vs. SSC-A: on all events (minus RBCs/debris)</span></div>
<div style="position:absolute;left:72.00px;top:100.56px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:100.56px" class="cls_005"><span class="cls_005">SSC-H vs. SSC-W: doublet discriminator</span></div>
<div style="position:absolute;left:72.00px;top:128.88px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:128.88px" class="cls_005"><span class="cls_005">FSC-H vs. FSC-W: doublet discriminator</span></div>
<div style="position:absolute;left:72.00px;top:157.44px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:157.44px" class="cls_005"><span class="cls_005">BUV395-A (FVD) (detected with 355 nm laser off of a 450/50 band pass filter) vs. FSC-A: on</span></div>
<div style="position:absolute;left:90.00px;top:184.80px" class="cls_005"><span class="cls_005">FVD negative cells (i.e. viable cells)</span></div>
<div style="position:absolute;left:72.00px;top:213.36px" class="cls_020"><span class="cls_020">•</span></div>
<div style="position:absolute;left:90.00px;top:213.36px" class="cls_005"><span class="cls_005">biexponential histograms of all detection channels</span></div>
<div style="position:absolute;left:108.00px;top:240.96px" class="cls_022"><span class="cls_022">o</span><span class="cls_021"> </span><span class="cls_005">  laser line, band pass filter, long pass filter, antibody detected:</span></div>
<div style="position:absolute;left:144.00px;top:268.56px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:268.56px" class="cls_005"><span class="cls_005">405 nm, 525/50, 505, V500-CD45</span></div>
<div style="position:absolute;left:144.00px;top:296.16px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:296.16px" class="cls_005"><span class="cls_005">488 nm, 710/50, 690, PerCP/Cy5.5-CD45R/B220</span></div>
<div style="position:absolute;left:144.00px;top:323.76px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:323.76px" class="cls_005"><span class="cls_005">355 nm, 740/35, 690, BUV737-CD11b</span></div>
<div style="position:absolute;left:144.00px;top:351.36px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:351.36px" class="cls_005"><span class="cls_005">594 nm, 660/20, 640, Alexa Fluor 647-CD11c</span></div>
<div style="position:absolute;left:144.00px;top:378.96px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:378.96px" class="cls_005"><span class="cls_005">488 nm, 525/50, 505, Alexa Fluor 488-CD3ε</span></div>
<div style="position:absolute;left:144.00px;top:406.56px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:406.56px" class="cls_005"><span class="cls_005">405 nm, 670/35, 635, BV605-CD4</span></div>
<div style="position:absolute;left:144.00px;top:434.16px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:434.16px" class="cls_005"><span class="cls_005">488 nm, 780/60, 755, PE/Cy7-CD49b</span></div>
<div style="position:absolute;left:144.00px;top:461.76px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:461.76px" class="cls_005"><span class="cls_005">488 nm, 610/20, 600, PE-CF594-CD8α</span></div>
<div style="position:absolute;left:144.00px;top:489.36px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:489.36px" class="cls_005"><span class="cls_005">488 nm, 575/26, 505, PE-F4/80</span></div>
<div style="position:absolute;left:144.00px;top:516.96px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:516.96px" class="cls_005"><span class="cls_005">594 nm, 780/60, 735, APC/Cy7-Ly6C</span></div>
<div style="position:absolute;left:144.00px;top:544.56px" class="cls_020"><span class="cls_020">§</span></div>
<div style="position:absolute;left:162.00px;top:544.56px" class="cls_005"><span class="cls_005">405 nm, 780/60, 750, BV711-Ly6G</span></div>
<div style="position:absolute;left:54.00px;top:582.00px" class="cls_005"><span class="cls_005">Samples were run in the following order (</span><span class="cls_012">see Supplementary Fig. 2e for plate details</span><span class="cls_005">):</span></div>
<div style="position:absolute;left:85.50px;top:609.60px" class="cls_005"><span class="cls_005">1.  SP beads (PRE): to check that PMT voltages were within previously defined tolerability</span></div>
<div style="position:absolute;left:103.50px;top:637.20px" class="cls_005"><span class="cls_005">ranges prior to data acquisition.</span></div>
<div style="position:absolute;left:85.50px;top:664.80px" class="cls_005"><span class="cls_005">2.  unstained control splenocytes (UNS)</span></div>
<div style="position:absolute;left:85.50px;top:692.40px" class="cls_005"><span class="cls_005">3.  unstained control splenocytes labeled with FVD (FVD)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">53</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:42506px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-54.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:85.50px;top:71.04px" class="cls_005"><span class="cls_005">4.  control splenocytes labeled with FVD and CD45 isotype control antibodies (ISO)</span></div>
<div style="position:absolute;left:85.50px;top:98.64px" class="cls_005"><span class="cls_005">5.  single-color compensation controls stained with FVD (control splenocytes were used for each</span></div>
<div style="position:absolute;left:103.50px;top:126.24px" class="cls_005"><span class="cls_005">antibody except CD49b, in which case WBCs were used due to the increased abundance of</span></div>
<div style="position:absolute;left:103.50px;top:152.40px" class="cls_005"><span class="cls_005">CD49b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> cells in blood)</span></div>
<div style="position:absolute;left:85.50px;top:181.44px" class="cls_005"><span class="cls_005">6.  experimental samples (16 mice x 5 tissue samples = 80 total)</span></div>
<div style="position:absolute;left:85.50px;top:209.04px" class="cls_005"><span class="cls_005">7.  SP beads (POST): to check that PMT voltages remained stable over the acquisition period</span></div>
<div style="position:absolute;left:103.50px;top:236.64px" class="cls_005"><span class="cls_005">(fluidic anomalies can impact laser delay stability)</span></div>
<div style="position:absolute;left:54.00px;top:274.32px" class="cls_005"><span class="cls_005">Raw data were exported as FCS3.0 files upon completion of each data acquisition session.</span></div>
<div style="position:absolute;left:54.00px;top:329.52px" class="cls_012"><span class="cls_012">Spectral Deconvolution and Data Cleanup</span></div>
<div style="position:absolute;left:54.00px;top:357.12px" class="cls_005"><span class="cls_005">FlowJo software was used to spectrally deconvolve raw flow cytometry data. Data from 15 of the 16</span></div>
<div style="position:absolute;left:54.00px;top:384.72px" class="cls_005"><span class="cls_005">optical control wells (UNS was excluded) were imported into the </span><span class="cls_019">compensation group</span><span class="cls_005"> of a new FlowJo</span></div>
<div style="position:absolute;left:54.00px;top:412.32px" class="cls_019"><span class="cls_019">workspace</span><span class="cls_005">. CD45 signal intensity distributions were invariably unimodal making it difficult to</span></div>
<div style="position:absolute;left:54.00px;top:439.92px" class="cls_005"><span class="cls_005">objectively define a compensation gate for the CD45 single-positive control alone. Thus, the data</span></div>
<div style="position:absolute;left:54.00px;top:467.52px" class="cls_005"><span class="cls_005">corresponding to well E11 (ISO) was merged with that of F9 (CD45 single-color compensation control)</span></div>
<div style="position:absolute;left:54.00px;top:495.12px" class="cls_005"><span class="cls_005">using FlowJo’s </span><span class="cls_019">concatenate</span><span class="cls_005"> feature. The merged data were saved as a new FCS3.0 file and imported</span></div>
<div style="position:absolute;left:54.00px;top:522.72px" class="cls_005"><span class="cls_005">into the </span><span class="cls_019">compensation group</span><span class="cls_005"> of the current </span><span class="cls_019">workspace</span><span class="cls_005">. The original CD45 single-positive compensation</span></div>
<div style="position:absolute;left:54.00px;top:550.32px" class="cls_005"><span class="cls_005">control and ISO samples were then deleted from the </span><span class="cls_019">workspace</span><span class="cls_005">. The merging procedure resulted in a</span></div>
<div style="position:absolute;left:54.00px;top:577.68px" class="cls_005"><span class="cls_005">bimodal distribution and the ability to objectively define a CD45 compensation gate between its two</span></div>
<div style="position:absolute;left:54.00px;top:605.28px" class="cls_005"><span class="cls_005">peaks. The merged CD45 file plus the other 10 single-color compensation controls and the FVD well</span></div>
<div style="position:absolute;left:54.00px;top:632.88px" class="cls_005"><span class="cls_005">(E10)—which served as the compensation control for the for the FVD itself—were gated for viable</span></div>
<div style="position:absolute;left:54.00px;top:660.48px" class="cls_005"><span class="cls_005">singlets according to the following strategy:</span></div>
<div style="position:absolute;left:72.00px;top:689.04px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  FSC-A vs. SSC-A: on all events (minus RBCs/debris)</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">54</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:43308px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-55.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:72.00px;top:72.00px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  SSC-H vs. SSC-W: doublet discriminator</span></div>
<div style="position:absolute;left:72.00px;top:100.56px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  FSC-H vs. FSC-W: doublet discriminator</span></div>
<div style="position:absolute;left:72.00px;top:128.88px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  BUV395-A (FVD) vs. FSC-A (viewed as a contour plot at the 2% level): on FVD negative cells</span></div>
<div style="position:absolute;left:90.00px;top:156.48px" class="cls_005"><span class="cls_005">(i.e. viable cells)</span></div>
<div style="position:absolute;left:72.00px;top:185.04px" class="cls_020"><span class="cls_020">•</span><span class="cls_021"> </span><span class="cls_005">  backgate to FSC-A vs. SSC-A: on total viable singlets (or a subset for scarce populations)</span></div>
<div style="position:absolute;left:54.00px;top:222.48px" class="cls_005"><span class="cls_005">Once gated, viable singlets of each compensation control sample were visualized as histograms in their</span></div>
<div style="position:absolute;left:54.00px;top:250.08px" class="cls_005"><span class="cls_005">respective detection channel. Signal intensity values of each histogram were bisected at the interface of</span></div>
<div style="position:absolute;left:54.00px;top:277.68px" class="cls_005"><span class="cls_005">the penultimate and ultimate modes of each signal intensity distribution using FlowJo’s </span><span class="cls_019">bisector tool</span><span class="cls_005">. Its</span></div>
<div style="position:absolute;left:54.00px;top:305.28px" class="cls_019"><span class="cls_019">compensation</span><span class="cls_005"> tool was then opened and the subsets to the left and right of the bisection were dragged</span></div>
<div style="position:absolute;left:54.00px;top:332.88px" class="cls_005"><span class="cls_005">into fields labeled </span><span class="cls_019">negative</span><span class="cls_005"> and positive, respectively. The process was repeated for all 11 antibodies</span></div>
<div style="position:absolute;left:54.00px;top:360.48px" class="cls_005"><span class="cls_005">plus the FVD. Next, a new </span><span class="cls_019">group</span><span class="cls_005"> in the </span><span class="cls_019">workspace</span><span class="cls_005"> window was generated and titled “cocktails” to</span></div>
<div style="position:absolute;left:54.00px;top:388.08px" class="cls_005"><span class="cls_005">which the 80 experimental samples were imported. The finalized compensation matrix was then applied</span></div>
<div style="position:absolute;left:54.00px;top:415.68px" class="cls_005"><span class="cls_005">to the “cocktails” group. Viable singlets from each experimental sample were gated in the same way as</span></div>
<div style="position:absolute;left:54.00px;top:443.28px" class="cls_005"><span class="cls_005">the single-color compensation controls according to the first 4 steps of the gating strategy outlined above</span></div>
<div style="position:absolute;left:54.00px;top:470.88px" class="cls_005"><span class="cls_005">then exported as new FCS3.0 files; the aggregate of such files from each of the study’s 3 time points</span></div>
<div style="position:absolute;left:54.00px;top:498.48px" class="cls_005"><span class="cls_005">served as input into our computational data analysis algorithm.</span></div>
<div style="position:absolute;left:54.00px;top:553.68px" class="cls_012"><span class="cls_012">Weighted Random Sampling</span></div>
<div style="position:absolute;left:54.00px;top:581.28px" class="cls_005"><span class="cls_005">A 10 million cell weighted random sample (WRS) was derived from the cleaned flow cytometry data to</span></div>
<div style="position:absolute;left:54.00px;top:608.88px" class="cls_005"><span class="cls_005">help balance the number of cells per tissue sample. Sample weights were defined per tissue per cell</span></div>
<div style="position:absolute;left:54.00px;top:636.48px" class="cls_005"><span class="cls_005">according to the formula [(1/ω) x (1/N</span><span class="cls_023"><sub>i</sub></span><span class="cls_005">)] where ω was the number of unique tissue types (5 in this cases)</span></div>
<div style="position:absolute;left:54.00px;top:662.64px" class="cls_005"><span class="cls_005">and N</span><span class="cls_023"><sub>i</sub></span><span class="cls_005"> was the number of events associated with the </span><span class="cls_019">i</span><span class="cls_006"><sup>th</sup></span><span class="cls_005"> tissue where </span><span class="cls_019">i</span><span class="cls_005"> took the categorical values blood,</span></div>
<div style="position:absolute;left:54.00px;top:691.68px" class="cls_005"><span class="cls_005">marrow, nodes, spleen, thymus.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">55</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:44110px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-56.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_012"><span class="cls_012">Histogram Gating</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">Cleaned flow cytometry data (i.e. compensated viable singlets) were displayed as 2,640 histograms (240</span></div>
<div style="position:absolute;left:54.00px;top:152.40px" class="cls_005"><span class="cls_005">samples x 11 antibody detection channels) plotted on a Logicle</span><span class="cls_006"><sup>70</sup></span><span class="cls_005"> scale and formatted as scalable vector</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">graphics (SVGs) in the context of a scrolling HTML table viewable with a web browser. A KDE of the</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">signal intensity distribution of cells from the FVD well (i.e. compensated unstained viable splenocytes)</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">was superimposed on each to identify signal intensity values corresponding to autofluorescence</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">background. This allowed for the rapid recording of bisection point at the interface of the first</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">(autofluorescence) and second (true signal) peaks for all histograms in a .TXT file. A vertical line was</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">then programmatically rendered at the location of each bisection point and again visualized as a</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">scrolling HTML table on the web for confirmation or refinement. The numerical value of each bisection</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">point was then Logicle-transformed and subtracted from the Logicle-transformed signal intensity values</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">of its corresponding histogram (i.e. Logicle[data point</span><span class="cls_006"><sub>i</sub></span><span class="cls_005">]) - Logicle[bisection point]) resulting in the</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">Logicle-transformed bisection point assuming the numerical value of zero and background signal</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">intensity values becoming negative valued. Since the 5 lymphoid tissue types predominately consisted of</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">immune cells, the CD45 signal intensity distributions were invariable unimodal with no discernable</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">local minima. Thus, for each time point and tissue combination, a common CD45 bias was curated by</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">pooling the corresponding samples, computing Q25 - [1.5 * [Q75 - Q25]], then rounding to the nearest</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">multiple of 5. (where Q25 and Q75 were the first and third quartiles of the Logicle-transformed data,</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">respectively).</span></div>
<div style="position:absolute;left:90.00px;top:621.60px" class="cls_005"><span class="cls_005">Immunophenotypes interpreted as CD49b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> granulocytes were identified in the blood of both</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">control and GBM mice. Since granulocyte interaction with CD49b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> platelets is a described phenomenon</span></div>
<div style="position:absolute;left:54.00px;top:676.80px" class="cls_005"><span class="cls_005">thought to represent a physiologic process required for neutrophil extracellular trap formation</span><span class="cls_006"><sup>71,72</sup></span><span class="cls_005">, we</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">56</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:44912px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-57.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">considered these cells as a likely artifact of residual platelets present within blood samples. In taking a</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">conservative approach to correct for this discrepancy, cell status of the CD49b immunomarkers was only</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">considered in cases where the immunophenotype was otherwise consistent with NK cells (e.g. CD45</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">,</span></div>
<div style="position:absolute;left:54.00px;top:152.40px" class="cls_005"><span class="cls_005">CD49b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">, CD11b</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">). Thus, a Boolean truth value of zero for the CD49b channel was uniformly applied to</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">all cells whose immunophenotype was not matching NK cells.</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_012"><span class="cls_012">t-CyCIF</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">The reader is referred to </span><A HREF="http://www.cycif.org/">www.cycif.org</A> for detailed instruction t-CyCIF methodology<span class="cls_006"><sup>54</sup></span><span class="cls_005">. Briefly, 5µm-</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">thick </span><span class="cls_016">FFPE </span><span class="cls_005">tissue sections of the tumor-ipsilateral brain hemisphere of a C57BL/6J mouse engrafted</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">with 3x10</span><span class="cls_006"><sup>4</sup></span><span class="cls_005"> GL261 cells 36-days prior was serially scanned using 2x2 binning with a CyteFinder slide</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">scanning fluorescence microscope (RareCyte, Seattle, WA, USA) equipped with a 40X (0.6NA) long</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">working distance objective on each of 4 imaging cycles. Before immunolabeling with the first set of 3</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">fluorophore-conjugated antibodies, the tissue was counterstained with DAPI at a 1:3,000 dilution of a 10</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">mg/mL stock in 1x PBS for 15 minutes at RT then blocked with Odyssey Blocking Buffer at RT for 1</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">hour to limit non-specific antibody binding. Blocked slides were then imaged to document background</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">autofluorescence prior to immunolabeling. Antibody incubations were performed at 4°C overnight in an</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">opaque and humidified chamber. After immunolabeling, slides were washed 3 times with 1x PBS,</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">temporarily coverslipped in 1x PBS containing 10% glycerol, and re-imaged. Immediately after</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">imaging, slides were de-coverslipped by vertical submersion in Coplin jars containing 1x PBS until the</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">coverslip spontaneously fell way from the slide. Antibody fluorescence was deactivated by submersion</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">of the slide in a 3% H</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">O</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> solution containing 20 mM NaOH in 1x PBS for 2 hours at RT in the presence</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">of intense fluorescent light. Following fluorophore deactivation, slides were rinsed briefly in 1x PBS,</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">then subject to the next round of immunolabeling.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">57</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:45714px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-58.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:90.00px;top:71.04px" class="cls_005"><span class="cls_005">Upon completion of the cyclic imaging procedure, autofluorescence background was</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">computationally subtracted on a pixel-by-pixel basis using ImageJ. The final 12-plex mosaic image was</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">generated by aligning (or registering) the 168 400x300µm imaging fields acquired during each imaging</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">cycle on their DAPI counterstained nuclei using ImageJ’s Multistack Registration Plugin. Registered</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">images were compiled into multi-image stacks and segmented based on the DAPI signal acquired during</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">the last imaging cycle by applying a uniform threshold on the DAPI channel across all images and then</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">converting this signal into binary regions of interest (ROIs) using the ImageJ’s </span><span class="cls_019">Analyze Particles</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">function. ROIs corresponding to single-cell nuclei were enlarged by 3 pixels to cover immune cell</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">cytoplasm and membrane. Respective ROIs where overlaid on each image to obtain integrated</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">fluorescence signal intensity data on each cell for across all 12 immunofluorescence channels.</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_012"><span class="cls_012">RNA-seq</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">The spleens of five 12-week-old tumor-naïve C57BL/6J mice were immunodepleted of B cells using</span></div>
<div style="position:absolute;left:54.00px;top:428.40px" class="cls_005"><span class="cls_005">MojoSort mouse CD19 nanobeads (Biolegend, Cat. No. 480002) before FACS-purifying B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD3ε</span><span class="cls_006"><sup>+</sup></span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">CD8α</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> (i.e. B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells) and B220</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> CD3ε</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8α</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> (i.e. CD8T cells) into respective 15 ml conical</span></div>
<div style="position:absolute;left:54.00px;top:483.60px" class="cls_005"><span class="cls_005">tubes containing ice-cold with 0.5% BSA DPBS. Approximately 1x10</span><span class="cls_006"><sup>6</sup></span><span class="cls_005"> cells of each type were</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">centrifuged at 400 x g (max RCF) for 10 minutes at 4°C and resuspended in RLT lysis buffer prior to</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">total RNA extraction with Qiagen’s RNeasy mini kit using the optional DNase treatment (Qiagen, Cat.</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">No. 74104). Total RNA was then used to prepare sequencing libraries of the coding transcriptome using</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">Illumina’s TruSeq stranded mRNA protocol. Libraries were sequenced by synthesis on a Nextseq 500</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">instrument. Fastq files were processed on a high-performance computer cluster using a standardized data</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">analysis pipeline involving the analysis programs FASTQC, STAR, Salmon, featureCounts, EdgeR,</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">58</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:46516px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-59.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">Kallisto. Kallisto transcript abundance files were then analyzed on a desktop computer using the Sleuth</span></div>
<div style="position:absolute;left:54.00px;top:97.20px" class="cls_005"><span class="cls_005">RNA-seq analysis program</span><span class="cls_006"><sup>49</sup></span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_012"><span class="cls_012">Software</span></div>
<div style="position:absolute;left:67.50px;top:181.44px" class="cls_005"><span class="cls_005">•  FACSDiva (version 8.0)</span></div>
<div style="position:absolute;left:67.50px;top:209.04px" class="cls_005"><span class="cls_005">•  FlowJo (version 10.3.0)</span></div>
<div style="position:absolute;left:67.50px;top:236.64px" class="cls_005"><span class="cls_005">•  Python (version 3.6.1)</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_012"><span class="cls_012">Statistics</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">Statistical tests were performed using SciPy.stats, a Python-based library of validated statistical</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">functions. Statistical tests used throughout this paper are described at their point of reference. Two-</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">sample hypothesis tests were considered statistically significant if their FDR adjusted p-value (or q-</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">value) reached a cutoff of </span><span class="cls_025">&lt;</span><span class="cls_005"> 0.05.</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_012"><span class="cls_012">Data and Code Availability</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">SYLARAS source code and the cleaned GBM flow cytometry dataset are freely available for download</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">at </span><A HREF="http://www.sylaras.org./">www.sylaras.org.</A> </div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">59</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:47318px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-60.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">FIGURE LEGENDS</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_012"><span class="cls_012">Figure 1 </span><span class="cls_005">Workflow for systemic immunophenotyping of GBM-bearing mice by 12-color flow</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">cytometry. </span><span class="cls_019">(1)</span><span class="cls_005"> GBM cell suspension or vehicle alone was stereotactically-injected into the striata of age-</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">matched cohorts of 24 mice. </span><span class="cls_019">(2)</span><span class="cls_005"> Lymphoid tissues of 8 mice per treatment group were harvested at days</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">7, 14, and 30-days after the engraftment procedure. </span><span class="cls_019">(3)</span><span class="cls_005"> Tissues were processed into single-cell</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">suspensions and plated in a 96 well V-bottom plate (see Supplementary Fig. 1h for details on optical</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">controls). </span><span class="cls_019">(4)</span><span class="cls_005"> Cells were immunolabeled with a cocktail of 11 fluorophore-conjugated antibodies</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">followed by incubation with a fixable viability dye (FVD). </span><span class="cls_019">(5)</span><span class="cls_005"> Single-cell data were acquired by high-</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">throughput flow cytometry. </span><span class="cls_019">(6)</span><span class="cls_005"> Raw data files were spectrally deconvolved and gated for viable singlets.</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_019"><span class="cls_019">(7)</span><span class="cls_005"> Cleaned data underwent a computer-assisted unidimensional gating procedure (see Fig. 2 for details)</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">prior to being computationally analyzed by the SYLARAS data analysis pipeline.</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_012"><span class="cls_012">Figure 2 </span><span class="cls_005">Systematic cell subset identification via computer-assisted unidimensional gating. (</span><span class="cls_012">a</span><span class="cls_005">) Logicle-</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">displayed signal intensity histograms for two hypothetical antibody detection channels (IM1=green,</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">IM2=blue). (</span><span class="cls_012">b</span><span class="cls_005">) Tabular data structure for recording histogram gate values on a replicate, tissue,</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">condition, time point, and detection channel basis. (</span><span class="cls_012">c</span><span class="cls_005">) Histograms shown in panel (a) after SYLARAS</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">renders a vertical line at the gate value for rapid gate confirmation or refinement. (</span><span class="cls_012">d</span><span class="cls_005">) Histograms shown</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">in panels (a and c) after programmatic subtraction of their gate values from the values in their</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">corresponding signal intensity distribution. Note how the procedure causes the zero point of each</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">distribution to become zero-centered at the signal-to-noise interface.</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_012"><span class="cls_012">Figure 3 </span><span class="cls_005">Mouse lymphoid tissue architecture characterized in 30 cellular immunophenotypes. (</span><span class="cls_012">a</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">Sunburst plot of the hierarchical organization of a dictionary-based immunophenotype classification</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">60</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:48120px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-61.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">scheme. Standard cell classes are represented by the plot’s inner wedges whose expressed antigens are</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">indicated at the center of the plot (except for DC, ISPT, NK, Precursor, LTi to avoid text overlap). Outer</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">wedges represent variants of each landmark class whose aliases are prefixed according to conventional</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">+/- immunophenotyping notation. (</span><span class="cls_012">b</span><span class="cls_005">) Heatmap diagram of the 11-dimensional Boolean vector</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">associated with all 30 immunophenotypes characterized in this study. Also shown is the percentage of</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">the 10M cell WRS accounted for by each immunophenotype (left) and their presumed cell type identity</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">(right) (</span><span class="cls_012">c</span><span class="cls_005">) Immunophenotype FSC and SSC distributions shown for control (Ctrl, blue) and GBM-</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">burdened (GBM, green) mice. FSC low/high cutoff = 35,000; SSC low/high cutoff = 97,000.</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">Immunophenotypes are shown in ascending rank order from left-to-right according to the median value</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">of the control group. Box plot elements: horizontal line, median; box limits, first to third quartile (Q1 to</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">Q3); whiskers, from Q1-1.5 x interquartile range (IQR) to Q3 + 1.5 x IQR; diamond points, outliers. (</span><span class="cls_012">d</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">Bar and stair plots showing the individual (bars) and cumulative (steps) contribution of successive</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">immunophenotypes in accounting for the percentage of each of 5 mouse lymphoid tissue types. (</span><span class="cls_012">e</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">Stacked bar chart showing the fraction of each immunophenotype across 5 mouse lymphoid tissue types</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">(top x-axis) listed in ascending rank order along the y-axis according to their information entropy</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">(superimposed dashed line, bottom x-axis). (</span><span class="cls_012">f</span><span class="cls_005">) Average antigen expression level of 30 Phenograph</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">clusters obtained after running the algorithm using k=20 nearest neighbors and the Euclidian distance</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">metric on an 8K cell subsample of the full dataset. Also shown is the percentage of the subsample</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">accounted for by each cluster (left) and the information entropy associated with their cell type</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">composition as determined by our dictionary-based immunophenotype classification scheme (right). The</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">most populous dictionary immunophenotype accounting for the majority </span><span class="cls_025">&gt;</span><span class="cls_005">50% of each cluster are</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">indicated along with a categorical score of agreement between knowledge-based and data-driven cell</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">subset identification.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">61</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:48922px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-62.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_012"><span class="cls_012">Figure 4 </span><span class="cls_005">SYLARAS reveals GBM-induced perturbation in the frequency of peripheral immune cell</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">subsets hitherto undescribed in the disease. (</span><span class="cls_012">a</span><span class="cls_005">) Weighted log</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> fold-change in time and tissue-specific</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">immunophenotype frequency between GBM-bearing and age-matched, mock-engrafted control mice</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">(GBM/Ctrl). Icons denote one of three different q-value ranges. (</span><span class="cls_012">b</span><span class="cls_005">) Mean difference in the absolute</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">magnitude of time and tissue-specific immunophenotype frequency between GBM-bearing and age-</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">matched, mock-engrafted control mice (GBM - Ctrl). (</span><span class="cls_012">c</span><span class="cls_005">) SYLARAS dashboard of the B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">immunophenotype revealing 10 attributes: </span><span class="cls_019">(1)</span><span class="cls_005"> dictionary alias; </span><span class="cls_019">(2)</span><span class="cls_005"> major immune lineage; </span><span class="cls_019">(3)</span><span class="cls_005"> condition-</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">specific FSC and SSC distributions; </span><span class="cls_019">(4)</span><span class="cls_005"> Logicle-transformed immunomarker signal intensity</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">distributions; </span><span class="cls_019">(5 and 6)</span><span class="cls_005"> time and tissue-specific mean difference (left) and log</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> fold-change (right)</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">between GBM-burdened and control mice (n=8 mice/group/tissue/time point Icons denote one of three</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">different q-value ranges; </span><span class="cls_019">(7)</span><span class="cls_005"> color-coded tissue distribution (the same as in attributes 5 and 6); </span><span class="cls_019">(8)</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">fraction of the 10M cell WRS accounted for by the immunophenotype; </span><span class="cls_019">(9)</span><span class="cls_005"> percentage of 5 lymphoid</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">tissues accounted for by the immunophenotype across the study’s 48 mice; </span><span class="cls_019">(10)</span><span class="cls_005"> priority rank—a metric</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">of the measurability and statistical significance of GBM-induced effect size calculated as [1-q]</span><span class="cls_006"><sup>3</sup></span><span class="cls_005"> x |R + Δ|</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">+ σ) where q = q-value, R = log</span><span class="cls_006"><sub>2</sub></span><span class="cls_005">(0.01 + mean GBM % / 0.01 + mean Ctrl %), Δ = (mean GBM % -</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">mean Ctrl %), and σ = number of time points a given cell type was involved in a statistically significant</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">difference between the two treatment groups.</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_012"><span class="cls_012">Figure 5</span><span class="cls_005"> Analysis of mouse-to-mouse variation in immunophenotype frequency provides network-level</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">insight into GBM’s immunoregulatory control over the peripheral immune system. (</span><span class="cls_012">a</span><span class="cls_005">) Score plot of the</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">first 2 PCs of a PCA where the input was a n x m data matrix of the percentage of the study’s 240 tissue</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">samples (rows) accounted for by each of 30 immunophenotypes (columns). Circles highlight outlier</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">62</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:49724px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-63.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">samples from late-stage GBM-burdened mouse 3. (</span><span class="cls_012">b</span><span class="cls_005">) Clustermaps showing the log</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> ratio between the</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">tissue-specific immunophenotype frequency for each late-stage GBM-burdened mouse (columns) and</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">the median value for the group of 8 mice. Significant outliers indicated by red text. Clustering was</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">performed using the unweighted pair group method with arithmetic mean (UPGMA) linkage and</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">Euclidean distance. (</span><span class="cls_012">c</span><span class="cls_005">) Percentage of Ly6C</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> PMN cells in the bone marrow of each of the study’s 48</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">mice. Late-stage GBM-burdened mouse 3 is indicated. (</span><span class="cls_012">d</span><span class="cls_005">) </span><span class="cls_016">T-tests (computed across rows; experimental</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">conditions) and Spearman’s rank-order correlation (computed across columns; immunophenotypes)</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_016"><span class="cls_016">demonstrating that statistically significant co-variation</span><span class="cls_005"> among biological replicates of the same</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">experimental condition can arise independently of significant differences in population average values of</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">different experimental conditions</span><span class="cls_016">. </span><span class="cls_005">(</span><span class="cls_012">e</span><span class="cls_005">) Seven (7) statistically significant time and tissue-specific</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">Spearman’s rank-order correlations in immunophenotype frequency. Asterisk denotes the correlation</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">pair shown in the top row of panel (d). (</span><span class="cls_012">f</span><span class="cls_005">) Schematic diagram of correlation profile clustering. Positive</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">correlation = red, negative correlation = blue, identity correlation = grey. </span><span class="cls_016">Clustering tissue-specific</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_016"><span class="cls_016">immunophenotypes are presumptively </span><span class="cls_005">co-regulated while anti-correlated clusters are presumed to be</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">functionally-antagonistic. (</span><span class="cls_012">g</span><span class="cls_005">) Agglomerative hierarchical clustering of Spearman’s rank-order</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">correlation coefficients computed across biological replicates of control (left) and GBM-burdened</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">(right) mice regardless of time point. Clustering was performed using the UPGMA linkage algorithm</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">and Euclidean distance using </span><span class="cls_016">only immunophenotypes with a frequency &gt;0.1% across replicates</span><span class="cls_005">.</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">Clusters have been color-coded independently for in each treatment group. Members of GBM-burdened</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">cluster 3 are specified in order from top-to-bottom.</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_012"><span class="cls_012">Figure 6</span><span class="cls_005"> B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells are a transcriptionally-distinct subset of CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T lymphocytes. (</span><span class="cls_012">a</span><span class="cls_005">) Scatter</span></div>
<div style="position:absolute;left:54.00px;top:676.80px" class="cls_005"><span class="cls_005">plot stratifying CD8T, B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T, CD4T, and B cells in the the blood of the study’s 24 control mice</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">63</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:50526px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-64.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:69.60px" class="cls_005"><span class="cls_005">according to their expression of B220 and CD8α. (</span><span class="cls_012">b</span><span class="cls_005">) Percentage of CD8T cells and B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">in the thymus of the study’s 48 mice. (</span><span class="cls_012">c</span><span class="cls_005">) Heatmaps of the mean log</span><span class="cls_006"><sub>2</sub></span><span class="cls_005"> fold-change in CD8T (left) and</span></div>
<div style="position:absolute;left:54.00px;top:124.80px" class="cls_005"><span class="cls_005">B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cell (right) frequency between GBM-burdened and age-matched, mock-engrafted control</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">mice (n=8 mice/group/tissue/time point). Icons denote one of three different q-value ranges. (</span><span class="cls_012">d</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">Regression plots showing the frequency of B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells (x-axes) relative to CD8T cells (y-axes) in</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">the spleen of control (left column) and GBM (right column) mice during each of the study’s three time</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">points (rows). Data points represent 8 replicate mice. Regression plot elements: line, regression line;</span></div>
<div style="position:absolute;left:54.00px;top:262.80px" class="cls_005"><span class="cls_005">transparency, 95% confidence interval for the regression; R</span><span class="cls_006"><sup>2</sup></span><span class="cls_005">, coefficient of determination. (</span><span class="cls_012">e</span><span class="cls_005">) Gating</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_005"><span class="cls_005">strategy for the FACS-purification of B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> and B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005"> CD8T cells from the CD19-immunodepleted,</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">tumor-naïve, C57BL/6 mouse spleen. (</span><span class="cls_012">f</span><span class="cls_005">) Scree plot showing the percentage of explained variance</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">accounted for by individual (bars) and cumulative (stairs) PCs from a PCA performed on normalized</span></div>
<div style="position:absolute;left:54.00px;top:373.20px" class="cls_005"><span class="cls_005">mRNA transcript counts table of B220</span><span class="cls_006"><sup>hi</sup></span><span class="cls_005"> (n=2) and B220</span><span class="cls_006"><sup>lo</sup></span><span class="cls_005"> (n=3) CD8T cells. (</span><span class="cls_012">g</span><span class="cls_005">) PC1 vs. PC2 score plot.</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">Samples are color-coded by immunophenotype; biological replicates are indicated by number. (</span><span class="cls_012">h</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">mRNA transcripts with the top 5 contribution scores for PC1 (left) and PC2 (right). (</span><span class="cls_012">i</span><span class="cls_005">) Table of</span></div>
<div style="position:absolute;left:54.00px;top:456.00px" class="cls_005"><span class="cls_005">transcripts differently expressed between B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T and CD8T cells; transcripts of interest are</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">highlighted. b = beta-value (approximate fold-change).</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_012"><span class="cls_012">Figure 7 </span><span class="cls_005">B220+ CD8T cells are a morphologically and spatially-distinct subset of GBM-infiltrating</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">CD8</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T lymphocytes.</span><span class="cls_012"> </span><span class="cls_005">(</span><span class="cls_012">a</span><span class="cls_005">) Spatial coordinates of</span><span class="cls_016"> cells composing a 5-µm thick slice of the tumor-</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_016"><span class="cls_016">ipsilateral brain hemisphere of a GL261-bearing mouse </span><span class="cls_005">36-days after tumor engraftment. Tumor area is</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">outlined by the black dashed perimeter and characterized by high cell density compared to normal tissue</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">constituting the cerebral cortex. (</span><span class="cls_012">b</span><span class="cls_005">) Cell counts for the 7 most frequent immunophenotypes identified in</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">the late-stage GL261 GBM TME. An thumbnail example of a cell bearing each immunophenotype is</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">64</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:51328px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-65.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">shown. Antibody clones used in the immunolabeling are indicated in the banner across the top of the</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">panel. (</span><span class="cls_012">c</span><span class="cls_005">) Examples of CD8α</span><span class="cls_006"><sup> </sup></span><span class="cls_005">single-positive and B220/CD8α double-positive cells within the GL261</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">TME. Tile coordinates of each cell permit cross-referencing with the full-resolution PDF provided as</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">Supplementary Fig. 7. (</span><span class="cls_012">d</span><span class="cls_005">) Box plot distributions of Logicle-transformed CD8α signal intensity from</span></div>
<div style="position:absolute;left:54.00px;top:180.00px" class="cls_005"><span class="cls_005">CD8T (left) and B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells (right) in the blood of the study’s 24 control mice. (</span><span class="cls_012">e</span><span class="cls_005">) KDEs</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">(concentric black contour lines) of the cellular density for the 7 abundant immune subsets identified in</span></div>
<div style="position:absolute;left:54.00px;top:235.20px" class="cls_005"><span class="cls_005">the TME superimposed on the same scatter plot of vimentin</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> lineage</span><span class="cls_006"><sup>-</sup></span><span class="cls_005"> tumor cells colored-coded</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">according to cell density relating immune cell spatial distribution with tumor cell density. (</span><span class="cls_012">f</span><span class="cls_005">) Clustermap</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_005"><span class="cls_005">of the correlation in spatial distribution of the 7 abundant immunophenotype and tumor cells</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">comprising the TME. The minimal overlap observed between B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cells and CD8T cells is</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">indicated by the red box</span></div>
<div style="position:absolute;left:54.00px;top:400.80px" class="cls_012"><span class="cls_012">Figure 8 </span><span class="cls_005">Identification of a B220</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> CD8T cell analogue in human blood clinical GBM. (</span><span class="cls_012">a</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">Representative 20x image of normal human WBCs affixed to a glass microscope slide and</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">immunolabeled with anti-human antibodies targeting CD3δ (green), CD8α (red), and CD45R (blue). (</span><span class="cls_012">b</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">Counts of the top 6 most abundant cellular immunophenotypes present in the WBC preparation</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">exemplified in panel (a). (</span><span class="cls_012">c</span><span class="cls_005">) Low-resolution, 4-color mosaic micrograph of a human brain tumor TMA</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">showing (316) core biopsies (each 0.6mm in diameter) from 173 different human brain tumor patients</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">immunolabeled with antibodies targeting anti-human CD45R, CD3δ, and CD8α. (</span><span class="cls_012">d</span><span class="cls_005">) TMA metadata</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">map showing patient ID (# above each core), number of triple-positive cells (heatmap intensity), and</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">tissue status: primary tumor, recurrent tumor, or tonsil (positive controls for immunolabeling). (</span><span class="cls_012">e</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:649.20px" class="cls_005"><span class="cls_005">Example core imaged at 20x demonstrating the presence of a triple-positive CD45R</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">, CD3δ</span><span class="cls_006"><sup>+</sup></span><span class="cls_005">, CD8α</span><span class="cls_006"><sup>+</sup></span><span class="cls_005"> T</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">lymphocyte (white box and inset). CD3δ single-positive and CD8α/CD3δ double-positive cells are also</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">65</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:52130px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-66.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">indicated. (</span><span class="cls_012">f</span><span class="cls_005">)   Representative 40x micrographs of CD45R/CD3δ/CD8α</span><span class="cls_006"><sup> </sup></span><span class="cls_005">triple-positive T lymphocytes in</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">various core biopsies of the TMA shown in panel (c).</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_012"><span class="cls_012">Supplementary Figure 1 </span><span class="cls_005">Experimental optimization and quality control for 12-color</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">immunophenotyping by flow cytometry. (</span><span class="cls_012">a</span><span class="cls_005">) 8 point, 2-fold serial dilution titration curves for each</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">fluorophore-conjugated antibody used in the study. Splenocytes from 12-week-old female C57BL/6J</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">mice were used as cells for immunolabeling. The separation index (SI) was calculated as [MFI</span><span class="cls_006"><sub>pos</sub></span><span class="cls_005">-</span></div>
<div style="position:absolute;left:54.00px;top:264.24px" class="cls_005"><span class="cls_005">MFI</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">]/[[84%</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">-MFI</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005">]/0.995], where MFI</span><span class="cls_006"><sub>pos</sub></span><span class="cls_005"> = median fluorescence intensity of the first positive</span></div>
<div style="position:absolute;left:54.00px;top:290.40px" class="cls_005"><span class="cls_005">peak, MFI</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005"> = median fluorescence intensity of the autofluorescence peak, 84%</span><span class="cls_006"><sub>neg</sub></span><span class="cls_005"> = 84</span><span class="cls_006"><sup>th</sup></span><span class="cls_005"> percentile of</span></div>
<div style="position:absolute;left:54.00px;top:318.00px" class="cls_005"><span class="cls_005">the autofluorescence peak as previously described</span><span class="cls_006"><sup>69</sup></span><span class="cls_005">. Cubic splines were used to interpolate adjacent data</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">points. Antibody concentrations yielding peak separation indices are indicated by a green dot. Data</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">acquisition gating strategy: (FSC-A vs. SSC-A) → (SSC-H vs. SSC-W) → (FSC-H vs. FSC-W) →</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">(DAPI-A vs. FSC-A) → (specific antibody vs. count). (</span><span class="cls_012">b</span><span class="cls_005">) Signal intensity distributions of splenocytes</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">from 12-week-old female C57BL/6J mice immunolabeled with the 11 antibodies indicated in panel (a)</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">plus a fixable viability dye (FVD) as detected in the 12 optical channels of a BD LSR II flow cytometer.</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_005"><span class="cls_005">Pre- and post-compensation conditions are shown. Antibodies (rows) are color-coordinated with their</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">respective detection channel (columns). Histograms forming the left-to-right downward diagonal across</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">the matrix show the location of the respective channel’s compensation gate (blue/green interface). (</span><span class="cls_012">c</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:566.40px" class="cls_005"><span class="cls_005">Kaplan-Meier survival curve of 10 C57BL/6J mice intracranially engrafted with 3x10</span><span class="cls_006"><sup>4</sup></span><span class="cls_005"> syngeneic GL261</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">GBM cells at 12-week-of-age. Median survival indicated by intersecting dashed red lines. (</span><span class="cls_012">d</span><span class="cls_005">) Settings</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">for the cytometer setup &amp; tracking and PMT calibration beads used to standardize cytometer</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">performance across 3 consecutive data acquisitions. (</span><span class="cls_012">e</span><span class="cls_005">) Cytometer settings. (</span><span class="cls_012">f</span><span class="cls_005">) Configuration diagrams</span></div>
<div style="position:absolute;left:54.00px;top:678.24px" class="cls_005"><span class="cls_005">of the BD LSR II cytometer long-pass and band-pass filters used in this study. (</span><span class="cls_012">g</span><span class="cls_005">) High-throughput</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">66</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:52932px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-67.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_005"><span class="cls_005">sampler (HTS) settings used in the study. (</span><span class="cls_012">h</span><span class="cls_005">) Well locations of the optical controls run in a 96 well V-</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">bottom plate during each data acquisition. UNS = unstained, FVD = fluorescent FVD only, ISO = CD45</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_005"><span class="cls_005">isotype plus FVD, PRE = single-positive beads run before data acquisition, POST = single-positive</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_005"><span class="cls_005">beads run after data acquisition. (</span><span class="cls_012">i</span><span class="cls_005">) Radial bar chart showing the percentage of dead cells in the study’s</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">240 tissue samples. C = control, T = GBM. (</span><span class="cls_012">j</span><span class="cls_005">) Radial bar charts of cell number in each tissue sample</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_005"><span class="cls_005">from control (left) and GBM (right) mice after a weighted random sampling to help balance the number</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_005"><span class="cls_005">of cells per sample.</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_012"><span class="cls_012">Supplementary Figure 2</span><span class="cls_005"> Gate placement for each of the study’s 2,640 signal antibody intensity</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">histograms (11 detection channels x 5 tissue types x 48 mice). Kernel density estimates (KDEs) of</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">unstained splenocytes are superimposed (blue outlines). Data for the CD45 channel are shown as box-</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">and-whisker plots. uns = unstained splenocytes, cd45 = splenocytes labeled with CD45 single-color</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">control, cd45i = splenocytes labeled CD45 isotype control antibodies, 0 = experimental data</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">immunonegative for all other immunomarkers, 1 = experimental data immunopositive for at least of the</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">other immunomarkers, all = combined data from 0 and 1.</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_012"><span class="cls_012">Supplementary Figure 3</span><span class="cls_005"> A comparison of knowledge-based and data-drive cell subset identification on</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">the same dataset subsample. (</span><span class="cls_012">a</span><span class="cls_005">) Box plot distributions of Logicle-transformed antigen expression of the</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">30 immunophenotypes identified by SYLARAS and 30 clusters identified by the Phenograph algorithm.</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">In both cases antigen expression is shown side-by-side for control and GBM mice in descending rank</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">order according the median value of the control group. (</span><span class="cls_012">b</span><span class="cls_005">) Box plot distributions of Logicle-transformed</span></div>
<div style="position:absolute;left:54.00px;top:650.64px" class="cls_005"><span class="cls_005">antigen expression showing data for clusters 1, 7, 9, 15, 16, and 18.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">67</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:53734px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-68.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">Supplementary Figure 4</span><span class="cls_005"> Set of 30 SYLARAS dashboards (1 per immunophenotype).</span></div>
<div style="position:absolute;left:54.00px;top:126.24px" class="cls_012"><span class="cls_012">Supplementary Figure 5</span><span class="cls_005"> </span><span class="cls_016">Silhouette analysis identifying appropriate dendrogram linkage thresholds for</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_016"><span class="cls_016">condition-specific correlation clustermaps shown in Figure 5g. The number of clusters chosen for each</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_016"><span class="cls_016">clustermap (indicated by a white asterisk) was defined as the greatest number of clusters among which</span></div>
<div style="position:absolute;left:54.00px;top:209.04px" class="cls_016"><span class="cls_016">the fraction of members with negative silhouette scores was &lt; 10%; cluster members are indicated at the</span></div>
<div style="position:absolute;left:54.00px;top:236.64px" class="cls_016"><span class="cls_016">leafs of each dendrogram.</span></div>
<div style="position:absolute;left:54.00px;top:291.84px" class="cls_012"><span class="cls_012">Supplementary Figure 7</span><span class="cls_005"> Validation and application of a 12-color CyCIF antibody panel to survey the</span></div>
<div style="position:absolute;left:54.00px;top:319.44px" class="cls_005"><span class="cls_005">late-stage GBM immune landscape. (</span><span class="cls_012">a</span><span class="cls_005">) 11 antibodies validated on FFPE tissue sections of an inguinal</span></div>
<div style="position:absolute;left:54.00px;top:347.04px" class="cls_005"><span class="cls_005">lymph node from a B16 melanoma-bearing C57BL/6 mouse. Merged image is shown at left; thumbnail</span></div>
<div style="position:absolute;left:54.00px;top:374.64px" class="cls_005"><span class="cls_005">images of individual channels are shown at right. (</span><span class="cls_012">b</span><span class="cls_005">) Single-cell data derived from the segmented</span></div>
<div style="position:absolute;left:54.00px;top:402.24px" class="cls_005"><span class="cls_005">micrograph showing the fraction of total cells immunoreactive to each antibody. Antibody signal</span></div>
<div style="position:absolute;left:54.00px;top:429.84px" class="cls_005"><span class="cls_005">intensity is indicated by the heatmap. (</span><span class="cls_012">c</span><span class="cls_005">) Full-resolution PDF of a 168-tile, 12-plex mosaic</span></div>
<div style="position:absolute;left:54.00px;top:457.44px" class="cls_005"><span class="cls_005">immunofluorescence micrograph of the </span><span class="cls_016">tumor-ipsilateral mouse brain hemisphere bearing GL261 GBM</span></div>
<div style="position:absolute;left:54.00px;top:485.04px" class="cls_016"><span class="cls_016">36-days after its engraftment </span><span class="cls_005">acquired with a 40x objective. Tiles are indicated by X/Y coordinate and</span></div>
<div style="position:absolute;left:54.00px;top:512.64px" class="cls_005"><span class="cls_005">tile number and are shown in increasing order from the bottom-left to top-right. Immunomarker colors</span></div>
<div style="position:absolute;left:54.00px;top:540.24px" class="cls_005"><span class="cls_005">are as follows: Ly6C (orange), CD8α (yellow), CD68 (cyan), B220 (magenta), CD4 (red), CD49b</span></div>
<div style="position:absolute;left:54.00px;top:567.84px" class="cls_005"><span class="cls_005">(white), Ly6G (brown), Foxp3 (blue), CD11b (green), Ki67 (pink), Vimentin (violet), DAPI (gray)</span><span class="cls_012">. </span><span class="cls_005">(</span><span class="cls_012">d-f</span><span class="cls_005">)</span></div>
<div style="position:absolute;left:54.00px;top:595.44px" class="cls_005"><span class="cls_005">Examples of B220/CD8α double-positive cells, CD8α single-positive cells, and several low-abundance</span></div>
<div style="position:absolute;left:54.00px;top:623.04px" class="cls_005"><span class="cls_005">cell types from the micrograph shown in panel (c); tile coordinates are provided for cross-referencing.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">68</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:54536px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-69.jpg" width=612 height=792></div>
<div style="position:absolute;left:223.71px;top:35.04px" class="cls_005"><span class="cls_005">SYLARAS (Systemic Lymphoid Architecture Response Assessment)</span></div>
<div style="position:absolute;left:54.00px;top:71.04px" class="cls_012"><span class="cls_012">Supplementary Table 1 </span><span class="cls_005">Antibody details and protocol for 12-plex CyCIF on mouse FFPE tissue</span></div>
<div style="position:absolute;left:54.00px;top:98.64px" class="cls_005"><span class="cls_005">sections.</span></div>
<div style="position:absolute;left:54.00px;top:153.84px" class="cls_012"><span class="cls_012">Supplementary Table 2</span><span class="cls_005"> Excel spreadsheet for calculating the amounts of antibody stock and diluent,</span></div>
<div style="position:absolute;left:54.00px;top:181.44px" class="cls_005"><span class="cls_005">fixable viability dye, Fc block, resuspension volume for cell count estimation, and injectable anesthetics.</span></div>
<div style="position:absolute;left:546.00px;top:727.44px" class="cls_005"><span class="cls_005">69</span></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:55338px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-70.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:56140px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-71.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:56942px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-72.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:57744px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-73.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:58546px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-74.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:59348px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-75.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:60150px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-76.jpg" width=612 height=792></div>
</div>
<div style="position:absolute;left:50%;margin-left:-306px;top:60952px;width:612px;height:792px;border-style:outset;overflow:hidden">
<div style="position:absolute;left:0px;top:0px">
<img src="BIORXIV-2019-555854v1-Baker_files/background-77.jpg" width=612 height=792></div>
</div>

</body>
</html>
